<html><head>
 <title>
  Abstract 145410121 | 28th Fungal Genetics Conference
 </title>
</head>
<style type="text/css">
 body {
    font-family: Verdana, Geneva, sans-serif;
}
br.pagebreak {page-break-after: always;}
</style>
<body>
 Abstract Number:
 <b>
  145410121
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolism    Evolution    RNA-seq
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Examining the evolution of the regulatory circuit controlling secondary metabolism and development in
 <em>
  Aspergillus
 </em>
 .
 <br/>
 <b>
  Abigail Lind
 </b>
 <sup>
  1
 </sup>
 , Jennifer Wisecaver
 <sup>
  2
 </sup>
 , Timothy Smith
 <sup>
  3
 </sup>
 , Xuehan Feng
 <sup>
  3
 </sup>
 , Ana Calvo
 <sup>
  3
 </sup>
 , Antonis Rokas
 <sup>
  1,2
 </sup>
 . 1) Biomedical Informatics, Vanderbilt University, Nashville, TN.; 2) Biological Sciences, Vanderbilt University, Nashville, TN; 3) Biological Sciences, Northern Illinois University, Dekalb, IL
 <br/>
 <p>
  Filamentous fungi produce diverse secondary metabolites (SMs) essential to their ecology and adaptation. Although each SM is typically produced by only a handful of species, global SM production is governed by widely conserved transcriptional regulators in conjunction with other cellular processes, such as development. We examined the interplay between the taxonomic narrowness of SM distribution and the broad conservation of global regulation of SM and development in
  <em>
   Aspergillus
  </em>
  , a diverse fungal genus whose members produce well-known SMs. Evolutionary analysis of the 2,124 genes comprising the 262 SM pathways in four
  <em>
   Aspergillus
  </em>
  species showed that most SM pathways were species-specific, that the number of SM gene orthologs was significantly lower than that of orthologs in primary metabolism, and that the few conserved SM orthologs typically belonged to non-homologous SM pathways. RNA sequencing of two master transcriptional regulators of SM and development,
  <em>
   veA
  </em>
  and
  <em>
   mtfA,
  </em>
  showed that the effects of deletion of each gene, especially
  <em>
   veA
  </em>
  , on SM pathway regulation were similar in
  <em>
   A. fumigatus
  </em>
  and
  <em>
   A. nidulans
  </em>
  , even though the underlying genes and pathways regulated in each species differed. In contrast, examination of the role of these two regulators in development, where 94% of the underlying genes are conserved in both species showed that whereas the role of
  <em>
   veA
  </em>
  is conserved,
  <em>
   mtfA
  </em>
  regulates development in the homothallic
  <em>
   A. nidulans
  </em>
  but not in the heterothallic
  <em>
   A. fumigatus
  </em>
  . Thus, the regulation of these highly conserved developmental genes is divergent, whereas&#8211;despite minimal conservation of target genes and pathways&#8211;the global regulation of SM production is largely conserved. We suggest that the evolution of the transcriptional regulation of secondary metabolism in
  <em>
   Aspergillus
  </em>
  represents a novel type of regulatory circuit rewiring and hypothesize that it has been largely driven by the dramatic turnover of the target genes involved in the process.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410819
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  ChIP-seq    Secondary metabolism    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 KdmB, a histone H3 lysine K4 demethylase modulates gene expression of secondary metabolite gene clusters.
 <br/>
 <b>
  Agnieszka Gacek
 </b>
 <sup>
  1
 </sup>
 , Harald Berger
 <sup>
  2
 </sup>
 , Takahiko Sasaki
 <sup>
  3
 </sup>
 , Clemens Gruber
 <sup>
  4
 </sup>
 , Kathrin Wittstein
 <sup>
  5
 </sup>
 , Marc Stadler
 <sup>
  5
 </sup>
 , Zachary Lewis
 <sup>
  3
 </sup>
 , Joseph Strauss
 <sup>
  1,2
 </sup>
 . 1) Fungal Genetics and Genomics Unit, BOKU-University of Natural Resources and Life Sciences, Vienna, Austria; 2) Health and Environment Department, AIT - Austrian Institute of Technology GmbH, Austria; 3) Department of Microbiology, University of Georgia, Athens, USA; 4) Department of Chemistry, BOKU-University of Natural Resources and Life Sciences, Vienna, Austria; 5) Department Microbial Drugs, Helmholtz-Centre for Infection Research, Braunschweig, Germany
 <br/>
 <p>
  Chromatin remodelling regulates processes such as DNA repair, recombination, replication and transcription. Histone acetylation/deacetylation, methylation/demethylation and the associated binding of non-histone proteins, such as heterochromatin protein-1 (HepA) profoundly alter the chromatin landscape and associated gene expression profiles in toxigenic
  <em>
   Aspergilli
  </em>
  and pathogenic
  <em>
   Fusarium
  </em>
  species. These processes proved to be important for controlling the ON/ OFF state of SM biosynthetic genes as well as the fine tuning of their expression.
 </p>
 <p>
  Here, we present the dynamic process of histone H3 demethylation by a jumonji-type H3K4 demethylase termed KdmB. The purified enzyme removes the methyl mark from tri-methylated histone substrate
  <em>
   in vitro
  </em>
  . Furthermore, LC-MS/MS profiling of
  <em>
   A. nidulans
  </em>
  histone post-translational modifications (HPTMs) revealed increases in global H3K4me3 levels in
  <em>
   kdmB&#916;
  </em>
  . Remarkably, upon
  <em>
   kdmB
  </em>
  deletion most of the SM clusters expressed in our culture condition such as sterigmatocystin, penicillin, terrequinone and orsellinic acid were downregulated. In order to determine the locus specific KdmB targets and the functionality of observed HPTMs, ChIP-seq and RNA-seq analysis were performed.
 </p>
 <p>
  The genome wide profile of H3K4me3, H3K9me3, H3K36me3 and H3ac in primary and secondary metabolism condition in the wide type and upon deletion of
  <em>
   A. nidulans
  </em>
  sole H3K4 tri- demethylase, KdmB, will be presented.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410667
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Other    Whole Genome Sequencing (Other Keyword: Aspergillus)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Whole-Genus Sequencing: 300 Aspergilli.
 <br/>
 <b>
  Alan Kuo
 </b>
 <sup>
  1
 </sup>
 , Robert Riley
 <sup>
  1
 </sup>
 , Alicia Clum
 <sup>
  1
 </sup>
 , Asaf Salamov
 <sup>
  1
 </sup>
 , Scott Baker
 <sup>
  2
 </sup>
 , Blake Simmons
 <sup>
  2
 </sup>
 , Mikael Andersen
 <sup>
  3
 </sup>
 , Igor Grigoriev
 <sup>
  1
 </sup>
 . 1) Fungal Genomics Program, DOE Joint Genome Institute, Walnut Creek, CA; 2) Joint BioEnergy Institute, Emeryville, CA; 3) Department of Systems Biology, Technical University of Denmark, Kgs. Lyngby, Denmark
 <br/>
 <p>
  <em>
   Aspergillus
  </em>
  is a ubiquitous and phenotypically diverse genus of filamentous Ascomycota, many of which play key roles as fermenters in food production, platforms for biotechnology and industrial production of enzymes and chemicals, plant and opportunistic animal pathogens, and agents of agricultural toxigenesis and biomass conversion for bioenergy. As part of a DOE Joint BioEnergy Institute initiative to characterize the entire genus, the JGI plans to sequence, assemble, and annotate the genomes of each of the ~300 species of the genus
  <em>
   Aspergillus
  </em>
  . To accomplish this massive task in a timely manner without sacrificing quality, we have sought to streamline our existing processes as well as explore alternative technologies, especially assembly and annotation of long PacBio sequencing reads. Over the past year we have released on MycoCosm the genomes of an additional 24
  <em>
   Aspergillus
  </em>
  sp. with preliminary analyses of their phylogenies, secretomes, and secondary metabolism. The next tranche of 92 species is expected soon.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410389
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Metabolism    Virulence
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Undergraduate Student
 </b>
 <br/>
 <br/>
 The proteinogenic and non proteinogenic function of histidine in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Anna-Maria Dietl
 </b>
 <sup>
  1
 </sup>
 , Nicola Beckmann
 <sup>
  1
 </sup>
 , Ulrike Binder
 <sup>
  2
 </sup>
 , Jorge Amich
 <sup>
  3
 </sup>
 , Sixto Leal
 <sup>
  4
 </sup>
 , Eric Pearlman
 <sup>
  4
 </sup>
 , Hubertus Haas
 <sup>
  1
 </sup>
 . 1) Division of Molecular Biology, Medical University Innsbruck, Tyrol, Austria; 2) Division of Hygiene and Medical Microbiology, Medical University Innsbruck, Tyrol, Austria; 3) Medical University Würzburg, Germany; 4) Case Western Reserve University, Ohio, United States of America
 <br/>
 <p>
  <em>
   Aspergillus fumigatus
  </em>
  is the most prevalent airborne fungal pathogen causing invasive fungal infections in immunosuppressed individuals. Limitations in antifungal therapy arise from non-specific symptoms of infection, poor diagnostics and comparatively few options for treatment. Novel antifungal therapy approaches target fungal-specific pathways that are essential for virulence. One potential example is the biosynthesis of the essential amino acid histidine. Bacteria, plants und fungi produce histidine via a highly conserved pathway, in
  <em>
   A. fumigatus
  </em>
  encoded by eight genes. In contrast, animals and humans do not synthesize histidine and satisfy their demand via nutritional uptake. Here we demonstrate that lack of histidine biosynthesis due to genetic abrogation of the gene encoding imidazoleglycerol-phosphate dehydratase (HisB) causes histidine auxotrophy and virulence attenuation in four virulence models:
  <em>
   Galleria mellonella
  </em>
  , murine pulmonary aspergillosis, murine systemic infection and murine keratitis. In agreement with the
  <em>
   in vivo
  </em>
  importance of histidine biosynthesis, the HisB inhibitor 3-amino-1,2,4-triazol (3-AT) reduced the virulence of the
  <em>
   A. fumigatus
  </em>
  wildtype in
  <em>
   Galleria mellonella
  </em>
  . In line with a crucial role of histidine in cellular handling of metals due to its chelator activity, HisB-deficiency decreased the resistance of
  <em>
   A. fumigatus
  </em>
  to a variety of metals including iron, zinc, nickel, cobalt, copper and manganese. Taken together, this study reveals (i) limited histidine availability in different
  <em>
   A. fumigatus
  </em>
  host niches and (ii) the histidine biosynthetic pathway as target for novel antifungal therapy approaches. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410556
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Secondary metabolism    Development
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Sclerotia formation in
 <em>
  Aspergillus niger
 </em>
 is accompanied by expression of otherwise silent secondary metabolite gene clusters.
 <br/>
 <b>
  Arthur Ram
 </b>
 , Anne-Marie Burggraaf-van Welzen, Mark Arentshorst, Thomas Jorgensen. Molecular Microbiology and Biotechnology, Leiden University, Institute Biology Leiden, Leiden, Netherlands
 <br/>
 <p>
  Sclerotia are compact mycelial masses with hardened, thick walls and a less dense stroma. They play a role in dormancy, serving to survive adverse environmental conditions and are considered to be an important prerequisite for sexual development. Several environmental factors have been shown to be correlated to the production of sclerotia, such as absence or presence of light, medium composition, oxygen availability and temperature.
  <em>
   Aspergillus niger
  </em>
  is a biotechnologically important fungus which is only known to proliferate asexually. Sclerotia formation of naturally isolated strains of
  <em>
   A. niger
  </em>
  has only be recently reported in certain
  <em>
   A. niger
  </em>
  strains grown on specific medium containing raisins, other fruits or rice. We previously described an
  <em>
   Aspergillus niger
  </em>
  mutant (
  <em>
   scl-2
  </em>
  ) displaying a reduced conidiation phenotype and forming abundant sclerotia on commonly used rich medium agar plates not supporting sclerotium formation in the WT. In this study, we characterized the sclerotia forming mutant (
  <em>
   scl-2
  </em>
  ) in detail. Several lines of evidence support that the multicellular structures are indeed sclerotia:
  <em>
   i)
  </em>
  Safranin staining of microscopic coupes of the sclerotia-like structures produced showed the typical cellular structure of a cell dense outer layer and a less dense inside in the sclerotium,
  <em>
   ii)
  </em>
  formation of the sclerotia-like structures is inhibited by light iii) dependent on the oxidative state of the mycelium requiring a functional
  <em>
   noxA
  </em>
  /
  <em>
   noxR
  </em>
  -dependent NADPH oxidase complex. Genome-wide expression analysis of the
  <em>
   scl-2
  </em>
  mutant suggests specific sclerotium dependent production of indoloterpenes. Inspection of gene expression data available for
  <em>
   Aspergillus niger
  </em>
  which includes over 150 growth conditions revealed that three secondary metabolite gene clusters of which two are indicated to be related to indoloterpene synthesis were uniquely expressed in sclerotia.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410559
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Development    Whole Genome Sequencing
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 The use of the parasexual cycle and bulk segregant analysis followed by high-throughput sequencing to characterize a sclerotia-forming mutant in
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  Arthur Ram
 </b>
 , Anne-Marie Burggraaf-van Welzen, Jing Niu, Thomas Jørgensen, Mark Arentshorst. Molecular Microbiology and Biotechnology, Leiden University, Institute Biology Leiden, Leiden, Netherlands
 <br/>
 <p>
  Much of our current understanding in fungal growth and development is derived from forward genetic screens to select developmental mutants. Identification of the molecular basis for a particular phenotype in UV-generated mutants is tedious and time consuming. Whole genome sequencing (WGS) provides an effective alternative to identify the molecular lesion in a mutant isolated from a genetic screen. We have successfully used the so called &#8220;bulk segregants analysis&#8221; approach in combination with WGS to characterize the molecular basis for a hypersclerotial mutant (
  <em>
   scl-2
  </em>
  ) in
  <em>
   A. niger
  </em>
  . Since
  <em>
   A. niger
  </em>
  has no sexual cycle, we used the parasexual cycle to obtain segregants. In the bulk-segregant analysis approach, the mutant of interest is crossed to a wild-type strain and segregants displaying the phenotype of interest are pooled and DNA from this pool of segregants is sequenced using a deep sequencing technology (e.g. Illumina). In addition, the genomes of the parental strains were also sequenced and SNPs and indels were identified. SNPs and indels between the parental strains not related with the phenotype, have a 50% chance to be present in the pool; SNPs and indels responsible for the phenotype or closely linked to the mutation responsible for the phenotype, are conserved in the pool. Sclerotia formation was found to be caused by a non-sense mutation in the Zn(II)2Cys6 domain of an until now unknown transcription factor. Subsequent complementation analysis and targeted deletion are in progress to confirm its role as a potential repressor of sclerotia formation in
  <em>
   A. niger
  </em>
  . The opportunity to use WGS approaches to pinpoint the molecular lesion in a mutated strain isolated from a genetic screen will speed up genetic identification making is suitable for new approaches of near complete mutational saturation of a biological process and the resulting unravelling of entire genetic pathways and networks.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410563
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Cell Wall    RNA-seq
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Genome-wide transcriptome analysis of cell wall remodeling in
 <em>
  Aspergillus niger
 </em>
 in response to the absence of galactofuranose biosynthesis.
 <br/>
 Joohae Park
 <sup>
  1
 </sup>
 , Mark Arentshorst
 <sup>
  1
 </sup>
 , Boris Tefsen
 <sup>
  2
 </sup>
 , Ellen Lagendijk
 <sup>
  1
 </sup>
 , Cees van den Hondel
 <sup>
  1
 </sup>
 , Irma van Die
 <sup>
  2
 </sup>
 ,
 <b>
  Arthur Ram
 </b>
 <sup>
  1
 </sup>
 . 1) Molecular Microbiology and Biotechnology, Leiden University, Institute Biology Leiden, Leiden, The Netherlands; 2) Department of Molecular Cell Biology and Immunology, VU University Medical Center, Amsterdam, The Netherlands
 <br/>
 <p>
  The biosynthesis of cell wall galactofuranose (Gal
  <em>
   f
  </em>
  ) containing glycostructures such as galactomannan,
  <em>
   N
  </em>
  -glycans,
  <em>
   O
  </em>
  -glycans and glycosylinositolphosphoceramides in filamentous fungi are important to secure the integrity of the cell wall. A key gene in the biosynthesis of UDP-Galf is
  <em>
   ugmA
  </em>
  which encodes a UDP-galactopyranose mutase which is essential for the conversion of UDP-Gal
  <em>
   p
  </em>
  to UDP-Gal
  <em>
   f
  </em>
  .  In A
  <em>
   . niger
  </em>
  , the absence of Gal
  <em>
   f
  </em>
  synthesis results in activation of the cell wall integrity (CWI)-pathway indicating that the Gal
  <em>
   f
  </em>
  biosynthesis is important for maintaining cell wall strength. To identify genes involved in maintaining cell wall integrity in response to the absence of galactofuranose biosynthesis, a genome-wide expression study was performed with the
  <em>
   ugmA
  </em>
  deletion strain. RNAseq analysis revealed 432 upregulated genes to be differentially expressed (Q-value &lt;0.05) in the
  <em>
   ugmA
  </em>
  mutant compared to the wild-type and these genes encode enzymes involved in alpha-glucan synthesis (
  <em>
   agsA
  </em>
  ), chitin synthesis (
  <em>
   gfaB, gnsA,
  </em>
  and
  <em>
   chsA
  </em>
  ), beta-glucan remodeling (
  <em>
   bgxA, gelF,
  </em>
  and
  <em>
   dfgC
  </em>
  ) and several (GPI)-anchored cell wall protein encoding genes. Interestingly, also the gene encoding the CWI-specific Map-kinase-kinase (
  <em>
   mkkA
  </em>
  ) was induced in the
  <em>
   ugmA
  </em>
  mutant.
  <em>
   In silico
  </em>
  analysis of the 1-kb promoter regions of the differentially up-regulated genes in the
  <em>
   ugmA
  </em>
  mutant using an in house developed transcription factor binding site finder program, indicated overrepresentation of genes with RlmA or SteA binding sites.  The importance of the RlmA and SteA transcription to induce cell wall remodelling genes is currently under investigation by constructing a
  <em>
   rlmA-ugmA
  </em>
  and
  <em>
   steA-ugmA
  </em>
  double mutants.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410108
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Fumigatin-oxide production by
 <em>
  Aspergillus fumigatus
 </em>
 is regulated by iron availability and temperature involving the transcription factors HapX and SrbA.
 <br/>
 <b>
  Beatrix E. Lechner
 </b>
 <sup>
  1
 </sup>
 , Ernst R. Werner
 <sup>
  2
 </sup>
 , Markus A. Keller
 <sup>
  2
 </sup>
 , Kirstin Scherlach
 <sup>
  3
 </sup>
 , Falk Hillmann
 <sup>
  4
 </sup>
 , Hubertus Haas
 <sup>
  1
 </sup>
 . 1) Division of Molecular Biology, Innsbruck Medical University, Innsbruck, Austria; 2) Division of Biological Chemistry, Innsbruck Medical University, Innsbruck, Austria; 3) Department of Biomolecular Chemistry, HKI, Jena, Germany; 4) Department of Molecular and Applied Microbiology, HKI, Jena, Germany
 <br/>
 <p>
  Iron is an essential metal for the metabolism of virtually all species. For the opportunistic fungal pathogen
  <em>
   Aspergillus fumigatus,
  </em>
  adaptation to iron starvation has been shown to be an essential virulence determinant. Here we found that in
  <em>
   A. fumigatus
  </em>
  liquid cultures, iron starvation induces the secretion of a yellow pigment optimally at 20-25°C, but not at 37°C, within 48-72 h of growth. In contrast, starvation for nitrogen, carbon, phosphate or other metals such as copper or zinc did not trigger production of this extracellular pigment. Deficiency in HapX or SrbA, the master regulators for adaptation to iron starvation and secondary metabolism, respectively, impaired biosynthesis of the pigment. In contrast to
  <em>
   A. fumigatus,
  </em>
  <em>
   Aspergillus nidulans, Aspergillus terreus
  </em>
  or
  <em>
   Penicillium chrysogenum
  </em>
  were not found to synthesize this pigment. The pigment was purified by solid phase extraction and reversed-phase HPLC separations. High-resolution mass spectrometry revealed a molecular mass of 183.0297 corresponding to the chemical formula of C
  <sub>
   8
  </sub>
  H
  <sub>
   8
  </sub>
  O
  <sub>
   5
  </sub>
  . 1H-NMR together with the photosensitivity and the pH dependence of UV absorption spectra identified the compound as fumigatin-oxide. The aromatic, maroon-colored metabolite fumigatin, showing antibiotic activity against bacteria and toxicity against animals as well as anti-inflammatory activity, was first isolated from
  <em>
   A. fumigatus
  </em>
  culture media in 1938. Subsequently, fumigatin and its derivatives were detected by mass spectrometry in various
  <em>
   A. fumigatus
  </em>
  isolates but the biosynthetic pathway for fumigatin biosynthesis remains to be elucidated. This study represents the first characterization of the regulation of fumigatin production and emphasizes the impact of iron availability on fungal secondary metabolism.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410079
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Metabolism    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Regulation of Nuclear Import of the
 <em>
  Aspergillus nidulans
 </em>
 GATA Transcription Factor AreA.
 <br/>
 <b>
  Cameron Hunter
 </b>
 <sup>
  1
 </sup>
 , Margaret Katz
 <sup>
  2
 </sup>
 , Richard Todd
 <sup>
  1
 </sup>
 . 1) Plant Pathology, Kansas State University, Manhattan, KS; 2) Molecular and Cellular Biology, University of New England, Armidale, NSW
 <br/>
 <p>
  The TOR signaling pathway is conserved throughout eukaryotes and coordinates cell growth in response to nutrient availability. In
  <em>
   Saccharomyces cerevisiae
  </em>
  the TOR pathway is inhibited by the anti-tumorigenic immunosuppressant, rapamycin. This leads to downstream effects, including nuclear localization of the GATA transcription factors Gln3p and Gat1p, which activate nitrogen metabolic genes. The components of the TOR pathway are conserved between yeast and
  <em>
   Aspergillus nidulans.
  </em>
  We have used an epitope tagged version of AreA to investigate the effects of rapamycin and the TOR pathway on AreA subcellular localization. In stark contrast to triggering nuclear import of Gln3p and Gat1p in yeast, rapamycin does not cause nuclear localization of AreA. Instead, rapamycin prevents AreA nuclear accumulation in response to nitrogen starvation. Nuclear accumulated AreA is rapidly exported from the nucleus in response to nitrogen nutrients or carbon starvation, but addition of rapamycin to nitrogen-starved cells does not trigger AreA nuclear export, suggesting that the TOR pathway controls AreA nuclear import. We are investigating the effects on AreA nuclear localization of the other components in the TOR pathway SitA, JipA, and GstA. In yeast, deletion of the cytoplasmic anchor Ure2p confers Gln3p nuclear localization. In contrast, deletion of the Ure2p homolog GstA prevents AreA nuclear accumulation. We also show that AreA does not accumulate in the nucleus in a loss of function mutant affecting the carbon starvation and autolysis transcription factor XprG. Collectively our data suggests rewiring of the TOR pathway for controlling AreA nuclear import in
  <em>
   A. nidulans..
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410605
 </b>
 <br/>
 Research Area:
 <b>
  Pathogenic and Mutalistic Interactions
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Pathogenicity    Cell Wall
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Application of the high-throughput
 <em>
  Aspergillus fumigatus
 </em>
 cell wall-stress reporter system to identify synthetic peptides increasing the sensitivity for antifungal medicines.
 <br/>
 <b>
  Cees van den Hondel
 </b>
 <sup>
  1,2
 </sup>
 , Ellen Lagendijk
 <sup>
  1
 </sup>
 , Sophie Meier zu Ummeln
 <sup>
  1
 </sup>
 , Christien Lokman
 <sup>
  3
 </sup>
 , Arthur Ram
 <sup>
  1
 </sup>
 . 1) DLeiden University, Institute of Biology Leiden, Molecular Microbiology, Sylviusweg 72, 2333 BE Leiden, The Netherlandsept Mol Microbiol.,Inst Biol, Univ Leiden, Leiden, Netherlands; 2) HiTexacoat, Gouda, The Netherlands; 3) HAN BioCentre, Laan van Scheut 2, P.O.Box 6960, NL-6503 GL Nijmegen, The Netherlands
 <br/>
 <p>
  Increased resistance to currently used antifungal compounds and the fact that these agents are often harmful to man and environment have resulted in a growing demand for new antifungals, which selectively act on cellular processes that are unique to fungi. To meet this demand, we have established in A. niger  a luciferin/luciferase based reporter system for high-throughput screening of natural products for identification of potential new antifungal drugs). Our system allows us to identify compounds that specifically inhibited the fungal cell wall biosynthesis. Recently a similar system has been developed for A. fumigatus as well as for other non-Aspergillus species.
 </p>
 <p>
  On our poster we will show the validation of our system by analysing its performance with several antifungal drugs which are commonly used in the clinic like Caspofungin, Amphotericin B, Voriconazol (V fend) as well as Nikkomycin. This analysis clearly shows not only  a doses response  behaviour of the compounds tested but also that in time different types of stress responses occur. Importantly, also the analysis shows a clear indication whether the compound tested is fungistatic or fungicidal.
 </p>
 <p>
  Subsequent analysis of different synthetic antimicrobial peptides, HTX1-4, revealed also a moderate effect on cell wall stress induction indicated by an increase in Lux activity. Interestingly, incubation of these peptides at sublethal concentrations together with some of the antifungal medicines showed a considerable increase in lux activity and  a significant increase in sensitivity of A.fumigatus for these medicines.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410616
 </b>
 <br/>
 Research Area:
 <b>
  Other
 </b>
 <br/>
 Keyword(s):
 <b>
  DNA Repair    Gene Deletion    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 A CRISPR/Cas9 system for genetic engineering of filamentous fungi.
 <br/>
 <b>
  Christina S Noedvig
 </b>
 , Jakob B. Nielsen, Uffe H. Mortensen. DTU Systems Biology, Technical University of Denmark, Kgs. Lyngby, Denmark
 <br/>
 <p>
  The harnessing of the prokaryotic and archaeal immune mechanism CRISPR (clustered regularly interspaced short palindromic repeats) as a tool for genetic engineering in eukaryotes, has proved to be a powerful technology. CRISPR/Cas9 introduces specific DNA double strand breaks (DSB) with high precision, which in turn can be employed to efficiently stimulate gene targeting. Consisting of two components, an RNA guided nuclease Cas9 and a chimeric guide RNA (gRNA), a specific DSB can be produced in the host organism. The cleavage target site is determined by 20 base pairs (bp) in the gRNA, and by exchanging those 20 bp, Cas9 can be programmed to target a specific chromosomal location with few constraints. The technology has had a huge impact on genetic engineering of organisms, such as plants or mammalian cells where gene targeting is notoriously inefficient, but has so far not been adapted to filamentous fungi. Low gene targeting frequencies is a common problem when attempting to do gene editing in filamentous fungi. A common strategy to circumvent this problem is to delete or disable one of the key genes in the non-homologous end-joining (NHEJ) pathway to greatly enhance gene-targeting frequencies. However, for fungi where a genetic toolbox is not in place, the initial establishment of genetic markers and NHEJ-deficiency can be laborious. Here we present a CRISPR/Cas9 system adapted for filamentous fungi and show that it can be efficiently used to introduce specific genomic modifications. Considering that the number of fully sequenced fungi is dramatically increasing, and that the vast majority of these fungi does not possess a genetic toolbox, our system will be a highly useful in developing the initial marker- and NHEJ gene mutations to establish such a toolbox. To this end, we have also developed a gRNA design software that facilitates identification of gRNA sequences that can target a desired gene in several different species, hence, reducing the plasmid construction workload. Together, we envision that our tools can be used to rapidly expand the repertoire of fungi where genetic engineering is possible and therefore greatly accelerate the exploration of fungal biology.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410208
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Secondary metabolism    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Influence of microgravity on the production of
 <em>
  Aspergillus nidulans
 </em>
 secondary metabolites onboard the International Space Station.
 <br/>
 <b>
  Clay Wang
 </b>
 <sup>
  1
 </sup>
 , Kasthuri Venkateswaran
 <sup>
  2
 </sup>
 , Junko Yaegashi
 <sup>
  1
 </sup>
 , Jillian Romsdahl
 <sup>
  1
 </sup>
 . 1) Pharma Sci &amp; Chemistry, Univ Southern California, Los Angeles, CA.; 2) Jet Propulsion Laboratory, Pasadena, CA
 <br/>
 <p>
  In this poster I will present our project recently funded by the NASA Space Biology program to study the production of secondary metabolites by
  <em>
   Aspergillus nidulans
  </em>
  onboard the International Space Station. Research from the Wang lab and many others in the field have shown that in filamentous fungi secondary metabolite production is highly sensitive to growth conditions. In addition it has been shown that many secondary metabolism pathways are triggered specifically in harsh or stressful conditions. Therefore we are interested in understanding how filamentous fungi respond to microgravity conditions. There are two long term goals for this project. One is to discover novel secondary metabolites in microgravity conditions. Second is to develop filamentous fungi as a synthetic biology platform for producing pharmaceutical compounds for future manned space mission. In addition I will present data on our metabolite analysis of fungi recovered from the International Space Station Filter Debri as part of the NASA funded ISS microbial observatory project.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410153
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Virulence    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Invited
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Calcium signaling and cytoskeletal dynamics during hyphal growth and infection by the human fungal pathogen
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Constanze Seidel
 </b>
 , Alberto Muñoz, Nick D. Read. Manchester Fungal Infection Group, University of Manchester, Manchester, United Kingdom
 <br/>
 <p>
  The most common human mould infection is caused by
  <em>
   Aspergillus
  </em>
  <em>
   fumigatus
  </em>
  , and it has been estimated that more than 3 million people worldwide are infected with it. The capacity of
  <em>
   A. fumigatus
  </em>
  to respond appropriately to external signals underpins its saprotrophic and parasitic lifestyles, and calcium signaling plays a major role in this. Ca
  <sup>
   2+
  </sup>
  -signaling and homeostasis are essential for the growth, differentiation and virulence of filamentous fungi although relatively little is known about the role of Ca
  <sup>
   2+
  </sup>
  in these processes in filamentous fungi, and particularly in human fungal pathogens during invasive growth. We have developed techniques for the routine imaging of Ca
  <sup>
   2+
  </sup>
  -dynamics in living
  <em>
   A. fumigatus
  </em>
  cells expressing the genetically encoded Ca
  <sup>
   2+
  </sup>
  -sensor, GCaMP6s. Time-lapse-imaging of continuously growing hyphae of different filamentous fungi has shown that, instead of growing hyphae possessing a constant tip-focused gradient, transient pulsatile increases in Ca
  <sup>
   2+
  </sup>
  -changes in hyphal tips occur with no discernible Ca
  <sup>
   2+
  </sup>
  -gradient between pulses. Furthermore, we have demonstrated different Ca
  <sup>
   2+
  </sup>
  -signatures in response to different types of environmental stress that might be encountered during infection. Cytoskeletal dynamics and motor proteins are also regulated by Ca
  <sup>
   2+
  </sup>
  . To determine how the F-actin and the microtubule cytoskeleton are dynamically organized whilst growing over alveolar epithelial cell layers, we generated strains expressing &#945;-tubulin and lifeact fused to GFP and tagRFP-T. These live-cell imaging studies have been combined with the use of pharmacological agents and mutant analyses to understand the roles of calcium signaling and the cytoskeleton during hyphal growth and infection. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410467
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Transcriptome    Aspergillus fumigatus    Secondary metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Comparative transcriptome analysis of
 <em>
  Aspergillus fumigatus
 </em>
 conidia reveals a novel secondary metabolism gene cluster specifically expressed at low temperature.
 <br/>
 <b>
  D. Hagiwara
 </b>
 <sup>
  1
 </sup>
 , K. Sakai
 <sup>
  1
 </sup>
 , S. Suzuki
 <sup>
  2
 </sup>
 , K. Kamei
 <sup>
  1
 </sup>
 , T. Gonoi
 <sup>
  1
 </sup>
 , S. Kawamoto
 <sup>
  1
 </sup>
 . 1) Medical Mycology Research Center, Chiba university, Chiba, Japan; 2) National Food Research Institute, Ibaraki, Japan
 <br/>
 <p>
  Filamentous fungi vigorously produce asexual spores (conidia) under appropriate conditions. Conidia are reproductive structures that are important for both distribution and survival for fungi. To understand effects of culture temperature on tolerance of conidia to various stresses, we compared sensitivities to heat, hydrogen peroxide, and UV irradiation among
  <em>
   Aspergillus fumigatus
  </em>
  conidia harvested from cultures at different temperatures (25, 37, and 45°C). The conidia from 25°C-culture showed a lower tolerance to heat stress (60°C) and to oxidative stress (H
  <sub>
   2
  </sub>
  O
  <sub>
   2
  </sub>
  ) compared with the other conidia, and showed a marked resistance to UV stress. We found that accumulation of trehalose, which plays a protective role in heat stress, was reduced in the conidia from 25°C-culture. Furthermore, the color of conidia from 25°C-culture was darker than those from 37 and 45°C-cultures, suggesting an increased melanin on the surface of conidia from 25°C-culture.
 </p>
 <p>
  To gain more insight into temperature-specific accumulation of secondary metabolites other than melanin in conidia, we investigated the transcriptome in the conidia from 25°C-, 37°C-, and 45°C-cultures. The transcriptome data revealed that melanin biosynthesis gene cluster was increasingly expressed in the conidia from 25°C-culture, supporting the above hypothesis. We also found one novel secondary metabolite gene cluster, which showed a higher expression in the conidia from 25°C-culture compared to those from 37°C- and 45°C-culture. This cluster contains 13 genes including genes encoding a PKS and a C6-type transcription factor (AfAflR) homologous to aflatoxin biosynthesis regulating AflR. In the AfaflR gene deletion mutant, expression levels of 11 genes of the cluster were greatly reduced, suggesting that the AfAflR plays a central role in transcriptional regulation for this cluster. We are going to identify the metabolite produced from this cluster at low-temperature, and to discuss temperature-dependent secondary metabolite production in the conidia of the pathogenic fungus.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410707
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Metabolism    Gene Deletion    Other (Other Keyword: Amino acid biosynthesis)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Duplication and redundancy of leucine, valine and isoleucine biosynthesis genes in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  Damien Downes
 </b>
 , Pierre Migeon, Cameron Hunter, Richard Todd. Department of Plant Pathology, Kansas State University, Manhattan, Kansas, USA
 <br/>
 <p>
  The branched chain amino acids (BCAA) leucine, isoleucine, and valine are essential dietary amino acids in mammals; fungi, however, can synthesize these three amino acids. For this reason, BCAA biosynthesis enzymes have been suggested as possible drug targets for treatment of infections by opportunistic pathogens such as
  <em>
   Aspergillus fumigatus
  </em>
  . Synthesis of the three BCAAs has been well characterized in the yeast
  <em>
   Saccharomyces cerevisiae
  </em>
  . However, recent work on the BCAA pathway enzymes dihydroxyacid dehydratase in
  <em>
   A. fumigatus
  </em>
  and &#945;-isopropylmalate synthase
  <em>
   Aspergillus nidulans
  </em>
  has shown divergence with
  <em>
   S. cerevisiae
  </em>
  in the number of genes encoding functional enzymes for these steps. The final two steps of leucine biosynthesis are carried out by &#946;-isopropylmalate dehydrogenase (&#946;-IDH) and BCAA aminotransferase (BAT), but the genes encoding these enzymes have not yet been characterized in the Aspergilli. In
  <em>
   S. cerevisiae
  </em>
  , there is one &#946;-IDH gene and two BAT genes. The BATs also catalyze the final step of isoleucine and valine production. Using protein sequence similarity we identified two &#946;-IDH encoding genes in
  <em>
   A. nidulans
  </em>
  :
  <em>
   leuD
  </em>
  and
  <em>
   leuE.
  </em>
  We have deleted these genes by gene replacement and shown that
  <em>
   leuD
  </em>
  &#916;, but not
  <em>
   leuE
  </em>
  &#916;, causes a leaky leucine auxotrophy. A
  <em>
   leuD
  </em>
  &#916;
  <em>
   leuE
  </em>
  &#916; double mutant is a strict leucine auxotroph and therefore both genes encode functional enzymes. Using quantitative RT-PCR we showed the difference in phenotypes is likely due to higher expression of
  <em>
   leuD
  </em>
  than
  <em>
   leuE
  </em>
  . We have also identified six putative BAT encoding genes,
  <em>
   batA-F,
  </em>
  by protein sequence similarity to Bat1p and Bat2p from yeast. Deletion of these six genes separately does not result in BCAA auxotrophy. However, simultaneous deletion of the two most highly expressed genes
  <em>
   batA
  </em>
  and
  <em>
   batB
  </em>
  is sufficient to confer BCAA auxotrophy, suggesting that
  <em>
   batC-F
  </em>
  may have evolved new roles within
  <em>
   A. nidulans
  </em>
  since duplication. With the identification of
  <em>
   leuD
  </em>
  ,
  <em>
   leuE
  </em>
  ,
  <em>
   batA
  </em>
  and
  <em>
   batB
  </em>
  the genes encoding the final steps of branched chain amino acid biosynthesis in
  <em>
   A. nidulans
  </em>
  have now been elucidated.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410710
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  RNA-seq    ChIP-seq    Transcription factor
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Pan-genomic analysis of TamA in
 <em>
  Aspergillus nidulans
 </em>
 : the regulatory network of a dual function Zn(II)2Cys6 transcription factor.
 <br/>
 <b>
  Damien Downes
 </b>
 <sup>
  1
 </sup>
 , Koon Ho Wong
 <sup>
  2
 </sup>
 , Richard Todd
 <sup>
  1
 </sup>
 . 1) Department of Plant Pathology, Kansas State University, Kansas, USA; 2) Faculty of Health Sciences, University of Macau, Macau SAR, China
 <br/>
 <p>
  Zn(II)2Cys6 zinc binuclear cluster proteins represent the largest family of transcription factors in fungi and regulate genes of diverse pathways, including primary and secondary metabolism, and development. For most transcription factors the DNA binding domain is considered essential for function. However proteins in the emerging class of dual function transcription factors operate by either direct DNA binding or as protein-binding co-activators and co-repressors (non-DNA-binding factors). The switch between functions can be determined by promoter context and cell type. Dual function has been reported in different cell types for the Hairy repressor in
  <em>
   Drosophilia, Xenopus
  </em>
  and zebrafish as well as mouse SCL/TAL-1. We recently reported the first example of a fungal dual function transcription factor, the
  <em>
   Aspergillus nidulans
  </em>
  Zn(II)2Cys6 transcription factor TamA. TamA is unique in that dual function occurs in the same cell type, where it binds the
  <em>
   gdhA
  </em>
  promoter directly while acting as a co-activator of the global nitrogen GATA transcription factor AreA at
  <em>
   amdS
  </em>
  and
  <em>
   fmdS
  </em>
  . To determine the extent of the TamA dual function regulatory network we have taken a pan-genomic approach. Using FLAG-epitope-tagged TamA for ChIP-seq, we have identified novel targets of TamA, including genes involved in primary metabolism, secondary metabolism and siderophore biosynthesis. This analysis has indicated that TamA target sites lack sequence conformity. Therefore TamA may act via multiple DNA-binding sequences, potentially on the basis of interaction with novel proteins. To determine the TamA activator and co-activator regulons we characterized the effect of TamA on global gene expression using stranded RNA-seq in
  <em>
   tamA
  </em>
  &#916; and
  <em>
   tamA
  </em>
  <sup>
   C90L
  </sup>
  loss-of-DNA binding mutants. The dual functions of transcription factors such as TamA provide an additional level of combinatorial control to mediate gene-specific expression. Analysis of the TamA regulatory network at a pan-genomic level provides insights into how dual function transcription factors facilitate promoter plasticity and act as a vehicle for adaptive evolution.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410359
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Sexual Development    Transcription factor
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Nuclear localization of NsdD affected by various genetic and environmental factors is important in development decision of
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  Dong-Min Han
 </b>
 <sup>
  1
 </sup>
 , Ki-Hong Park
 <sup>
  1
 </sup>
 , Lee-Han Kim
 <sup>
  1
 </sup>
 , Kap-Hoon Han
 <sup>
  2
 </sup>
 . 1) Div Life Sci, Wonkwang Univ, Iksan, Jeonbuk, South Korea; 2) Dept. Pharmaceutical Engineering, Woosuk Univ. Wanju 565-701, South Korea
 <br/>
 <p>
  NsdD, the GATA type transcription factor, which controls sexual development of
  <em>
   Aspergillus nidulans
  </em>
  has a nuclear localization sequence (NLS) just upstream of C4 zinc finger domain. The sGFP gene was fused onto N-terminus of
  <em>
   nsdD
  </em>
  ORF and used to transform the
  <em>
   nsdD
  </em>
  deletion mutant. The transformed strains partially complementing the mutant phenotype were isolated and the green fluorescence was traced by confocal laser microscope. The NsdD was found to be localized in nuclei near germination origin in 12 h old hyphae. After 16 h cultivation the hyphae had whole nuclei with NsdD-GFP fluorescence. Missense mutations in NLS inhibited the NsdD from entering nuclei indicating that the NLS play a critical role in nuclear localization of NsdD. The mutation could not complement the
  <em>
   nsdD
  </em>
  deletion mutation and also could not repress the
  <em>
   nrsA
  </em>
  transcription. The mutant NsdD that had shifted amino acid from cysteine to phenylalanine in a Zn finger entered nuclei normally suggesting that Zn finger did not affect the localization of NsdD. The IndB and IndD known to bind to zinc finger of NsdD and control sexual development negatively also inhibited NsdD from entering nuclei when they were over-expressed. Zn finger mutation to which Inds could not bind did not affect the entrance of NsdD to nuclei. The result implies that the binding of IndB and IndD to NsdD zinc finger inactivates the NLS function, which eventually results in the failure of decision of sexual development. In
  <em>
   veA
  </em>
  deletion mutant NsdD was detected in whole hyphae raising the possibility of the involvement VeA in NsdD nuclear localization or NsdD degradation in cytoplasm. Under the stress conditions where sexual development was inhibited most NsdD-GFP were found in cytoplasm, which strongly suggested that some unknown components in response to the stresses inhibit the nuclear localization of NsdD resulting in the decision of asexual development. Conclusively, the nuclear localization of NsdD is a very important process in the decision of sexual development at late vegetative stage, which is under the control of various genetic and environmental factors.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410690
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Conidiation    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Insights into the nuclear transport and function of the transcription factor FlbB.
 <br/>
 <b>
  Elixabet Oiartzabal
 </b>
 <sup>
  1
 </sup>
 , Aitor Garzia
 <sup>
  2
 </sup>
 , Eduardo A. Espeso
 <sup>
  3
 </sup>
 , Unai Ugalde
 <sup>
  1
 </sup>
 , Oier Etxebeste
 <sup>
  1
 </sup>
 . 1) Dept. of Applied Chemistry, Faculty of Chemistry, University of The Basque Country. Manuel de Lardizabal 3, 20018, San Sebastian; 2) Howard Hughes Medical Institute, Laboratory of RNA Molecular Biology, Rockefeller University, 1230 York Avenue, New York, NY 10065; 3) Dept. of Cellular and Molecular Biology, Centro de Investigaciones Biológicas (CSIC), Ramiro de Maeztu, 9, 28040, Madrid, Spain
 <br/>
 <p>
  The UDA (Upstream Developmental Activator) pathway of
  <em>
   Aspergillus nidulans
  </em>
  transduces environmental signals and activates asexual development. FlbB is a key UDA transcription factor (TF) which is accumulated at the most apical nucleus and the tip of vegetative hyphae, through a mechanism that requires the activity of additional UDAs. Tip accumulation of FlbB is mediated by the adapter UDA FlbE. The present work has focused on the retrograde transport of FlbB from the tip to the nucleus, and its transcriptional activity there. Nuclear import of FlbB is mediated by a bipartite nuclear localization signal located at the N-terminus. It has been previously reported that, once in the nucleus, FlbB activates the expression of
  <em>
   flbD
  </em>
  (another UDA factor coding gene), and then both factors are jointly required for the induction of the first specific asexual development regulator,
  <em>
   brlA
  </em>
  . We have now found that FlbD activity is essential for the nuclear accumulation of FlbB. FlbD and FlbB interact, and the deletion of
  <em>
   flbD
  </em>
  causes a significant reduction in the nuclear accumulation of FlbB and an increase in its apical fluorescent intensity. Bi-directional, cytoplasmic and microtubule-dependent movement of FlbB::GFP can be easily followed in a
  <em>
   &#916;flbD
  </em>
  background. RNA sequencing analyses comparing &#916;
  <em>
   flbB
  </em>
  and wild-type transcriptomes show that FlbB has additional roles, such as the suppression of the
  <em>
   dba
  </em>
  metabolic cluster, responsible for the synthesis of DHMBA, an antibacterial metabolite. In addition, our study also identifies FlbB-mediated regulation of the putative Helix-Loop-Helix type TF UrdA, an inducer of asexual and repressor of sexual development. The results presented widen the scope on the transcriptional roles of FlbB and the mechanisms that allow the transduction of environmental signals from the tip to the nucleus in
  <em>
   A. nidulans
  </em>
  .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410547
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Secondary metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 The MFS transporter gene in GT biosynthetic cluster is necessary for GT secretion and protection.
 <br/>
 <b>
  Elizabeth Smith
 </b>
 , Stephen Hammel, Sean Doyle, Grainne O' Keeffe, Gary Jones. Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland
 <br/>
 <p>
  Gliotoxin is a non ribosomally synthesized metabolite, secreted
  <em>
   by
  </em>
  <em>
   Aspergillus fumigatus.
  </em>
  It is an epipolythiodioxopiperazine (ETP) which is characterised by an intramolecular disulphide bridge. The
  <em>
   gli
  </em>
  gene cluster is comprised of thirteen genes which are involved in aspects of gliotoxin biosynthesis and self-protection. The cluster includes
  <em>
   gliA
  </em>
  (CADRE locus identifier: AFUA_ 6G09710), a gene which is predicted to encode a transmembrane gliotoxin efflux pump, which is a member of the Major Facilitator Superfamily (MFS).
  <em>
   A. fumigatus
  </em>
  also secretes an inactive bis-S-methylated form of gliotoxin (BmGT). The role of
  <em>
   gliA
  </em>
  in the biosynthesis and secretion of gliotoxin by
  <em>
   A. fumigatus
  </em>
  is unknown, however a previous study in which the
  <em>
   sirA
  </em>
  gene, an ortholog of
  <em>
   gliA,
  </em>
  from
  <em>
   Leptosphaeria maculans
  </em>
  was deleted, resulted in increased sensitivity to both gliotoxin and sirodesmin, and also an increase the secretion of sirodesmin from
  <em>
   L. maculans
  </em>
  . Deletion of
  <em>
   gliA
  </em>
  was undertaken in
  <em>
   A. fumigatus
  </em>
  ATCC26933, previously shown to produce gliotoxin at high levels, using a split marker strategy and acquisition of pyrithiamine resistance. Both Southern and qRT-PCR analysis confirmed deletion of
  <em>
   gliA
  </em>
  and absence of
  <em>
   gliA
  </em>
  expression in
  <em>
   A. fumigatus
  </em>
  &#916;
  <em>
   gliA
  </em>
  <sup>
   26933
  </sup>
  , respectively. Deletion of
  <em>
   gliA
  </em>
  completely abolished gliotoxin secretion, as determined by both RP-HPLC and LC-MS analysis, compared to that from
  <em>
   A. fumigatus
  </em>
  ATCC26933. Interestingly, secretion of the gliotoxin derivative, BmGT was not inhibited, indeed, there was a significant increase in the levels of BmGT secreted by
  <em>
   A. fumigatus
  </em>
  &#916;
  <em>
   gliA
  </em>
  <sup>
   26933
  </sup>
  compared to wild type between 48-96 h growth (
  <em>
   p
  </em>
  &lt; 0.001). Exposure of
  <em>
   A. fumigatus
  </em>
  &#916;
  <em>
   gliA
  </em>
  <sup>
   26933
  </sup>
  to exogenous gliotoxin revealed a significantly (
  <em>
   p
  </em>
  &lt; 0.001) sensitive phenotype, compared to wild type. These results strongly suggest a role for
  <em>
   gliA
  </em>
  in the secretion of endogenously produced gliotoxin, but not bis-methyl gliotoxin, and that
  <em>
   gliA
  </em>
  functionality is necessary to protect against exogenous gliotoxin.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410203
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Metabolism    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Lactose transport in
 <em>
  Aspergillus nidulans
 </em>
 : Identification and expression of a second permease gene.
 <br/>
 <b>
  Erzsébet Fekete
 </b>
 , Anita Orosz, László Kulcsár, Michel Flipphi, Levente Karaffa. Biochemical Engineering, University of Debrecen, Debrecen, Hungary
 <br/>
 <p>
  Lactose (milk sugar) is the main carbohydrate in whey, an abundant high-energy dairy residue. Yet, for most micro-organisms that can convert it (into glucose and galactose), it is a gratuitous carbon source that is slowly assimilated. To optimalise fermentation processes that use whey residue, and to further its use in second-generation biofuel generation and its removal from contaminated soil and water (bioremediation), we study lactose catabolism in the industrial cell factory
  <em>
   Penicillium chrysogenum
  </em>
  and the genetic model
  <em>
   Aspergillus nidulans
  </em>
  , a soil-borne saprophyte. Recently, we characterised the lactose permease LacpA, responsible for a considerable part of the prevalent uptake in
  <em>
   A. nidulans
  </em>
  , and showed that uptake rather than hydrolysis is the limiting step in its lactose catabolism (1). We now have identified a second physiologically relevant lactose transporter, LacpB. Glycerol-grown mycelia from mutants deleted for both
  <em>
   lacpA
  </em>
  and
  <em>
   lacpB
  </em>
  only appear to take up minute amounts of lactose. Moreover, mycelia of the double deletant strains appear unable to produce new biomass from lactose. Although transcription of both
  <em>
   lacp
  </em>
  genes was strongly induced by lactose in pregrown wild-type mycelia, their inducer profiles differ markedly.
  <em>
   lacpB
  </em>
  responded also strongly to
  <em>
   beta
  </em>
  -linked glucopyranose dimers, cellobiose and sophorose, while these inducers of the cellulolytic system did not provoke any
  <em>
   lacpA
  </em>
  response. Nevertheless,
  <em>
   lacpA
  </em>
  /
  <em>
   B
  </em>
  double mutants grew like wild type on cellobiose which suggests that, in contrast to the situation with lactose, efficient cellobiose uptake in
  <em>
   A. nidulans
  </em>
  is mediated by third systems that make
  <em>
   lacpB
  </em>
  functionally redundant. We shall also report on currently ongoing studies with single
  <em>
   lacpB
  </em>
  deletion mutants and on the interplay of the inductory circuits that allow uptake and assimilation of lactose.
 </p>
 <p>
  (1) Fekete
  <em>
   et al
  </em>
  ., Fungal Genet Biol 49: 215-225.
 </p>
 <p>
  Acknowledgement: This research was supported by the EU and co-financed by the European Social Fund under the project ENVIKUT (TAMOP-4.2.2.A-11/1/KONV-2012-0043), and by the Hungarian Scientific Research Fund (OTKA Grant K1006600).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410666
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Secondary metabolism    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 A copper-responsive secondary metabolite pathway in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Fang Yun Lim
 </b>
 <sup>
  1
 </sup>
 , Joshua A. Baccile
 <sup>
  2
 </sup>
 , Frank C. Schroeder
 <sup>
  2
 </sup>
 , Nancy P. Keller
 <sup>
  1
 </sup>
 . 1) Medical Microbiology and Immunology, University of Wisconsin-Madison, Madison, WI.; 2) Boyce Thompson Institute Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY
 <br/>
 <p>
  Copper is a redox-active transition metal indispensible for life. It serves as a cofactor to enzymes involved in various fundamental cellular processes (e.g. respiration, reductive iron uptake, superoxide detoxification, etc.). In pathogenic fungi, copper is required for the function of virulence modulators such as multicopper laccases and superoxide dismutases. Here we report on the identification of a
  <u>
   c
  </u>
  opper-
  <u>
   r
  </u>
  esponsive secondary
  <u>
   m
  </u>
  etabolite cluster (
  <em>
   crm
  </em>
  ) in the human opportunistic pathogen
  <em>
   Aspergillus fumigatus
  </em>
  . Bioinformatic prediction coupled with expression analyses suggests that the current
  <em>
   crm
  </em>
  cluster spans a minimal of five enzymatic genes. Preliminary results show that expression of the
  <em>
   crm
  </em>
  cluster is enhanced during vegetative growth and under copper-deficient condition. Furthermore, our preliminary observations suggest a potential crosstalk between the
  <em>
   crm
  </em>
  cluster and the recently identified iron-responsive
  <em>
   has
  </em>
  cluster in this species. Future studies will elucidate the potential role of the
  <em>
   crm
  </em>
  pathway product(s) in copper-uptake, the nature of this crosstalk between two copper- and iron-responsive secondary metabolite clusters, and potential role in the biology and pathogenicity of this fungus.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410099
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Gene Expression    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Choose On or Off: A regulatable gene expression system for filamentous fungi.
 <br/>
 <b>
  Franziska Wanka
 </b>
 , Vera Meyer. Institute of Biotechnology, Department Applied and Molecular Microbiology, Berlin University of Technology, Gustav-Meyer-Allee 25, 13355 Berlin, Germany
 <br/>
 <p>
  The inducible tetracycline-dependent expression system is a versatile tool to control and fine-tune gene expression in eukaryotic cells in a metabolism-independent manner. By the addition of doxycycline, genes can either be switched on (Tet-on system) or switched off (Tet-off system). Recently, the Tet-on system has successfully been established for
  <em>
   Aspergillus niger
  </em>
  (Meyer et al., 2011) and proven to be useful for high-level production of secondary metabolites (Richter &amp; Wanka et al., 2014).
 </p>
 <p>
  Here, we present an optimum Tet-off system for use in
  <em>
   A. niger.
  </em>
  Both, amino acid sequence and medium-level expression of the Tet off repressor were shown to be crucial for efficient down-regulation of a gene of interest as well as for keeping the system stable in the genome of
  <em>
   A. niger
  </em>
  . We furthermore show that a gene of interest can be down-regulated in a Dox-dependent manner. This opens the possibility to establish and analyse gene-knockdown and gene-knockout mutants in
  <em>
   A. niger
  </em>
  .
 </p>
 <p>
  Meyer et al. (2011) Appl Environ Microbiol 77:2975-83
 </p>
 <p>
  Richter &amp; Wanka et al. (2014). Fungal Biology and Biotechnology 1:4  .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410222
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolism    Aspergillus fumigatus    Toxin
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Systems effects of gliotoxin
 <em>
  bis
 </em>
 -thiomethylation in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 Rebecca A. Owens, Stephen K. Dolan, Grainne O'Keeffe, Elizabeth B. Smith, Stephen Hammel, Kevin J. Sheridan, David A. Fitzpatrick, Sean Doyle,
 <b>
  Gary W. Jones
 </b>
 . Department of Biology, Maynooth University, Maynooth, County Kildare, Ireland
 <br/>
 <p>
  Gliotoxin (GT) is a nonribosomally synthesized redox active metabolite secreted by several fungal species and contributes to the virulence of the human fungal pathogen
  <em>
   Aspergillus fumigatus
  </em>
  . The biosynthetic, regulatory and self-protection genes associated with GT metabolism are grouped in a 13 gene cluster. GT is an epipolythiodioxopiperazine (ETP) class fungal toxin that contains a disulfide bridge, which plays a key role in determining the deleterious effects of this toxin. Modification of this rare structural motif by reduction and
  <em>
   S
  </em>
  -methylation significantly depletes the bioactivity of this metabolite. Bisdethiobis(methylthio)gliotoxin (BmGT) is the most well characterized of such GT derivatives. Recent work from our group identified an enzyme, gliotoxin
  <em>
   bis
  </em>
  -thiomethyltransferase (GtmA), which is responsible for converting dithiol-GT to BmGT. The primary role of BmGT production in
  <em>
   A. fumigatus
  </em>
  appears to be related to regulating the GT biosynthetic gene cluster rather than working in conjunction with the GT oxidoreductase, GliT, in a GT self-protection mechanism. GliA, which is a member of the major facilitator superfamily of membrane proteins, appears to be a GT-specific membrane efflux pump and BmGT production allows
  <em>
   A. fumigatus
  </em>
  to efflux this molecule in a GliA-independent manner. Given that the main intracellular methyl donor source is
  <em>
   S
  </em>
  -adenosylmethionine (SAM), which is generated via the methyl/methionine cycle and involves
  <em>
   S
  </em>
  -adenosylhomocysteine (SAH) production, the generation of BmGT provides a direct link between GT biosynthesis and primary metabolic processes. In support of this hypothesis we have identified and characterized changes that occur in SAM and SAH homeostasis within selected
  <em>
   gli
  </em>
  gene deletions, which also uncovers new findings that illuminate additional systems interactions which have evolved in gliotoxin-producing, compared to gliotoxin-naïve, fungi to facilitate its cellular presence. Our results suggest that proteins involved in gliotoxin self-protection, regulation and efflux work in concert to maintain homeostasis of important cellular metabolites.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410122
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Biomass    Proteomics    Mass Spectrometry
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Colocalization of wood modifications and secretome composition during the colonization of wood by
 <em>
  Postia placenta
 </em>
 and
 <em>
  Gloeophyllum trabeum
 </em>
 .
 <br/>
 <b>
  Gerald Presley
 </b>
 , Jiwei Zhang, Jonathan Schilling. Bioproducts and Biosystems Engineering, University of Minnesota, 2004 Folwell St. Paul, MN 55108.
 <br/>
 <p>
  Wood decay fungi drive the mineralization of the largest biotic sink of carbon on earth, but the biochemistry behind this process is not fully understood. Brown rot fungi are theorized to generate an extracellular fenton system to generate hydroxyl radicals that degrade wood polysaccharides. They also secrete hydrolytic enzymes which may be spatially separated from the fenton system.  Attempts to recreate brown rot
  <em>
   in vitro
  </em>
  have not been successful and thus the mechanistic paradigm is incomplete. To address this we have developed a wood wafer decay system that utilizes a spatial gradient along an advancing hyphal front to co-localize wood modifications and secretome variations along a fine-scale time series of decay stages. Two model brown rot fungi,
  <em>
   Gloeophyllum trabeum
  </em>
  and
  <em>
   Postia placenta
  </em>
  were grown up spruce wafers and total extracellular protein was extracted from 5 mm segments along the advancing hyphal front. Screening for hydrolytic enzyme activity showed that endo-acting hydrolase (cellulase, mannanase, and xylanase) specific activity was only detectable 5 mm or farther behind the advancing hyphal front and it increased thereafter reaching a plateau in the most decayed portions of the wafer. Identification of proteins from the same extracts with LC-MS showed differential expression of proteins along the advancing hyphal front. In early decay stages, less than 5 mm behind the hyphal front, peptides from low molecular weight proteins of unknown function along with some hemicellulose-degrading glycosyl hydrolases were by far the most abundant. In later stages of decay, endo-acting glycosyl hydrolases including the major known endocellulases from
  <em>
   P. placenta
  </em>
  and
  <em>
   G. trabeum
  </em>
  were the most commonly identified proteins. The colonization and decay of wood is a dynamic process as shown by the changes in the secretome from early to late decay stages. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410675
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Membrane    Ustilago maydis    Transport
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Early endosomes organize the fungal cell.
 <br/>
 <b>
  Gero Steinberg
 </b>
 . Sch Biosci, Univ Exeter, Exeter, United Kingdom
 <br/>
 <p>
  Fungal early endosomes are a highly motile compartment. The individual organelles travel over long distances along microtubules and extensive work in the basidomycete
  <em>
   Ustilago maydis
  </em>
  and the ascomycete
  <em>
   Aspergillus nidulans
  </em>
  has demonstrated that the molecular machinery, underlying this process is conserved amongst the filamentous fungi (overview in Steinberg,
  <em>
   Curr Opin Microbiol
  </em>
  , 2014, 20:55). However, the biological reason for early endosome motility is largely unknown. Recent papers have revealed unexpected roles in RNA and polysome motility and a have shown a pivotal role in triggering effector production during plant infection. In this talk, I will review briefly these data and present novel roles for moving endosomes in organizing the fungal cell. Taken together, it emerges that early endosomes are of key importance for fungal growth and pathogenicity.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410527
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    RNA-seq    Secondary metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Attenuation of gliotoxin self-protection in
 <em>
  A. fumigatus
 </em>
 reveals global effects on the cell, in particular on secondary metabolism.
 <br/>
 <b>
  Grainne O' Keeffe
 </b>
 <sup>
  1
 </sup>
 , Stephen Hammel
 <sup>
  1
 </sup>
 , Rebecca Owens
 <sup>
  1
 </sup>
 , Thomas Keane
 <sup>
  2
 </sup>
 , David Fitzpatrick
 <sup>
  1
 </sup>
 , Gary Jones
 <sup>
  1
 </sup>
 , Sean Doyle
 <sup>
  1
 </sup>
 . 1) Department of Biology, Maynooth University, Co. Kildare, Ireland; 2) The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, United Kingdom
 <br/>
 <p>
  <em>
   Aspergillus fumigatus
  </em>
  produces a number of secondary metabolites, one of which, gliotoxin, has been shown to exhibit anti-fungal activity and consequently an endogenous self-protection system is required. One of the genes in the gliotoxin biosynthetic gene cluster in
  <em>
   A. fumigatus
  </em>
  ,
  <em>
   gliT
  </em>
  , is required for self-protection against the toxin- however the global self-protection mechanism deployed is unclear. RNA-seq identified genes differentially regulated upon exposure to gliotoxin in
  <em>
   A. fumigatus
  </em>
  wild-type and
  <em>
   A. fumigatus &#916;gliT
  </em>
  , a gliotoxin hypersensitive strain. Expression of 164 genes was differentially regulated (log
  <sub>
   2
  </sub>
  fold change of 1.5) in
  <em>
   A. fumigatus
  </em>
  wild-type when exposed to gliotoxin, consisting of 101 genes with up-regulated expression and 63 genes with down-regulated expression. Interestingly, a much larger number of genes, 1700, were differentially regulated (log
  <sub>
   2
  </sub>
  fold change of 1.5) in
  <em>
   A. fumigatus &#916;gliT
  </em>
  when challenged with gliotoxin. These consisted of 508 genes with up-regulated expression, and 1192 genes with down-regulated expression. FunCat classification of differentially regulated genes revealed an enrichment of genes involved in both primary metabolic functions and secondary metabolism. Specifically, genes involved in gliotoxin biosynthesis, helvolic acid biosynthesis, siderophore-iron transport genes and also ribosome biogenesis genes underwent altered expression. It was confirmed that gliotoxin biosynthesis is induced upon exposure to exogenous gliotoxin, production of unrelated secondary metabolites is attenuated in
  <em>
   A. fumigatus &#916;gliT
  </em>
  , while quantitative proteomic analysis confirmed disrupted translation in
  <em>
   A. fumigatus &#916;gliT
  </em>
  challenged with exogenous gliotoxin. Our data highlight the global and extensive affects of exogenous gliotoxin on a sensitive strain devoid of a self-protection mechanism and infer that GliT functionality is required for the optimal biosynthesis of selected secondary metabolites in
  <em>
   A. fumigatus
  </em>
  .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410018
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Gene Expression    Development
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 PpoA coordinates fungal development through phialide formation and regulation of secondary metabolite clusters in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Gregory Fischer
 </b>
 <sup>
  1
 </sup>
 , Erwin Berthier
 <sup>
  2
 </sup>
 , Chun-Jun Guo
 <sup>
  3
 </sup>
 , Jonathan Palmer
 <sup>
  2
 </sup>
 , Clay C.C. Wang
 <sup>
  3
 </sup>
 , Nancy Keller
 <sup>
  2,1
 </sup>
 . 1) Department of Genetics, University of Wisconsin-Madison, Madison, WI.; 2) Department of Medical Microbiology and Immunology, University of Wisconsin-Madison, Madison, WI.; 3) Pharmacology and Pharmaceutical Sciences and Chemistry, University of Southern California, Los Angeles, CA
 <br/>
 <p>
  The opportunistic pathogen
  <em>
   Aspergillus fumigatus
  </em>
  contains three cyclooxygenase (COX)-like enzymes termed PpoA, PpoB, and PpoC (
  <u>
   <strong>
    P
   </strong>
  </u>
  si-factor
  <u>
   <strong>
    p
   </strong>
  </u>
  roducing
  <u>
   <strong>
    o
   </strong>
  </u>
  xygenase).  Ppos produce bioactive oxygenated lipid signaling molecules (oxylipins) important for fungal development and similar in structure to mammalian prostaglandins. Here, we identified that disruption of PpoA accelerates asexual development, shifts secondary metabolite production, and significantly disrupts phialide development.  PpoA loss and overexpression (OE) increased and delayed spore development, respectively. Northern blot and biochemical analysis of wild-type,
  <em>
   &#916;ppoA,
  </em>
  and
  <em>
   OE::ppoA
  </em>
  strains indicate a major shift in secondary metabolism including production of several spore specific metabolites such as DHN melanin and endocrocin.  Furthermore, deletion of PpoA resulted in long, filamentous phialides protruding from the vesicle of the conidiophore. The abnormal phialide phenotype could be rescued by co-culture of the
  <em>
   &#916;ppoA
  </em>
  strain with the
  <em>
   OE::ppoA
  </em>
  strain.  This work demonstrates the critical role PpoA-derived oxylipins play in
  <em>
   A. fumigatus
  </em>
  development and future work will attempt to identify the mechanism by which PpoA affects phialide formation.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410093
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Protein Localization    Morphogenesis
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 The role of septin AspD in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  I. Dörter
 </b>
 , M. Momany. Plant Biology Dept, UGA, Athens, GA
 <br/>
 <p>
  Septins are evolutionarily conserved GTP-binding proteins from the GTPase superfamily that form filaments, which play roles as important and diverse as those of actin and microtubules. These new and increasingly characterized cytoskeletal components act as diffusion barriers and coordinate cytokinesis and nuclear division.
 </p>
 <p>
  Septins are classified into five orthologous groups, and monomers from different groups associate to form nonpolar heteropolymeric rods, which in turn assemble into higher-order structures including rings and filaments that can be visualized by fluorescent microscopy of GFP-tagged septins. The mechanisms driving septin heteropolymer and higher-order structure assembly are only beginning to be understood.
 </p>
 <p>
  The filamentous fungus
  <em>
   Aspergillus nidulans
  </em>
  has one septin from each phylogenetic group. Four of the
  <em>
   A. nidulans
  </em>
  septins are orthologs of the core septins in
  <em>
   S. cerevisiae
  </em>
  and the fifth septin, AspE, is lacking in unicellular yeasts and appears to be ancestral. Our results show that at least two distinct septin heteropolymer populations co-exist in
  <em>
   A. nidulans
  </em>
  .
 </p>
 <p>
  According to our data, the septin AspD, which is homologous to
  <em>
   Saccharomyces cerevisiae
  </em>
  Cdc10p, is only present in one of the heteropolymers. In contrast to the other three core septins, deletion of AspD did not show emergence of extra germ tubes and branches in early development, but
  <em>
   &#916;
  </em>
  <em>
   aspD
  </em>
  deletion strains revealed abnormal nuclear morphology. Further,
  <em>
   &#916;
  </em>
  <em>
   aspD
  </em>
  colonies showed an increased frequency of sectoring relative to wildtype. Fluorescence microscopy and time-lapse analyses revealed AspD-GFP tagged septin rods contact nuclei and the cell cortex. Thus our results suggest that AspD might play a role in coupling nuclear division to cytokinesis. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410331
 </b>
 <br/>
 Research Area:
 <b>
  Pathogenic and Mutalistic Interactions
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Microarray    Transcriptome
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 <em>
  Bacillus subtilis
 </em>
 attachment to
 <em>
  Aspergillus niger
 </em>
 hyphae results in mutually altered metabolism.
 <br/>
 <b>
  Isabelle Benoit
 </b>
 <sup>
  1,2,3,4
 </sup>
 , Marielle van den Esker
 <sup>
  5
 </sup>
 , Aleksandrina Patyshakuliyeva
 <sup>
  1,2,3
 </sup>
 , Derek Mattern
 <sup>
  6
 </sup>
 , Felix Blei
 <sup>
  8
 </sup>
 , Miaomiao Zhou
 <sup>
  1,3
 </sup>
 , Jan Dijksterhuis
 <sup>
  1
 </sup>
 , Axel Brakhage
 <sup>
  6,7
 </sup>
 , Oscar Kuipers
 <sup>
  5
 </sup>
 , Ronald de Vries
 <sup>
  1,2,3,4
 </sup>
 , Ákos Kovács
 <sup>
  5,8
 </sup>
 . 1) Fungal Physiology, CBS-KNAW, Utrecht, Utrecht, Netherlands; 2) Microbiology, Utrecht University, Utrecht, The Netherlands; 3) Fungal Molecular Physiology, Utrecht University, Utrecht, The Netherlands; 4) Kluyver Centre for Genomics of Industrial Fermentations, Netherlands Genomics Initiative/Netherlands Organization for Scientific Research, Netherlands; 5) Molecular Genetics, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Groningen, The Netherlands; 6) Leibniz Institute for Natural Product Research and Infection Biology, HKI, Jena, Germany; 7) Department of Microbiology and Molecular Biology, Institute of Microbiology, Friedrich Schiller University Jena, Jena, Germany; 8) Terrestrial Biofilms Group, Institute of Microbiology, Friedrich Schiller University Jena, Jena, Germany
 <br/>
 <p>
  Interaction between microbes affects the growth, metabolism and differentiation of members of the microbial community. While direct and indirect competition, like antagonism and nutrient consumption have a negative effect on the interacting members of the population, microbes have  also evolved in nature not only to fight, but in some cases to adapt to or support each other, while  increasing the fitness of the community.
  <em>
   Bacillus subtilis
  </em>
  , when grown in the presence of
  <em>
   Aspergillus niger
  </em>
  interacts with the fungus by attaching and growing on the hyphae. Based on data obtained in a dual transcriptome experiment, we suggest that both fungi and bacteria alter their metabolism during this interaction. Interestingly, the transcription of genes related to the antifungal and putative antibacterial defense mechanism of
  <em>
   B. subtilis
  </em>
  and
  <em>
   A. niger
  </em>
  , respectively, are decreased upon attachment of bacteria to the mycelia. Analysis of the culture supernatantsuggests that surfactin production by
  <em>
   B. subtilis
  </em>
  was reduced when the bacterium was co- cultivated with the fungus. Our experiments provide new insights into the interaction between a bacterium and a fungus.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410514
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolism    Aspergillus oryzae    Annotation
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Novel Comparative Genomics Approach for Motif-Independent Prediction and Similarity Analysis of a Secondary Metabolism Gene Cluster.
 <br/>
 <b>
  Itaru Takeda
 </b>
 <sup>
  1,2
 </sup>
 , Myco Umemura
 <sup>
  2
 </sup>
 , Hideaki Koike
 <sup>
  2
 </sup>
 , Kiyoshi Asai
 <sup>
  3,4
 </sup>
 , Masayuki Machida
 <sup>
  1,2
 </sup>
 . 1) Biotechnology and Life Science, Tokyo University of Agriculture and Technology, Tokyo, Japan; 2) Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Tsukuba/Sapporo, Japan; 3) Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Chiba, Japan; 4) Computational Biology Research Center (CBRC), AIST, Tokyo, Japan
 <br/>
 <p>
  Despite biological importance, many genes for secondary metabolite biosynthesis (SMB) remain unspecified. It is due largely to their high sequence diversity and silence in expression under a general cultivation conditions. Some software tools including SMURF and antiSMASH have been developed to predict fungal SMB gene clusters by detecting core genes encoding non-ribosomal peptide synthetase, polyketide synthase and so on as well as several typical accessory genes, those encoding transcription factors and transporters, for example,. In this study, we have devised a novel prediction method of SMB gene clusters (MIPS-CG) by a comparative genomics approach, allowing motif-independent prediction. The gene cluster pairs were first detected based on similarity of the order of genes with high sequence similarity, followed by adjustment of cluster margins and exclusion of clusters in syntenic regions. By varying several parameters in the procedure above, we have successfully realized high detection sensitivity as well as accurate prediction of cluster margins. Our method allowed detection of 21 out of 24 known SMB gene clusters in the genome sequences of 10 filamentous fungi including the kojic acid biosynthesis gene cluster of
  <em>
   Aspergillus oryzae
  </em>
  . The result indicated that a pair of gene clusters was detected largely by similar gene contents rather than order of the genes. This means that our method is applicable to the detection of gene cluster pairs even having significant rearrangement possibly occurred during evolution from a common ancestral gene cluster. Furthermore, a significant difference in the sequence characteristics was found between the genes residing inside the clusters and those outside the clusters by a comparison between sets of the genes inside and outside the clusters.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410746
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Secondary metabolism    Diversity
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Promiscuity runs in the family - Analysis of nidulanins.
 <br/>
 <b>
  Jakob B. Nielsen
 </b>
 , Andreas Klitgaard, Maria L. Nielsen, Thomas O. Larsen, Mikael R. Andersen, Kristian F. Nielsen, Uffe H. Mortensen. DTU Systems Biology, Technical University of Denmark, 2800 Kgs. Lyngby, Denmark
 <br/>
 <p>
  Non-ribosomal peptides (NRPs) constitute a considerable group of secondary metabolites. These products are synthesized by large modular enzymes, the non-ribosomal peptide synthetases (NRPS). The selection of the amino acids incorporated in the NRP is decided by the adenylation domain(s) in the NRPS. The genome of
  <em>
   Aspergillus nidulans
  </em>
  encodes 14 putative NRPSs, and nearly as many pseudo NRPSs, NRPS-like enzymes, where some transfer a single amino acid to another moiety. Through gene expression analysis and deletions, we recently linked production of the prenylated cyclo-tetrapeptide nidulanin A, Phe-Kyn-Val-Val, to the activity of the NRPS AN1242 and the prenyl transferase AN11080. Through feeding experiments by stable isotope labeled amino acids (SILAAs), we observed that this particular NRPS is highly promiscuous. At least the two first adenylation domains allow incorporation of more than one type of amino acid to yield at least 8 other tetrapeptides including fungisporin, not previously described from
  <em>
   A. nidulans
  </em>
  . Strikingly, analysis of synteny for the locus of AN1242 in many of the sequenced
  <em>
   Aspergillus
  </em>
  and
  <em>
   Penicillium
  </em>
  species revealed extensive homology, among others
  <em>
   pes1
  </em>
  from
  <em>
   Aspergillus fumigatus
  </em>
  and
  <em>
   hcpA
  </em>
  of
  <em>
   Penicillium chrysogenum
  </em>
  . This was further verified by UHPLC-qTOFMS that detected fungisporins in all of 20
  <em>
   Penicillium
  </em>
  species as well as 15
  <em>
   Aspergillus
  </em>
  species analyzed. This cocktail of tetrapeptides synthetized from one NRPS and the presence of the gene cluster in so many species is intriguing and points to a central biological function of nidulanins.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410802
 </b>
 <br/>
 Research Area:
 <b>
  Other
 </b>
 <br/>
 Keyword(s):
 <b>
  Annotation    Other    Genome Sequencing (Other Keyword: Database)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 FungiDB: An integrated functional genomics database for fungi and oomycetes.
 <br/>
 Edward Liaw
 <sup>
  1
 </sup>
 , Dai Gu
 <sup>
  1
 </sup>
 , Venkatesh Moktali
 <sup>
  1,2
 </sup>
 , Brett M Tyler
 <sup>
  2
 </sup>
 , Sufen Hu
 <sup>
  3
 </sup>
 , Hamin Wang
 <sup>
  3
 </sup>
 , John Brestelli
 <sup>
  3
 </sup>
 , Debbie Pinney
 <sup>
  3
 </sup>
 , Omar Harb
 <sup>
  3
 </sup>
 , Brian Brunk
 <sup>
  3
 </sup>
 , Jessica C Kissinger
 <sup>
  4
 </sup>
 , David Roos
 <sup>
  3
 </sup>
 ,
 <b>
  Jason E Stajich
 </b>
 <sup>
  1
 </sup>
 . 1) Plant Pathology &amp; Microbiology, Univ California, Riverside, Riverside, CA.; 2) 2Department of Botany and Plant Pathology, Oregon State University, Corvallis, OR; 3) 3Penn Center for Bioinformatics and Department of Biology, University of Pennsylvania, Philadelphia, PA; 4) 4Department of Genetics, Institute of Bioinformatics and Center for Tropical &amp; Emerging Global Diseases, University of Georgia, Athens, GA
 <br/>
 <p>
  FungiDB (http://FungiDB.org) is a free online database that enables data mining and analyses of the pan-fungal and oomycete genomic sequences and functional data. This resource was developed in partnership with the Eukaryotic Pathogen Bioinformatics Resource Center (http://EuPathDB.org). Using the same infrastructure and user interface as EuPathDB, FungiDB allows for sophisticated and integrated searches to be performed over an intuitive graphical system. Release 3.2 of FungiDB contains sequence and annotation for over 70 species spanning the Ascomycota
  <em>
   ,
  </em>
  Basidiomycota
  <em>
   ,
  </em>
  zygomycete, and chytrid fungi; including pathogenic species from the
  <em>
   Cryptococcus, Histoplasma
  </em>
  , and
  <em>
   Coccidiodes
  </em>
  genera. 20 Oomycete genomes are also included in this release. In addition to the genomic sequence data and annotation, FungiDB includes transcriptomic data based on 24 RNA sequence and microarray experiments and all expressed sequence tag data from GenBank. All genomes in FungiDB are run through a standard analysis pipeline that generates additional data such as signal peptide and transmembrane domain predictions, GO term and EC number associations and orthology profiles.
 </p>
 <p>
  The graphical user interface in FungiDB allows users to conduct
  <em>
   in
  </em>
  silico experiments that leverage the available data and analyses. For example, a search in FungiDB can identify all genes in
  <em>
   Candida albicans
  </em>
  that do not have orthologs in mammals, have a predicted signal peptide, are annotated as a kinase and are expressed under conditions of high oxygen stress.  FungiDB is supported in part by the Burroughs Wellcome Fund, the Alfred P. Sloan Foundation, USDA NIFA and NIH HHSN272201400030C.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410322
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Development    Transcription factor
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The velvet family of fungal regulators contains a DNA-binding domain structurally similar to NF-&#954;B.
 <br/>
 <b>
  Jennifer Gerke
 </b>
 <sup>
  1
 </sup>
 , Yasar Luqman Ahmed
 <sup>
  2
 </sup>
 , Hee-Soo Park
 <sup>
  3
 </sup>
 , Özgür Bayram
 <sup>
  1
 </sup>
 , Piotr Neumann
 <sup>
  2
 </sup>
 , Min Ni
 <sup>
  3
 </sup>
 , Achim Dickmanns
 <sup>
  2
 </sup>
 , Sun Chang Kim
 <sup>
  4
 </sup>
 , Jae-Hyuk Yu
 <sup>
  3
 </sup>
 , Ralf Ficner
 <sup>
  2
 </sup>
 , Gerhard Braus
 <sup>
  1
 </sup>
 . 1) Molecular Microbiology and Genetics, Georg-August University, Göttingen, Germany; 2) Molecular Structural Biology, Georg-August University, Göttingen, Germany; 3) Bacteriology and Genetics, University of Wisconsin-Madison, Madison, Wisconsin, USA; 4) Biological Sciences, Korea Advanced Institute of Science and Technology, Dae-Jon, Republic of Korea
 <br/>
 <p>
  Fungi produce small bioactive molecules (secondary metabolites) for signaling and protection, whereas in animals the inflammation and immune system is responsible for self-defense. The regulation of secondary metabolism and the control of development in fungi are coordinated by a family of velvet-domain containing regulators, the velvet proteins. These are conserved across the fungal kingdom and share a homologous region of about 150 amino acids that lacks sequence homology to any other known protein.
  <em>
   In vivo
  </em>
  chromatin immuno-precipitation (ChIP) and
  <em>
   in vitro
  </em>
  electrophoretic mobility shift assays (EMSAs) showed that the velvet-domain of the
  <em>
   Aspergillus nidulans
  </em>
  protein VosA is a novel DNA-binding domain that specifically recognizes an 11-nucleotide consensus sequence in the promoter regions of key developmental regulatory genes. The crystal structure analysis of the VosA velvet-domain revealed an unforeseen structural similarity with the Rel homology domain (RHD) of the mammalian transcription factor NF-&#954;B. RHD-containing proteins control inflammation, the immune system and development in animals. We identified several conserved amino acid residues in the velvet-domain and showed that they are essential for the DNA-binding ability of VosA. Additionally, we demonstrate by crystal structure analyses of the VosA homodimer and the VosA-VelB heterodimer that the velvet-domain is also important for dimer formation. These findings indicate that the coordination of development and defense mechanisms in fungi and animals might be controlled by the structurally related RHD and velvet proteins, and that they might have a common functional origin. The different homo- and heterodimers of velvet proteins might modulate gene expression of developmental and defensive pathways similar to NF-&#954;B.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410328
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Evolution    Metabolism    Phylogenetic
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The Evolution of Fungal Metabolic Pathways.
 <br/>
 <b>
  Jennifer Wisecaver
 </b>
 <sup>
  1
 </sup>
 , Jason Slot
 <sup>
  1,2
 </sup>
 , Antonis Rokas
 <sup>
  1
 </sup>
 . 1) Biological Sciences, Vanderbilt University, Nashville, TN.; 2) Plant Pathology, The Ohio State University, Columbus, OH.
 <br/>
 <p>
  Fungi contain a remarkable range of metabolic pathways, sometimes encoded by gene clusters, enabling them to digest most organic matter and synthesize an array of potent small molecules. In fungal genomes, the genes involved in these metabolic pathways can be physically linked on chromosomes, forming gene clusters. This extraordinary metabolic diversity is integral to the variety of ecological strategies that fungi employ, but we still know little about the evolutionary processes involved in its generation. To address this question, we analyzed 247,202 enzyme-encoding genes participating in hundreds of metabolic reactions from 208 diverse fungal genomes to examine how two major sources of gene innovation, namely gene duplication and horizontal gene transfer, have contributed to the evolution of clustered and non-clustered metabolic pathways. We discovered that gene duplication is the dominant and consistent driver of metabolic innovation across fungal lineages and metabolic categories; in contrast, horizontal gene transfer appears highly variable both across organisms and functions. The effects of both gene duplication and horizontal gene transfer were more pronounced in clustered genes than in their non-clustered counterparts suggesting that metabolic gene clusters are hotspots for the generation of fungal metabolic diversity. Finally, as a case study, we investigated how both gene duplication and horizontal gene transfer have contributed to the evolution of the bikaverin gene cluster found in some species of
  <em>
   Fusarium
  </em>
  and
  <em>
   Botrytis..
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410417
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolism    Aspergillus niger    Diversity
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 A new view on Aspergillus chemotaxonomy.
 <br/>
 <b>
  Jens Frisvad
 </b>
 . CMB, Dept Systems Biol, Kgs. Lyngby, Denmark
 <br/>
 <p>
  After the nomenclatural changes (One Fungus One Name) accepted by the Botanical Congress in 2011, Aspergillus is preferred by most scientists for Aspergillus itself and all the former names used for the sexual state of these fungi and genera with asexual state that are included in Aspergillus sensu stricto according to DNA sequence data: Chaetosartorya, Cristaspora, Dichotomomyces, Emericella, Eurotium, Fennellia, Hemisartorya (?), Neocarpenteles, Neopetromyces, Neosartorya, Petromyces, Phialosimplex, Polypaecilum, Saitoa, and Sclerocleista. The Aspergillus genus and sections can be regarded as polythetic classes regarding morphology and exometabolites, i.e. most, but not all, species express a given feature, and each species would have those features in different combinations. Even though the sections are monophyletic, as judged by DNA sequences, the combination of features include symplesiomorphies, synapomorphies and on top of that autapomorphies, making a hennigian cladistic analysis of such phenotypic features irrelevant. In the closely related sections Flavi, Nigri and Circumdati, only one biosynthetic family of exometabolites, ochratoxins, is common. A detailed analysis of all the exometabolite biosynthetic families in the Aspergilli shows, however, that species in these sections have very similar kinds of exometabolites in common. For example di-diketopiperazines are produced by species in the three sections: Ditryptophenalins in Flavi, asperazines in Nigri and aspergamides in Circumdati. Such di-diketopiperazines are also produced in more distantly related species, for example eurocristatine in section Aspergillus (formerly Eurotium). Another example is the epidithiodioxopiperazines, which are represented by gliotoxin in Section Fumigati, by acetylaranotin in section Terrei, by emethallicins, emestrins and dithiosilvatins in Nidulantes and by aspirochlorin in Section Flavi. Sterigmatocystin is produced by species in the distantly related sections Flavi, Ochraceorosei, Nidulantes and Cremei. Thus Aspergillus is a genus in which the species have many common morphological and chemical features, despite some important differences in ecophysiology and sexual features.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410044
 </b>
 <br/>
 Research Area:
 <b>
  Population and Evolutionary Genetics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Evolution    Hyphal Growth
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 A regular unicellular stage facilitates adaptation: the emergence of azole resistance in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Jianhua Zhang
 </b>
 <sup>
  1
 </sup>
 , Alfons Debets
 <sup>
  1
 </sup>
 , Paul Verweij
 <sup>
  2
 </sup>
 , Willem Melchers
 <sup>
  2
 </sup>
 , Bas Zwaan
 <sup>
  1
 </sup>
 , Sijmen Schoustra
 <sup>
  1
 </sup>
 . 1) Laboratory of Genetics, Wageningen University, Wageningen, Netherlands; 2) Department of Medical Microbiology, Radboud University Medical Centre, 6500 HB Nijmegen, The Netherlands
 <br/>
 <p>
  Understanding the occurrence and spread of azole resistance in
  <em>
   Aspergillus fumigatus
  </em>
  is crucial for public health. It has been hypothesized that a unicellular stage via asexual sporulation is essential for phenotypic expression of azole resistance mutations in
  <em>
   A. fumigatus
  </em>
  facilitating subsequent spread through natural selection. We assessed the advantage of unicellular stage via asexual sporulation by growing the fungus under pressure of one of five different azole fungicides and monitoring the rate of adaptation between scenarios of, (i) exclusive mycelium growth (multicellular stage) without asexual sporulation, and, (ii) growth allowing asexual sporulation (unicellular stage). Our results unequivocally show that unicellular stage via asexual sporulation enhances adaptation over a fixed period of evolutionary time. This can be explained by the combination of more effective selection because of the transition from a multicellular to a unicellular stage, and by increased mutation supply due to the sporulation process, which involves numerous mitotic divisions. Our insights are essential to unravel the fungal adaptation strategies are highly relevant for resistance development in the natural environment, but also for patients with chronic aspergillus diseases. Uncovering the pathways to adaptation will help to improve our current management strategies both in agricultural and medical fields.
 </p>
 <p>
  <strong>
   Key words:
  </strong>
  <em>
   Aspergillus fumigatus
  </em>
  ; azole resistance; unicellular and multicellular; asexual sporulation; experimental evolution; MIC value; mycelial growth rate.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410231
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Whole Genome Sequencing    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Bulk segregant analysis followed by high-throughput sequencing identifies a novel gene required for acid production in Aspergillus niger.
 <br/>
 <b>
  Jing Niu
 </b>
 <sup>
  1
 </sup>
 , Peter J. Punt
 <sup>
  1,2
 </sup>
 , Arthur F. J. Ram
 <sup>
  1
 </sup>
 . 1) Leiden University, Institute of Biology Leiden, Molecular Microbiology and Biotechnology, Sylviusweg 72, 2333 BE Leiden, The Netherlands; 2) TNO Microbiology and Systems Biology, PO Box 360, 3700 AJ Zeist, The Netherlands
 <br/>
 <p>
  We have combined high-throughput sequencing (Illumina) with bulk segregant analysis to identify the mutation in a previously isolated non-acidifying mutant in Aspergillus niger.  Because of the lack of a sexual cycle for A. niger, the parasexual cycle was used to generate a pool of segregants. A set of defined color mutants with auxotrofic markers was constructed by targeted gene deletion to facilitate the construction of diploid strains in A. niger. In the bulk-segregant analysis approach, the mutant of interest is crossed to a wild-type strain and haploid segregants displaying the phenotype of interest are pooled and DNA from this pool of segregants is sequenced using a deep sequencing technology (e.g. Illumina). The mutation causing the non-acidifying phenotype was found to be recessive and since about 50% of the segregants (78 of the 140 segregants analysed) showed the non-acidifying phenotype, the phenotype was likely to be caused by a single mutation. In addition to sequencing the genomic DNA pool of segregants, the parental strains were also sequenced and single nucleotide polymorphisms (SNPs) between the strains were identified. SNPs between the parental strains not related with the phenotype, have a 50% chance to be present in the pool; SNPs responsible for the phenotype or closely linked to the mutation responsible for the phenotype, are conserved in the pool. In total, 52 SNPs were identified between the two parental strains and three SNPs were 100% conserved in the pool of segregants. All three SNPs mapped to the right arm of Chromosome II, indicating that this region contains the genetic locus affecting the phenotype related to acid production. It is currently determined which of the three SNP is responsible for the non-acidifying phenotype of the mutant by complementation and targeted deletion studies.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410042
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Gene Expression    Transcriptome    Single Nucleotide Polymorphism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Genome-wide analysis of the regulations of genes involved in carbon catabolism through expression Quantitative Trait Loci (eQTL) in
 <em>
  Coprinopsis cinerea
 </em>
 .
 <br/>
 <b>
  J. Chang
 </b>
 , C.H. Au, C.K. Cheng, H.S. Kwan. School of life sciences, The Chinese University of Hong Kong, Hong Kong
 <br/>
 <p>
  Lignocellulose is the most abundant natural resource and its conversion into renewable energy attracts many research interests. Understanding the regulation of carbohydrate-active enzymes is fundamental to the use of wood-decaying basidiomycetes in lignocellulose conversion. Our goal is to identify eQTLs of lignocellulolytic enzymes in
  <em>
   Coprinopsis cinerea
  </em>
  , of which the genome harbors high number of Auxiliary Activities enzymes.
 </p>
 <p>
  We sequenced
  <em>
   C. cinerea
  </em>
  reference strain Okayama 7#130 and its mapping partner #172 to develop a panel of SNP markers. Single spore isolates from crosses of the two strains were sequenced. The parental strains and the 46 single spore isolates were cultured on softwood-enriched sawdust to induce lignocellulolytic enzymes. RNAs from these cultures were sequenced. The RNA-seq results were aligned to the reference transcriptome using BWA. To assess the genetic contribution to expression variations among the 46 segregants, we mapped eQTL genome-widely using the linear regression model. SNPs were estimated for their effect size to explain the expression variances. Genes expressed in similar patterns with the lignocellulolytic enzyme genes across the 46 segregants were also clustered and analyzed.
 </p>
 <p>
  Three cis-eQTLs and 97 trans-eQTLs (
  <em>
   P
  </em>
  &lt; 5 × 10
  <sup>
   &#8722;5
  </sup>
  ) have been obtained. These eQTLs have provided us with a wealth of information that the expressions of genes turning off the Carbon Catabolite Repression (CCR) are correlated with SNP. We report one transcription factor as trans-eQTL hotspot, which may regulate the expression of laccase3 and laccase5.
 </p>
 <p>
  The eQTL approach has identified transcriptional regulation that may contribute to the CAZymes expression. The results will be practically important to the enzyme production, which will benefit the bioethanol production from lignocellulose.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410421
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Wheat    RNA-seq
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The role of carbon starvation in the induction of enzymes that degrade plant-derived carbohydrates in
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  Jolanda van Munster
 </b>
 <sup>
  1
 </sup>
 , Paul Daly
 <sup>
  1
 </sup>
 , Stephane Delmas
 <sup>
  1
 </sup>
 , Steven Pullan
 <sup>
  1
 </sup>
 , Martin Blythe
 <sup>
  2
 </sup>
 , Sunir Malla
 <sup>
  2
 </sup>
 , Matthew Kokolski
 <sup>
  1
 </sup>
 , Xiaolan Yu
 <sup>
  3
 </sup>
 , Paul Dupree
 <sup>
  3
 </sup>
 , David Archer
 <sup>
  1
 </sup>
 . 1) School of Life Sciences, University of Nottingham, Nottingham, United Kingdom; 2) Deep Seq, Faculty of Medicine and Health Sciences, Queen's Medical Centre, University of Nottingham, Nottingham, United Kingdom; 3) Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
 <br/>
 <p>
  The saccharification of lignocellulosic biomass for the production of second generation biofuels requires cheaper and more effective enzyme mixtures. Fungi are an important source of such enzymes, but the understanding of the regulation and induction of the encoding genes is still incomplete. To explore the induction mechanism, we analysed the response of the industrially important fungus
  <em>
   Aspergillus niger
  </em>
  to wheat straw, with a focus on events occurring shortly after exposure to the substrate. RNA sequencing showed that the transcriptional response after 6h of exposure to wheat straw was different from the response at 24h of exposure: the complexity increased over time.
 </p>
 <p>
  Importantly, the influence of carbon starvation during lignocellulose degradation was demonstrated by a substantial overlap in CAZyme-encoding transcripts induced during both early carbon starvation and early exposure to straw. The up-regulation of the expression of a high number of genes encoding CAZymes that are active on plant-derived carbohydrates during early carbon starvation suggests that these enzymes could be involved in a scouting role during starvation, releasing inducing sugars from complex plant polysaccharides. We show that carbon-starved cultures indeed release CAZymes with predicted activity on plant polysaccharides. Analysis of the enzymatic activity and the reaction products, indicates that these proteins are enzymes that can degrade various plant polysaccharides to generate both known, as well as potentially new, inducers of CAZymes (van Munster, Daly
  <em>
   et al,
  </em>
  (2014) Fungal Genet. Biol. 72; 34-47).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410102
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  RNA-seq    Transcription    Transcriptome
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Potential microRNAs regulate the response to injury of the filamentous fungus
 <em>
  Trichoderma atroviride
 </em>
 .
 <br/>
 <b>
  Jose Manuel Villalobos-Escobedo
 </b>
 , Nohemí Carreras-Villaseñor, Joel Rodríguez-Medina, Cei Abreu-Goodger, Alfredo Herrera-Estrella. LANGEBIO-CINVESTAV, Km 9.6 Libramiento Norte Carretera Irapuato-León. Irapuato, Guanajuato, Mexico
 <br/>
 <p>
  Wound response in multicellular eukaryotes is essential for survival and is highly conserved in plants and animals. In our laboratory we recently discovered that the  filamentous fungus
  <em>
   Trichoderma atroviride
  </em>
  responds to injury by triggering hyphal regeneration and the formation of asexual reproductive structures. Our transcriptomic analysis revealed that the mechanism of response to this stimulus is very similar to that of animals and plants, suggesting that it is highly conserved amongst the three eukaryotic kingdoms. Additionally, several recent studies have reported that post-transcriptional regulation by microRNAs is involved in the response to injury in animals and plants.
 </p>
 <p>
  Based on this background we decided to evaluate the injury response in mutants of the RNAi synthesis machinery of
  <em>
   T. atroviride
  </em>
  . The
  <em>
   &#916;dcr2
  </em>
  and
  <em>
   &#916;rdr3
  </em>
  strains presented a dramatic defect in regeneration ability and asexual reproduction in response to injury. To understand the molecular processes affected by the absence of the RNAi pathway, we performed transcriptomic analysis of the WT and
  <em>
   &#916;dcr2
  </em>
  strains subjected to injury, showing that signaling processes, DNA repair and cell cycle progression are essential to overcome this stress and are affected in the
  <em>
   &#916;dcr2
  </em>
  mutant. Even more interesting was the presence of a population of small RNAs of 21-22nt in response to injury in the WT, which is absent in &#916;dcr2 mutant, implicating them as a product of the Dcr2 enzyme. microRNA prediction allowed to determine that most 21-22 nt small RNAs correspond to microRNAs.
 </p>
 <p>
  Our results indicate that gene regulation by putative-microRNAs is essential to respond to injury in
  <em>
   T. atroviride.
  </em>
  This phenomenon opens the possibility of demonstrating the relevance of post-transcriptional regulation by microRNAs in filamentous fungi in response to environmental stimuli. Furthermore, the knowledge generated may be helpful in understanding the fine regulation by microRNAs of the tissue repair process, which could culminate in enhancing regenerative therapies in humans. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410624
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Development of a community consensus model for
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  Julian Brandl
 </b>
 <sup>
  1
 </sup>
 , Mhairi Workman
 <sup>
  1
 </sup>
 , Mikko Arvas
 <sup>
  2
 </sup>
 , Vera Meyer
 <sup>
  3
 </sup>
 , Mikael R. Andersen
 <sup>
  1
 </sup>
 . 1) Department of Systems Biology, Technical University of Denmark, Kgs. Lyngby, Denmark; 2) VTT Technical Research Centre of Finland, Finland; 3) Department of Applied and Molecular Microbiology, Institute of Biotechnology, Technical University of Berlin, Germany
 <br/>
 <p>
  Fungal primary metabolism is an essential part of fungal physiology and affects all phenotypic traits of the organism as well as carrying the biotechnological potential for the fungal host. While the study of individual pathways have gained essential knowledge and important scientific breakthroughs, a genome-scale view of metabolism is required to gain a holistic understanding of the cell. Mathematical models based on the stoichiometry of known enzymatic reactions have been developed in order to facilitate this approach and proven useful for guiding metabolic engineering in well characterized model organisms like
  <em>
   S. cerevisiae
  </em>
  and
  <em>
   E. coli.
  </em>
  With the sustained interest in
  <em>
   Aspergillus niger
  </em>
  as a potent host organism for citric acid and enzyme production, it is timely to improve on previous genome-scale modeling efforts. Here we aim at updating the genome-scale model by a combination of experimental work and integration of published information. This joint effort of international collaborators and our group will yield a community-consensus model of the
  <em>
   Aspergillus niger
  </em>
  metabolism. In order to improve the gene assignments contained in the current version of the model, we will use comparative genomics to identify shared isoenzymes and gene groups between closely related species in the section
  <em>
   Nigri
  </em>
  . To accurately examine and model the catabolic potential of the fungus, we will apply Biolog plates for the screening of more than 270 carbon and nitrogen sources. This knowledge will aid to identify missing pathways in the model and validate the presence of many pathways already included. Additionally the biosynthesis of 2-300 secreted enzymes will be included in the new version of the model. In conclusion this project aims at generating an experimentally validated community-consensus model of the
  <em>
   A. niger
  </em>
  metabolism being able to describe and predict beneficial modifications to the metabolic network in order to improve protein production on a variety of different substrates.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410338
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Secondary metabolism    Transcription
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Distinct histone amino acids are involved in controlling secondary metabolism in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  Juliane Fischer
 </b>
 <sup>
  1,3
 </sup>
 , Hans-Wilhem Nützmann
 <sup>
  1
 </sup>
 , Kirstin Scherlach
 <sup>
  2
 </sup>
 , Christian Hertweck
 <sup>
  2
 </sup>
 , Axel A. Brakhage
 <sup>
  1,3
 </sup>
 . 1) Departments of Molecular and Applied Microbiology, Hans-Knöll-Institut, Jena, Germany; 2) Biomolecular Chemistry, Hans-Knöll-Institut, Jena, Germany; 3) Friedrich Schiller University Jena, Jena, Germany
 <br/>
 <p>
  Chromatin remodeling events have recently been discovered to play an important role in the secondary metabolism of filamentous fungi (1). Previously, we showed that a bacterium,
  <em>
   Streptomyces rapamycinicus
  </em>
  , is able to re-program the histone modifying SAGA/Ada complex including the histone acetyltransferase GcnE of the important model fungus
  <em>
   Apergillus nidulans
  </em>
  (2, 3). As a consequence, lysine 9 and lysine 14 of distinct secondary metabolism genes were specifically acetylated during the bacterial fungal interaction, which, furthermore, led to the activation of the silent orsellinic acid gene cluster. To prove the importance of histone modifications for distinct gene expression profiles, we exchanged several amino acids of histone H3 of
  <em>
   A. nidulans
  </em>
  (4). These amino acids included lysine residues 9, 14, 18, and 23 as well as serine 10 and threonine 11. Lysine residues and serine/ threonine were exchanged to arginine and alanine respectively. Interestingly, all the strains produced were viable, allowing to analyse directly the consequences of missing posttranslational histone modifications. In the mutant strains, for the penicillin, sterigmatocystin and orsellinic acid biosyntheses, we detected major changes of both the expression patterns at the transcriptional level and the metabolite level of the respective gene clusters. These effects were mainly due to lysine 9 and lysine 14 of histone H3. Taken together, for the first time we show a causal linkage between the lack of acetylation of lysine residues on histone H3 and the transcription and product formation of important secondary metabolites. This leads to the question how the SAGA/ADA-dependent histone acetylation program is specifically targeted to a distinct gene cluster on the genome and how this is induced by a bacterium. (1) Brakhage (2013)
  <em>
   Nat. Rev. Microbiol
  </em>
  ., (2) Schroeckh
  <em>
   et al
  </em>
  . (2009)
  <em>
   PNAS
  </em>
  , (3) Nützmann
  <em>
   et al
  </em>
  . (2011)
  <em>
   PNAS
  </em>
  , (4) Nützmann, Fischer
  <em>
   et al.
  </em>
  (2013)
  <em>
   AEM.
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410743
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The
 <em>
  Aspergillus fumigatus
 </em>
 genome-wide knock out library.
 <br/>
 <b>
  Juliane Macheleidt
 </b>
 <sup>
  1
 </sup>
 , Thorsten Heinekamp
 <sup>
  1
 </sup>
 , Vito Valiante
 <sup>
  1
 </sup>
 , Fabian Horn
 <sup>
  2
 </sup>
 , Reinhard Guthke
 <sup>
  2
 </sup>
 , Paul Carr
 <sup>
  3
 </sup>
 , Jane Gilsenan
 <sup>
  3
 </sup>
 , Isabelle Mouyna
 <sup>
  4
 </sup>
 , Jean-Paul Latgé
 <sup>
  4
 </sup>
 , Michael Bromley
 <sup>
  3
 </sup>
 , Axel Brakhage
 <sup>
  1
 </sup>
 . 1) Leibniz-Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Molecular and Applied Microbiology, Jena, Germany; 2) Leibniz-Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Systems Biology and Bioinformatics, Jena, Germany; 3) Institute of Inflammation and Repair, University of Manchester, UK; 4) Institute Pasteur, Aspergillus Unit, Paris, France
 <br/>
 <p>
  The opportunistic human pathogenic mould
  <em>
   Aspergillus fumigatus
  </em>
  is able to cause severe invasive infections. The most powerful tool to study the role of certain genes and proteins in virulence of
  <em>
   A. fumigatus
  </em>
  is to generate gene knock outs and to analyze the loss of function mutants. Up to now, these deletion strains were generated separately for each gene of interest, which is often a time consuming process. To facilitate and accelerate research on
  <em>
   A. fumigatus
  </em>
  , we started to build up a library of deletion mutants for all of the approximately 10,000 genes of this fungus.
 </p>
 <p>
  Prerequisite for a knock out library are accurately annotated genes. Therefore, the genome of the
  <em>
   A. fumigatus
  </em>
  A1163 derivative &#916;
  <em>
   akuB
  </em>
  was re-sequenced and RNAseq data were used for re-annotation. Based on that, gene deletion primers were generated in an automated process and a streamlined workflow combining generation of deletion cassettes by a two-step PCR reaction, protoplast-based transformation, isolation of genomic DNA and verification of the successful deletion by PCR was developed in 96-well plate format for high throughput application. Based on the introduction of an MmeI restriction site to the deletion cassette, each mutant carries a unique barcode allowing identification of specific strains when pools of mutants are tested. As a first subset, we deleted 132 glycosylphosphatidylinositol (GPI) anchored protein encoding genes and started phenotypic analysis of the resulting mutant strains.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410378
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Sexual Development    Hyphal Fusion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Invited
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Inducing sexual reproduction in the industrial fungus
 <em>
  Aspergillus oryzae
 </em>
 : Can the domesticated fungus get sexy again?
 <br/>
 <b>
  Jun-ichi Maruyama
 </b>
 , Katsuhiko Kitamoto. Department of Biotechnology, The University of Tokyo, Tokyo, Japan
 <br/>
 <p>
  <em>
   Aspergillus oryzae
  </em>
  is an industrially important fungus used for the traditional fermentative manufacture of Japanese foods and heterologous protein production. As sexual reproduction has not been observed in
  <em>
   A. oryzae
  </em>
  , it is quite difficult to breed strains with industrially useful characteristics. We identified two mating types of
  <em>
   A. oryzae
  </em>
  (MAT1-1 and MAT1-2)
  <sup>
   1)
  </sup>
  , indicating that this fungus has the potential for sexual reproduction in a heterothallic manner.
 </p>
 <p>
  Cell fusion is the first process in sexual reproduction, and we have recently demonstrated that
  <em>
   A. oryzae
  </em>
  can enter cell fusion
  <sup>
   2,3)
  </sup>
  . The second important step is the formation of sexual reproductive structures. A sclerotium, a survival mycelial structure, is capable of acting as repositories for ascocarps forming sexual ascospores in other
  <em>
   Aspergillus
  </em>
  species. However,
  <em>
   A. oryzae
  </em>
  is thought to have been domesticated from the ancestor
  <em>
   Aspergillus flavus
  </em>
  , and
  <em>
   A. oryzae
  </em>
  strains have lost the ability to form sclerotia or have a much lower ability.
 </p>
 <p>
  We deleted the
  <em>
   ecdR
  </em>
  gene encoding for a transcription factor negatively regulating sclerotia formation
  <sup>
   4)
  </sup>
  , which increased heterokaryotic sclerotia of the two
  <em>
   A. oryzae
  </em>
  mating-type strains. However, this was not sufficient to induce the formation of sexual reproductive structures. Then, sexual reproduction-related genes were overexpressed in the &#916;
  <em>
   ecdR
  </em>
  strain, and ascocarps, asci and ascospore-like structures were formed in the sclerotia. Thus, these sexual reproductive structures were found for the first time in
  <em>
   A. oryzae
  </em>
  . If genetic crossing is found in the ascospore-like structures, it will be possible to perform crossbreeding for industrially useful strains in
  <em>
   A. oryzae
  </em>
  .
 </p>
 <p>
  1) Wada
  <em>
   et al.
  </em>
  (2012)
  <em>
   Appl. Environ. Microbiol.
  </em>
  2) Tsukasaki
  <em>
   et al.
  </em>
  (2014)
  <em>
   Biosci. Biotechnol. Biochem.
  </em>
  3) Wada
  <em>
   et al.
  </em>
  (2014)
  <em>
   Appl. Microbiol. Biotechnol.
  </em>
  4) Jin
  <em>
   et al.
  </em>
  (2011)
  <em>
   Fungal Genet. Biol..
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410521
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Light    Conidiation
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Investigation of molecular mechanism regulating light-dependent repression of conidiation in
 <em>
  Aspergillus oryzae
 </em>
 .
 <br/>
 <b>
  Jun-ichi Maruyama
 </b>
 , Helge M. Dietrich, Feng Jie Jin, Katsuhiko Kitamoto. Department of Biotechnology, The University of Tokyo, Tokyo, Japan
 <br/>
 <p>
  The domesticated filamentous fungus
  <em>
   Aspergillus oryzae
  </em>
  is an important for Japanese fermentation industry. Its conidia are of high interest for industrial use such as
  <em>
   tane-koji
  </em>
  (
  <em>
   koji
  </em>
  starter) producing companies and fermentation companies. Although other
  <em>
   Aspergillus
  </em>
  species such as
  <em>
   Aspergillus nidulans
  </em>
  predominantly undergo conidiation under light illumination
  <sup>
   1)
  </sup>
  ,
  <em>
   A. oryzae
  </em>
  forms conidia in dark, but conidiation efficiency is reduced in light
  <sup>
   2)
  </sup>
  . It could be speculated that
  <em>
   A. oryzae
  </em>
  strains forming conidia in dark have been selected due to the industrial necessity to grow them in dark room. How the fungus reacts to light in this reverse way has not yet been understood. To investigate this question, we attempted to molecularly uncover the light response mechanism in
  <em>
   A. oryzae
  </em>
  .
 </p>
 <p>
  When LED illumination was used, blue light was sufficient for the repression of conidiation, whereas red light had no impact. Subsequently, deletion strains of the genes for putative blue light (
  <em>
   AolreA
  </em>
  ) and red light (
  <em>
   AofphA
  </em>
  ) receptors were generated. White and blue light illuminations resulted in the repression of conidiation for the wild-type and &#916;
  <em>
   AofphA
  </em>
  strains. Deletion of
  <em>
   AolreA
  </em>
  gene caused a complete attenuation of the light-induced repression. These results indicate that AoLreA but not AoFphA is essential for light-dependent repression of conidiation. RT-PCR analysis was performed for
  <em>
   brlA
  </em>
  gene encoding a positive conidiation regulator. White and blue light illumination decreased the
  <em>
   brlA
  </em>
  mRNA amount in the wild-type and &#916;
  <em>
   AofphA
  </em>
  strains, which is consistent with their repressive conidiation phenotypes in light. In contrast, no light-dependent decrease of
  <em>
   brlA
  </em>
  mRNA amount was observed in the absence of AoLreA. These data indicate that AoLreA governs the light-induced repression of conidiation in
  <em>
   A. oryzae
  </em>
  , which is an opposite way to that of
  <em>
   A. nidulans
  </em>
  LreA.
 </p>
 <p>
  1) Ruger-Herreros
  <em>
   et al
  </em>
  . (2011)
  <em>
   Genetics
  </em>
  2) Hatakeyama
  <em>
   et al.
  </em>
  (2007)
  <em>
   Biosci. Biotechnol. Biochem..
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410319
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Sexual Development    Aspergillus nidulans    Whole Genome Sequencing
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 The
 <em>
  nsdA4
 </em>
 mutation in NSD204 strain, which is defective in sexual development, is an allele of the
 <em>
  nsdC
 </em>
 gene in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 Chae-Ho Lim
 <sup>
  1
 </sup>
 , Mohammed Abdo Elgabbar
 <sup>
  1
 </sup>
 , Dong-Min Han
 <sup>
  2
 </sup>
 , Masayuki Machida
 <sup>
  3
 </sup>
 ,
 <b>
  Kap-Hoon Han
 </b>
 <sup>
  1
 </sup>
 . 1) Dept. Pharmaceutical Engineering, Woosuk Univ, Wanju, Jeonbuk, 565-701, South Korea; 2) 2Division of Life Science, Wonkwang University, Iksan, 570-749, South Korea; 3) 3Bioproduction Research Institute, Hokkaido Center, National Institute of Advanced Industrial Science and Technology (AIST), Sapporo, Hokkaido 062-8517 Japan
 <br/>
 <p>
  Fruiting body production and sexual development of fungi are crucial for producing of ascospores by meiosis as well as adapting various environmental changes. In a homothallic fungus
  <em>
   Aspergillus nidulans
  </em>
  , many environmental factors and genes affecting sexual development have been elucidated so far. To investigate sexual developmental process further, NSD mutants, which are defective in the sexual development, have been isolated and characterized. The NSD mutants were divided into four different complementation groups, NSDA-D, and the two genes responsible for the
  <em>
   nsdC
  </em>
  and
  <em>
   nsdD
  </em>
  mutation have already been reported. However,
  <em>
   nsdA4
  </em>
  and
  <em>
   nsdB5
  </em>
  mutations from NSD204 and NSD205 mutants, respectively, are remained to be unveiled. Whole genome sequence of NSD204 mutant obtained from Next Generation Sequencing (NGS) identified possible
  <em>
   nsdA4
  </em>
  mutation candidates. Recent intensive mutation analysis revealed that the NSD204 mutant strain carries missense mutations in
  <em>
   nsdC
  </em>
  ORF region, suggesting that phenotype of NSD204 mutant might be derived from the novel
  <em>
   nsdC
  </em>
  mutation, and eventually indicated that
  <em>
   nsdA4
  </em>
  is an allele of
  <em>
   nsdC
  </em>
  gene. To verify this, NSD204 was genetically crossed with test strain and check the correlation between
  <em>
   nsdA
  </em>
  <sup>
   -
  </sup>
  phenotype and
  <em>
   nsdC
  </em>
  mutation. As a result, all strains showing
  <em>
   nsdA
  </em>
  <sup>
   -
  </sup>
  phenotype carried
  <em>
   nsdC
  </em>
  mutation which is exactly same mutation found in NSD204 mutant strain, indicating that the
  <em>
   nsdA
  </em>
  gene is identical to the
  <em>
   nsdC
  </em>
  gene. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410370
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Conidiation    Transcription factor    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 The HLH transcription factor
 <em>
  ndrA
 </em>
 , which is necessary for conidiation, negatively controls sclerotia formation in
 <em>
  Aspergillus flavus
 </em>
 .
 <br/>
 <b>
  Kap-Hoon Han
 </b>
 <sup>
  1
 </sup>
 , Mohammed A. Abdo Elgabbar
 <sup>
  1
 </sup>
 , Dong-Min Han
 <sup>
  2
 </sup>
 . 1) Pharmaceutical Engineering, Woosuk University, Wanju, Jeonbuk, 565-701, South Korea; 2) Division of Life Science, Wonkwang University, Iksan, 570-749, South Korea
 <br/>
 <p>
  <em>
   Aspergillus flavus
  </em>
  is a saprophytic and pathogenic fungus that can infect animals and humans directly or indirectly by its secondary metabolites. It mainly reproduces clonally by means of conidia (asexual spores), although sexual developmental process has been recently reported. In eukaryotes, the helix-loop-helix (HLH) transcriptional factors play an important role in the developmental processes. One of these factors is
  <em>
   Aspergillus nidulans
  </em>
  HLH transcription factor
  <em>
   ndrA
  </em>
  , which is involved in the early stage of conidiophore development and sexual development. Previous unpublished results showed that expression of the
  <em>
   ndrA
  </em>
  (NsdD-Dependent Regulator) gene is largely affected by a GATA factor NsdD, and is negative regulator of sexual development as well as positive regulator of asexual development. By BLASTP of
  <em>
   A. niduans
  </em>
  NdrA, we identified the orthologue of
  <em>
   ndrA
  </em>
  (
  <em>
   AflndrA
  </em>
  ) in
  <em>
   A. flavus
  </em>
  . The deletion of
  <em>
   AflndrA
  </em>
  resulted in almost absence of conidia yet abundant production of sclerotia. The complementation of
  <em>
   AflndrA
  </em>
  deleted strain by the intact
  <em>
   AflndrA
  </em>
  ORF has restored the conidiation as in the wild type with diminishing sclerotia. Moreover, we found that,
  <em>
   AflndrA
  </em>
  dose not affect the aflatoxin production as well as the antifungal drug sensitivity or resistance. The expression of
  <em>
   AflndrA
  </em>
  is upregulated at 12 hours under asexual development favorable condition. Taken together, the
  <em>
   AflndrA
  </em>
  gene could be considered as a one of the conidiation-critical and sclerotia controlling genes in
  <em>
   A. flavus..
  </em>
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410388
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Conidiation    Aspergillus fumigatus    Transcription factor
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 <em>
  Aspergillus fumigatus
 </em>
 HLH transcription factor,
 <em>
  Afu
 </em>
 <em>
  ndrA
 </em>
 , is required for conidiation.
 <br/>
 Mohammed A. Abdo Elgabbar
 <sup>
  1
 </sup>
 , Sang-Cheol Jun
 <sup>
  2
 </sup>
 , Jong-Hwa Kim
 <sup>
  1
 </sup>
 , Dong-Min Han
 <sup>
  3
 </sup>
 ,
 <b>
  Kap-Hoon Han
 </b>
 <sup>
  1
 </sup>
 . 1) Pharmaceutical Engineering, Woosuk University, Wanju, Jeonbuk, 565-70, South Korea; 2) Division of Biological Sciences, Chonbuk National University, Jeonju, 561-756, South Korea; 3) Division of Life Science, Wonkwang University, Iksan, 570-749, South Korea
 <br/>
 <p>
  Asexual reproduction and conidiation in the Aspergillus spp. is a mean by which progeny arise from a single parent, and inherit the parental genes only. In an opportunistic pathogenic fungus
  <em>
   Aspergillus fumigatus
  </em>
  , conidia are the primary causative agent of invasive aspergillosis. The helix-loop-helix (HLH) transcriptional factors that control cell growth and differentiation are considered as key regulators for a wide range developmental processes. In a model fungus
  <em>
   Aspergillus nidulans
  </em>
  , one HLH gene, named
  <em>
   ndrA
  </em>
  , which is regulated by NsdD GATA factor, has been isolated and characterized as a negative regulator of sexual development as well as a positive regulator of asexual development. To study conserved and divergent role of the
  <em>
   A. nidulans
  </em>
  <em>
   ndrA
  </em>
  , we performed BLAST search and identified the
  <em>
   A. fumigatus
  </em>
  ortholog
  <em>
   AfundrA
  </em>
  gene, which is its knockout made this fungus unable to produce the conidia. The
  <em>
   AfundrA
  </em>
  complemented strain was able to produce numerous amounts of conidia, which is as same as the wild type strain. Northern analysis showed that the
  <em>
   AfundrA
  </em>
  gene was highly expressed in the early stage in the conidiation. There was no difference between the wild type and
  <em>
   AfundrA
  </em>
  deletion mutant when they subjected to antifungal sensitivity test. Moreover, there were no big differences in the growth rates between the wild type and
  <em>
   AfundrA
  </em>
  deletion mutant. Taken together, in
  <em>
   A. fumigatus
  </em>
  <em>
   AfundrA
  </em>
  gene plays a pivotal role in controlling conidiation.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410490
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Gene Deletion    Proteomics
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Phenotype analysis of
 <em>
  Rice koji
 </em>
 protein genes disruptants in
 <em>
  Aspergillus oryzae
 </em>
 .
 <br/>
 <b>
  Kazuhiro Iwashita
 </b>
 <sup>
  1,2
 </sup>
 , Teruaki Hanada
 <sup>
  1,2
 </sup>
 , Shinichiro Hukuhara
 <sup>
  1,2
 </sup>
 , Shingo Kakiuchi
 <sup>
  1,2
 </sup>
 , Hiroaki Kitamura
 <sup>
  1,2
 </sup>
 , Ken Oda
 <sup>
  1
 </sup>
 , Minori Kouno
 <sup>
  1
 </sup>
 . 1) National Research Institute of Brewing, Higashihiroshima, Hiroshima, Japan; 2) Graduate School of Advanced Sciences of Matter, Hiroshima University, Higashihiroshima, Hiroshima, Japan
 <br/>
 <p>
  <em>
   Aspergillus oryzae
  </em>
  are used for Japanese traditional fermentation industry, such as Sake, Miso-paste, and Rice vinegar. In sake industry,
  <em>
   A. oryzae
  </em>
  cultivate on rice to make
  <em>
   Rice koji
  </em>
  , which supply various hydrolytic enzymes and vitamins for following fermentation by yeast. The property of
  <em>
   Rice koji
  </em>
  significantly affects to the flavor of Sake.
  <em>
   A. oryzae
  </em>
  encode about12, 000 genes on the genome and will play important role for the property of
  <em>
   Rice koji
  </em>
  . Some hydrolytic enzymes, such as amylases and proteinases, were well studied but most of other genes were left uncharacterized. Thus, we prepared 4 different types of
  <em>
   Rice koji
  </em>
  and performed proteome analysis using MALDI-TOF/TOF MS. As the result, 159 genes were identified as
  <em>
   Rice koji
  </em>
  protein encoding (RKP) genes. The 38 genes were well characterized and 51 genes were predicted as heat shock, secretion and primary metabolism related genes. The remained 70 genes were poorly annotated proteins. Among 159 genes, we select 85 RKP genes and disrupted using
  <em>
   adeA
  </em>
  as a marker. We success the disruption of 73 RKP genes but only heterokaryons were isolated for remained 12 RKP genes, despite of several trials.  To evaluate the phenotype of RKP disruptants, we examined growth and conidia formation on plate culture. Only 2 RKP disruptants reduced their growth significantly but 11 RKPs disruptants were altered in conidia formation. Interestingly, 10 strains extremely reduced the growth in liquid culture, even though same medium was used.  We further prepared
  <em>
   Rice koji
  </em>
  and evaluate the growth and enzyme production. Comparing with the plate culture, 11 RKP disruptants reduced their growth. The growth and protein production was well correlated. In this study, we could identify new genes which affect the growth end enzyme production. However, many RKP disruptants did not show any phenotypes.  To examine the function of these non-phenotype RKP genes, we will prepare Sake to examine the effect to Sake metabolites and its quality.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410482
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Signal Transduction    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Ethylene response of the plant-fungal fusion histidine kinase in yeast and filamentous fungi.
 <br/>
 Mayumi Nakayama
 <sup>
  1,2
 </sup>
 , Kentaro Furukawa
 <sup>
  3
 </sup>
 , Akira Yoshimi
 <sup>
  2
 </sup>
 ,
 <b>
  Keietsu Abe
 </b>
 <sup>
  1,2
 </sup>
 . 1) Dept Microbial Biotechnology, Grad. Sch. Agricult. Sci., Tohoku Univ., Sendai, Miyagi, Japan; 2) New Industry Hatchery Center, Tohoku University, Sendai, Miyagi, Japan; 3) University of Gothenburg, Gothenburg, Sweden
 <br/>
 <p>
  The two-component signal transduction system (TCS) has been conserved widely in bacteria and eukaryotes, including plants and fungi. TCS typically consists of two types of common signal transducers: histidine kinase (HK), a response regulator (RR). In the plant
  <em>
   Arabidopsis thaliana
  </em>
  , the ethylene receptor AtETR1 acts as HK and its HK activity is regulated by ethylene. AtETR1 contains (i) an ethylene-binding domain (EBD) consisting of three transmembrane helices in the N-terminal half, (ii) an HK domain (HKD) containing HK, and (iii) a receiver domain of RR in the C-terminal half. As well as AtETR1, fungal HKs also consist of an N-terminal sensor detecting environmental stimuli, an HKD and an RR. Fungal signal transduction pathways containing  these fungal sensors control cellular responses to extrinsic and intrinsic signals. If these fungal sensor domains are replaced by EBD of AtETR1 and fungal signal pathways (e.g. HOG-pathway) can be controlled by ethylene, the hybrid HKs would be useful as a new gene regulation system in fungal industry. To create a novel system of gene regulation by ethylene, we constructed and examined expression systems of plant-fungal fusion HKs in yeast and filamentous fungi. Then we confirmed functional complementation and ethylene response of plant-fungal fusion HKs in a temperature-sensitive
  <em>
   sln1
  </em>
  yeast mutant and
  <em>
   Aspergillus nidulans
  </em>
  . Here, we report experiments for optimization of ethylene response using HOG-dependent reporter systems,
  <em>
   8xCRE-lacZ
  </em>
  and their modified genes.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410289
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The development of promoter shutoff system for functional analysis of essential genes with sorbitol metabolic pathway gene promoter in
 <em>
  Asepergillus oryzae
 </em>
 .
 <br/>
 S. Terado,
 <b>
  K. Oda
 </b>
 , A. Shimabara, R. Toyoura, M. Kawauchi, H. Fukuda, K. Iwashita. Fundamental Res. Division, Nat. Res. Inst. of Brewing, Hiroshima, Japan
 <br/>
 <p>
  The genome sequence of several filamentous fungi revealed that almost half of genes in genome are functional unknown. So the demand of analyzing such functional unknown gene has been increased but some of the genes are essential. The promoter shutoff is general way for analyzing such essential genes. In
  <em>
   A. nidulans
  </em>
  , several conditional promoters such as
  <em>
   alcA
  </em>
  exist and showed usefulness for molecular biological studies, while in
  <em>
   A.oryzae
  </em>
  there are a few promoters which can be tightly controlled. To expand the limitation of conditional promoter use in
  <em>
   A.oryzae
  </em>
  , we explored about sorbitol and galactose metabolic pathway gene for expecting tight regulation and the less physiological effect with conditional change. We performed microarray analysis in the sorbitol or galactose (induced condition) and glucose (repressed condition) culture with
  <em>
   A.oryzae
  </em>
  RIB40 strain. Two genes are found with over 50-fold induction at sorbitol culture condition, and designated as
  <em>
   sorA
  </em>
  and
  <em>
   sorB
  </em>
  gene. To evaluate the induction with both gene promoters, we developed the EGFP expression system under the
  <em>
   sorA
  </em>
  promoter (P
  <em>
   sorA
  </em>
  ) and P
  <em>
   sorB
  </em>
  control. Blight EGFP fluorescence and expression were detected at sorbitol condition, while not detected at glucose repressed condition in transformants. In addition the promoter of
  <em>
   brlA
  </em>
  which is a master regulator of the conidiation was replaced by the P
  <em>
   sorA
  </em>
  and P
  <em>
   sorB
  </em>
  , and the sorbitol dependent conidia formation was observed. This result indicated the useful of P
  <em>
   sorA
  </em>
  and P
  <em>
   sorB
  </em>
  for promoter shutoff.
 </p>
 <p>
  To apply this promoter shutoff system for the functional analysis of essential gene, we constructed inducible PsorA::
  <em>
   rhoA
  </em>
  which is known as the essential regulator of the beta-1,3-glucan synthase. PsorA::
  <em>
   rhoA
  </em>
  strain showed complete growth inhibition at repressed condition while the strain grew normal at the induction condition, indicating this system is available for essential gene analysis. This promoter shutoff system can be further applied for essential functional unknown gene.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410477
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Gene Expression    Metabolism    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Genome-wide transcriptional regulation and chromatin dynamics in response to nitrogen availability in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 Damien J. Downes
 <sup>
  2
 </sup>
 , Zhengqiang Miao
 <sup>
  1
 </sup>
 , Djordje Djordjevic
 <sup>
  3
 </sup>
 , Vinita Deshpande
 <sup>
  3
 </sup>
 , Ang Li
 <sup>
  1
 </sup>
 , Kaeling Tan
 <sup>
  1
 </sup>
 , Grethel Y Busot
 <sup>
  2
 </sup>
 , Joshua W.K. Ho
 <sup>
  3
 </sup>
 , Richard B. Todd
 <sup>
  2
 </sup>
 ,
 <b>
  Koon Ho Wong
 </b>
 <sup>
  1
 </sup>
 . 1) Faculty of Health Sciences, University of Macau, Taipa, Macau; 2) Department of Plant Pathology, Kansas State University, Manhattan KS, USA.; 3) Victor Chang Cardiac Research Institute, and the University of New South Wales, Sydney, NSW, Australia.
 <br/>
 <p>
  Fungi adapt their metabolism to nitrogen nutrient availability primarily via global transcriptional control of nitrogen uptake and metabolic genes. In
  <em>
   Aspergillus nidulans,
  </em>
  the GATA transcription factor AreA activates genes for nitrogen metabolism in response to nitrogen limitation or nitrogen starvation. We have performed ChIP-seq of RNA polymerase II in wild type and an
  <em>
   areA
  </em>
  &#916; mutant in the presence of ammonium and during nitrogen starvation to identify the genome-wide AreA-dependent gene expression program. Using ChIP-seq of HA-epitope-tagged AreA, we determined the pan-genomic direct targets of this global regulator. We have also mapped global chromatin dynamics and genome-wide chromatin modifications in response to nitrogen nutrient availability. Our data reveal nitrogen transport, metabolic and regulatory gene targets as well as new targets for AreA, including promoters of iron siderophore biosynthesis genes, heme metabolism genes, and secondary metabolism genes. Differential chromatin modifications occurred in response to nitrogen availability and were enriched in secondary metabolism genes. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410579
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Biomass    Other (Other Keyword: Penicillium)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Plant biomass degrading potential of a new
 <em>
  Penicillium
 </em>
 species,
 <em>
  Penicillium subrubescens
 </em>
 .
 <br/>
 Miia R. Mäkelä
 <sup>
  1
 </sup>
 , Sadegh Mansouri
 <sup>
  1
 </sup>
 , Ad Wiebenga
 <sup>
  2
 </sup>
 , Ronald P. de Vries
 <sup>
  2
 </sup>
 ,
 <b>
  Kristiina Hilden
 </b>
 <sup>
  1
 </sup>
 . 1) Department of Food and Environmental Sciences, Division of Microbiology and Biotechnology, Viikki Biocenter 1, University of Helsinki, Finland; 2) Fungal Physiology, CBS-KNAW Fungal Biodiversity Centre &amp; Fungal Molecular Physiology, Utrecht University, Uppsalalaan 8, 3584 CT Utrecht, The Netherlands
 <br/>
 <p>
  A recently identified
  <em>
   Penicillium
  </em>
  species,
  <em>
   P. subrubescens
  </em>
  (Mansouri et al. 2013), was evaluated for its ability to hydrolyse plant biomass. Growth on various plant biomass related substrates demonstrated the capacity of this species to degrade all the main polysaccharides present in plant biomass as well as metabolise all their monomeric components. The ability to degrade broad range of carbohydrates suggests a high potential in plant biomass saccharification.
 </p>
 <p>
  To evaluate this in more detail,
  <em>
   P. subrubescens
  </em>
  was grown on wheat bran and sugar beet pulp and a set of extracellular enzyme activities were analyzed from culture liquids. Also the ability to saccharify wheat bran and sugar beet pulp was determined. Compared to
  <em>
   P. chrysogenum
  </em>
  ,
  <em>
   P. subrubescens
  </em>
  produced higher levels of &#946;-glucosidase, endoglucanase, endoxylanase and cellobiohydrolase.
 </p>
 <p>
  Enzyme mixtures produced on wheat bran by
  <em>
   P. subrubescens
  </em>
  were more efficient in saccharification of wheat bran compared to enzymes produced on sugar beet pulp cultures. The opposite result was observed for saccharification of sugar beet pulp. This demonstrates that
  <em>
   P. subrubescens
  </em>
  produces enzyme mixtures that are closely tailored to the available substrate, suggesting the presence of a fine-tuned regulatory system that controls the production of these enzymes.
 </p>
 <p>
  Reference: Mansouri S, Houbraken J, Samson RA, Frisvad JC, Christensen M, Tuthill DE, Koutaniemi S, Hatakka A, Lankinen P (2013)
  <em>
   Penicillium subrubescens
  </em>
  , a new species efficiently producing inulinase. Antonie Van Leeuwenhoek 103:1343-1357.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410282
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Secondary metabolism    Virulence
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Commonalities and partial redundancy of two LaeA- and BrlA-regulated conidial polyketide metabolites in
 <em>
  A. fumigatus
 </em>
 .
 <br/>
 <b>
  Kurt Throckmorton
 </b>
 <sup>
  1
 </sup>
 , Fang Yun Lim
 <sup>
  2
 </sup>
 , Dimitrios Kontoyiannis
 <sup>
  3
 </sup>
 , Weifa Zheng
 <sup>
  4
 </sup>
 , Nancy Keller
 <sup>
  2
 </sup>
 . 1) Department of Genetics, University of Wisconsin-Madison, Madison, WI.; 2) Department of Medical Microbiology and Immunology, University of Wisconsin-Madison, Madison, WI; 3) MD Anderson Cancer Center, University of Texas, Houston, TX; 4) Key Laboratory for Biotechnology, Jiangsu Normal University, Xuzhou, China
 <br/>
 <p>
  The opportunistic human pathogen
  <em>
   Aspergillus fumigatus
  </em>
  produces many secondary metabolites, some of which are immunomodulatory or toxic and are thought to contribute to its virulence. Two such secondary metabolites, trypacidin and endocrocin, are both produced by non-reducing polyketide synthases, and are localized to the conidium of
  <em>
   A. fumigatus
  </em>
  , requiring LaeA and BrlA for their synthesis. Furthermore, they are predicted to share an early precursor, atrochrysone carboxylic acid, and are both thermally regulated, being produced at lower levels at higher temperatures. In an isolate of
  <em>
   A. fumigatus
  </em>
  which does not produce trypacidin, CEA10, deletion of the endocrocin polyketide synthase results in loss of endocrocin. However, in an isolate which produces trypacidin, AF293, the same deletion does not affect production of endocrocin. Genetic dissection of the biosynthetic pathways of these metabolites suggests that endocrocin is produced by both of these physically discrete clusters. This redundancy is of uncertain adaptive advantage, but is the first example of its kind.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410452
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Transcription factor    GFP
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Regulation and characterisation of the CreA carbon catabolite repressor in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  LAURE RIES
 </b>
 <sup>
  1
 </sup>
 , GUSTAVO H. GOLDMAN
 <sup>
  1,2
 </sup>
 . 1) Faculty of Pharmacy, University of Sao Paulo, RIBEIRAO PRETO, Sao Paulo, Brazil; 2) National Center of Bioethanol Sciences and Technology (CTBE), CAMPINAS, Sao Paulo, Brazil
 <br/>
 <p>
  Despite recent advances in the production of biofuels from plant biomass, readily metabolisable sugars such as glucose, released during fungal enzymatic hydrolysis of lignocellulose, still impede cellulase and hemicellulase production. In filamentous fungi, such as
  <em>
   Aspergillus nidulans
  </em>
  , glucose-mediated carbon catabolite repression (CCR) is carried out by the transcription factor CreA. Although several studies have described in detail the regulatory effects of CreA on the expression of cellulase- and hemicellulase-encoding genes, regulation of the protein itself as well as characterisation of the different CreA protein domains remains largely unknown. The aim of this study was therefore to investigate in detail the role of CreA in CCR.
 </p>
 <p>
  Firstly, it was determined that CreA does not require
  <em>
   de novo
  </em>
  protein synthesis and is imported into the nucleus from a preformed cytoplasmic pool. Tagging CreA with the
  <em>
   luciferase
  </em>
  gene confirmed that this transcription factor is always present within the cell even in the presence of different lignocellulosic components. Deletion of four different domains of CreA resulted in strains being unable to de-repress under cellulase and hemicellulase-inducing conditions. Furthermore CreA was unable to leave the nucleus in the same conditions. One of the CreA protein regions was shown to be important for germination on cellulose and to sustain growth on complex carbon sources, ethanol and amino acids.
 </p>
 <p>
  For the first time, CreA was shown to be always available and not completely degraded within the cell. Different domains are important for CreA nuclear localisation and one CreA protein domains appears to be important for mediating growth on a wide range of carbon and nitrogen sources.
 </p>
 <p>
  Financial support:  Fundação de Amparo a Pesquisa do Estado de São Paulo (FAPESP) and Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410727
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Metabolism    Gene Expression    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Fungal cell factories for the production of biochemicals in biorefineries.
 <br/>
 <b>
  L. Yang
 </b>
 , M. Lübeck, P. Lübeck. Aalborg University, Copenhagen, Copenhagen SV, Denmark
 <br/>
 <p>
  Currently there is a growing demand for sustainable production of biochemicals that substitute fossil based chemicals. Filamentous fungi are of great interests as biocatalysts in biorefineries as they naturally produce and secrete a variety of different organic acids that can be used as building blocks in the chemical industry. Aspergilli, as biotechnology workhorses, have great potentials as cell factories for production of organic acids. Strains of the black
  <em>
   Aspergillus carbonarius
  </em>
  naturally produce citric acid and gluconic acid in high amounts and have vast abilities in utilizing a broad range of substrates. The fungus has excellent tolerance to stress conditions and therefore is considered as a potential biocatalyst that could be used in lignocellulosic biorefineries. Ideally, by utilizing both its large potentials for secretion of hydrolytic enzymes and of organic acids, the fungus could be considered in a consolidated approach where it hydrolyses the plant biomasses and ferments the resulting sugars into different organic acids. However, for developing the fungus into an efficient biocatalyst for biochemical production, it is necessary to include metabolic engineering of biochemical pathways for increasing the glucose and xylose uptake and flux, and direct the carbon towards production of the selected organic acids. In our project, engineering of selected genes in the glycolytic pathway and in the pentose phosphate pathway have led to increased citric acid production. Furthermore, the effects of deleting the gluconic acid producing pathway and inserting an alternative cytosolic pathway on organic acid production were also evaluated. The impact of these genetic modifications on organic acid production will be presented.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410202
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Metabolism    Other (Other Keyword: Aspergillus terreus)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Manganese ion deficiency plays a pivotal role in the itaconic acid production of
 <em>
  Aspergillus terreus
 </em>
 .
 <br/>
 Rafael Díaz
 <sup>
  1
 </sup>
 , Benedek Papp
 <sup>
  2
 </sup>
 , Erzsébet Fekete
 <sup>
  1
 </sup>
 , Dávid Andrási
 <sup>
  2
 </sup>
 , Erzsébet Sándor
 <sup>
  2
 </sup>
 , Christian P. Kubicek
 <sup>
  3
 </sup>
 ,
 <b>
  Levente Karaffa
 </b>
 <sup>
  1
 </sup>
 . 1) Biochemical Engineering, University of Debrecen, Debrecen, Hungary; 2) Institute of Food Science, University of Debrecen, H-4032 Debrecen, Hungary; 3) Austrian Centre of Industrial Biotechnology, A-8010 Graz, Austria
 <br/>
 <p>
  Itaconic acid (IA) is an unsaturated dicarboxylic acid. The conjunction of the two carboxyl groups and the methylene group that is able to participate in polymerization reactions makes it a potential new platform chemical derived from sugars.
 </p>
 <p>
  IA is mostly produced by large-scale submerged fermentation by
  <em>
   Aspergillus terreus
  </em>
  . Although the biochemical pathway and the physiology leading to IA is almost the same as that leading to citric acid in
  <em>
   A. niger
  </em>
  , both the volumetric (g/L) and the specific yield (g/g carbon source) of IA are by far lower than for citric acid. Citric acid is known to accumulate to high levels only when a number of nutritional parameters are carefully adjusted, but these are not used in research on IA production. Two of those parameters are the concentration of the carbon source (D-glucose) and the concentration of Mn ions in the medium. We have here investigated the effect of variation in these parameters on IA production by
  <em>
   A. terreus
  </em>
  : we show that Mn (II) concentrations above 3 ppb decrease IA production in a concentration-dependent manner, with 1.000 ppb resulting in less than 40 percent of the volumetric yield achieved under identical conditions. We also provide evidence that increasing the D-glucose concentration increases the specific yield of IA. Both findings are in agreement with the effect of these parameters on citric acid production by
  <em>
   A. niger
  </em>
  , thus showing that the transfer of its fermentation technology to
  <em>
   A. terreus
  </em>
  and IA production can be used to arrive at high yields of this acid.
 </p>
 <p>
  Acknowledgement: This research was supported by the EU and co-financed by the European Social Fund under the project ENVIKUT (TAMOP-4.2.2.A-11/1/KONV-2012-0043), by the OTKA (K1006600) and by the QuantFung Project (FP7, Proposal Nr. 607332). LK is a recipient of a Bólyai János Research Scholarship.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410750
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    RNA-seq    Transcriptome
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Large-scale transcriptional response to hypoxia in
 <em>
  Aspergillus fumigatus
 </em>
 observed using RNAseq identifies a novel hypoxia regulated ncRNA
 <em>
  .
 </em>
 .
 <br/>
 <b>
  Liliana Losada
 </b>
 <sup>
  1
 </sup>
 , B Baker
 <sup>
  2
 </sup>
 , S Pakala
 <sup>
  1
 </sup>
 , S Pakala
 <sup>
  1
 </sup>
 , V Joardar
 <sup>
  1
 </sup>
 , N Zafar
 <sup>
  1
 </sup>
 , S Mounaud
 <sup>
  1
 </sup>
 , W Nierman
 <sup>
  1
 </sup>
 , R Cramer
 <sup>
  3
 </sup>
 . 1) JCVI, Rocville, MD; 2) TGen, Flagstaff, AZ; 3) The Geisel School of Medicine at Dartmouth, Hanover, NH
 <br/>
 <p>
  We utilized RNAseq analysis of the
  <em>
   Aspergillus fumigatus
  </em>
  response to early hypoxic condition exposure. The results show that more than 89% of the
  <em>
   A. fumigatus
  </em>
  genome is expressed under normoxic and hypoxic conditions. Replicate samples were highly reproducible; however, comparisons between normoxia and hypoxia revealed that greater than 23% and 35% of genes were differentially expressed after 30 and 120 minutes of hypoxia exposure, respectively. Consistent with our previous report detailing transcriptomic and proteomic responses at later time points, the results here show major repression of ribosomal function and induction of ergosterol biosynthesis, as well as activation of alternate respiratory mechanisms at the later time point. RNAseq data was used to define thirty-two hypoxia-specific genes, which are not expressed under normoxic conditions. Transcripts of a C6 transcription factor and a histidine kinase response regulator were found only in hypoxia. In addition, several genes involved in the phosphoenyl pyruvate (PEP) and D-glyceraldehyde-3-phosphate (G3P) metabolism were only expressed in hypoxia. Interestingly, a 216 bp ncRNA Afu-182 in the 3&#8217; region of
  <em>
   insA
  </em>
  (AFUB_064770), was significantly repressed under hypoxia with a 40-fold reduction in expression. A detailed analysis of Afu-182 showed similarity with several genes in the genome, many of which were also repressed in hypoxia. The results from this study show that hypoxia induces very early and widely drastic genome-wide responses in
  <em>
   A. fumigatus
  </em>
  that include expression of protein-coding and ncRNA genes. The role of these ncRNA genes in regulating the fungal hypoxia response is an exciting future research direction. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410278
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Gene Expression    Signal Transduction
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Golgi-localized and the palmitoyl transferase-related AkrA homologs mediates [Ca
 <sup>
  2+
 </sup>
 ]
 <sub>
  i
 </sub>
 transient to response ER and azole stresses.
 <br/>
 Yuanwei Zhang
 <sup>
  1
 </sup>
 , Qingqing Zhen
 <sup>
  1
 </sup>
 , Jinxing Song
 <sup>
  1
 </sup>
 , Lina Gao
 <sup>
  1
 </sup>
 , Alberto Muñoz
 <sup>
  2
 </sup>
 , Nick D. Read
 <sup>
  2
 </sup>
 ,
 <b>
  Ling Lu
 </b>
 <sup>
  1
 </sup>
 . 1) College of Life Science, Nanjing Normal Univeristy, Nanjing, jiangsu, China; 2) Manchester Fungal Infection Group, Institute of Inflammation and Repair, CTF Building, University of Manchester, Manchester M13 9NT, UK
 <br/>
 <p>
  Finely tuned [Ca
  <sup>
   2+
  </sup>
  ]
  <sub>
   i
  </sub>
  changes mediate several intracellular functions, resulting in subsequent activation or inactivation of a series of conserved Ca
  <sup>
   2+
  </sup>
  signaling components and their target proteins. Palmitoylation is a reversible post-translational modification involved in membrane protein trafficking and functional modulation. However, studies on the relationship between calcium signaling and palmitoylation have been limited. Here, we demonstrate that the homologs of yeast palmitoyl transferase ScAkr1p, AkrA in
  <em>
   Aspergillus nidulans
  </em>
  and SidR in
  <em>
   Aspergillus fumigatus
  </em>
  , play important roles under low calcium conditions. Deletion of
  <em>
   akrA
  </em>
  or
  <em>
   sidR
  </em>
  shows remarkable defects in hyphal growth and conidiation, but adding extracellular calcium can completely rescue the growth defects. Moreover, using the calcium probe aequorin in live cells, we found that all of the palmitoyl transferase-related
  <em>
   akrA
  </em>
  mutants induced larger decreases in the [Ca
  <sup>
   2+
  </sup>
  ]
  <sub>
   i
  </sub>
  response to extracellular Ca
  <sup>
   2+
  </sup>
  compared to the previously identified high-affinity calcium influx system members (CchA and MidA) and compared to the parent control strain. Moreover, ER stressors- or azole-induced calcium transient was completely blocked by AkrA defects, especially in low calcium conditions where we did not detect a calcium transient. Interestingly, all of the above-described functions AkrA are tightly related to cysteine residues in its DHHC-CRD and its palmitoyl transferase activity. Thus, Golgi-localized AkrA mediates the [Ca
  <sup>
   2+
  </sup>
  ]
  <sub>
   i
  </sub>
  transient likely by globally palmitoylating calcium signaling components and their target proteins. Our findings provide insight into a new link between calcium signaling and palmitoylation in the regulation of cell survival processes upon ER and membrane stress. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410088
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Transport    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 AmcA - a mitochondrial ornithine transporter involved in fungal siderophore biosynthesis.
 <br/>
 <b>
  Lukas Schafferer
 </b>
 <sup>
  1
 </sup>
 , Nicola Beckmann
 <sup>
  1
 </sup>
 , Ulrike Binder
 <sup>
  2
 </sup>
 , Hubertus Haas
 <sup>
  1
 </sup>
 . 1) Division of Molecular Biology, Innsbruck Medical University, Innsbruck, Austria; 2) Division of Hygiene and Medical Microbiology, Innsbruck Medical University, Innsbruck, Austria
 <br/>
 <p>
  Iron is an essential nutrient required for a wide range of cellular processes. The opportunistic fungal pathogen
  <em>
   Aspergillus fumigatus
  </em>
  employs low-molecular mass iron-specific chelators, termed siderophores, for uptake, storage and intracellular iron distribution, which play a crucial role in the pathogenicity of this fungus. Siderophore biosynthesis depends on coordination with the supply of its precursor ornithine, produced mitochondrially from glutamate or cytosolically via hydrolysis of arginine. In this study, we demonstrate a role of the mitochondrial transporter AmcA (AFUA_8g02760) in siderophore biosynthesis by
  <em>
   A. fumigatus.
  </em>
 </p>
 <p>
  Consistent with a role in mitochondrial ornithine export, AmcA-deficiency resulted in decreased cellular ornithine and arginine contents as well as decreased siderophore production on glutamine as the sole nitrogen source. In support, arginine and ornithine as nitrogen sources did not impact siderophore biosynthesis due to cytosolic ornithine availability. As revealed by Northern blot analysis, transcript levels of siderophore biosynthetic genes were unresponsive to the cellular ornithine level. In contrast to siderophore production, AmcA deficiency did not alter the cellular content of polyamines, demonstrating cellular prioritization of ornithine use. Nevertheless, AmcA-deficiency increased the susceptibility of
  <em>
   A. fumigatus
  </em>
  to the polyamine biosynthesis inhibitor eflornithine, most likely due to the decreased ornithine pool. AmcA-deficiency decreased the growth rate particularly on ornithine as the sole nitrogen source during iron starvation and sufficiency, indicating an additional role in the metabolism and fitness of
  <em>
   A. fumigatus
  </em>
  , possibly in mitochondrial ornithine import. In the
  <em>
   Galleria mellonella
  </em>
  larvae infection model, AmcA-deficiency did not affect virulence of
  <em>
   A. fumigatus,
  </em>
  most likely due to the residual siderophore production and arginine availability in the host niches.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410403
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Oxidative Stress    RNA-seq    Secondary metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 How menadione triggers aflatoxin synthesis in
 <em>
  A. flavus
 </em>
 .
 <br/>
 <b>
  M. Zaccaria
 </b>
 <sup>
  1
 </sup>
 , M. Ludovici
 <sup>
  1
 </sup>
 , M. Scarpari
 <sup>
  1
 </sup>
 , V. Scala
 <sup>
  1
 </sup>
 , A.A. Fabbri
 <sup>
  1
 </sup>
 , C. Fanelli
 <sup>
  1
 </sup>
 , W. Sanseverino
 <sup>
  2
 </sup>
 , M. Reverberi
 <sup>
  1
 </sup>
 . 1) Environmental Biology, University of Rome , Roma, Roma, Italy; 2) Sequentia Biotech, carrer comte d'Urgell, Barcelona, Spain
 <br/>
 <p>
  <em>
   Aspergillus flavus
  </em>
  is a saprophytic fungus responsible for world-wide spread harvest and post-harvest infections on cultivations, mainly on cereal grains and legumes
  <em>
   . A. flavus
  </em>
  is one of the main producers of aflatoxin B1, the most dangerous carcinogenic metabolite in nature. In relation to this, ongoing climate changes favor plant susceptibility to the attack by this fungus with a consequent, dangerous increase of aflatoxins into previously unexploited feed and foodstuff. In order to address effectively the economic and sanitary consequences of
  <em>
   A. flavus
  </em>
  contamination, a detailed and extensive knowledge of the pathogen metabolism and of the environmental conditions triggering the different biological processes, is of paramount importance.
 </p>
 <p>
  In this study, we focus on the effects of oxidative stress in the intracellular compartment, obtained by adding menadione 0,1mM to the culture medium. Menadione is a chinone and a precursor of vitamin K, its cytotoxicity has been extensively investigated in several human and murine cellular lines as an intracellular ROS inductor. Chinones are very common in nature. As substrates for flavoenzymes they may incur in one electron reduction to semichinone which, conversely, reduce O
  <sub>
   2
  </sub>
  to superoxide anion in the intracellular environment, therefore providing a stressing condition. We evaluate the response from
  <em>
   A
  </em>
  .
  <em>
   flavus
  </em>
  <em>
   via
  </em>
  several analytical approaches: mycelial growth, conidia quantification, aflatoxin B1 synthesis, antioxidant enzymes activity, intracellular ROS quantification. To evaluate gene expression, we exploit RNAseq technology for transcriptome analysis, plus RT-PCR of markers for cellular respiration, pentose phosphate pathway, and oxidative stress response and sirtuins expression. Lastly, we have extracted and evaluated oxylipins by an MRM based LC-MS/MS method, in order to ascertain if they may represent a more stable reactive signal able to trigger aflatoxin synthesis and conidiogenesis through a remodulation of
  <em>
   A. flavus
  </em>
  metabolism in oxidative stress conditions.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410783
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Population and Evolutionary Genetics
 </b>
 <br/>
 Keyword(s):
 <b>
  Phylogenetic    Population    Other (Other Keyword: Histoplasma)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Reevaluating the worldwide phylogenetic distribution and population structure of Histoplasma capsulatum based on four genetic markers.
 <br/>
 <b>
  Marcus de Melo Teixeira
 </b>
 <sup>
  1,8
 </sup>
 , José Salvatore Leister Patane
 <sup>
  2
 </sup>
 , Raquel Cordeiro Theodoro
 <sup>
  3
 </sup>
 , Maria Lucia Taylor
 <sup>
  4
 </sup>
 , Eduardo Bagagli
 <sup>
  5
 </sup>
 , Rosely Maria Zancope-Oliveira
 <sup>
  6
 </sup>
 , Maria Sueli Soares Felipe
 <sup>
  1,7
 </sup>
 . 1) Universidade de Brasília, Brasília-DF, Brazil; 2) Butantan Institute, São Paulo-SP, Brazil; 3) Universidade do Rio Grande do Norte-RN, Natal, Brazil; 4) Universidad Nacional Autónoma de México, Mexico city-DF, Mexico; 5) Universidade Estadual Julio de Mesquita Filho (UNESP), Botucatu-SP, Brazil; 6) Fundação Oswaldo Cruz (FIOCRUZ), Rio de Janeiro-RJ, Brazil; 7) Universidade Católica de Brasília, Brasília-DF, Brazil; 8) Translational Genomics Institute (Tgen), Flasgstaff-AZ, USA
 <br/>
 <p>
  <em>
   Histoplasma capsulatum
  </em>
  (Darling, 1906), comprises a worldwide complex of saprophyte fungi mainly found in nitrogen/phosphate (often bird guano) enriched soils.
  <em>
   Histoplasma ssp.
  </em>
  microconidia may be inhaled by mammalian hosts, conditioning a prompt conversion to yeasts cells that can persist in host tissues causing Histoplasmosis; a deep pulmonary/systemic mycosis. Accordingly to hallmark publication of Kasuga et al., 2003,
  <em>
   Histoplasma capsulatum
  </em>
  &#8220;
  <em>
   sensu stricto&#8221;
  </em>
  comprises a complex of at least eight phylogenetic species geographically distributed as follows: Australian, Dutch, Eurasian, North American class 1 (NAm 1), North American class 2 (NAm 2), Latin American group A (LAm A), Latin American group B (LAm B) and Africa populations. With the increasing of taxa sampling (n=234), a new phylogenetic distribution of
  <em>
   Histoplasma capsulatum
  </em>
  was revised thought 1,563 aligned nucleotides (arf, ole1, tub and anti-H loci). Accordingly, the African population was clustered in at least 3 different populations. The LAm B clade appears to be divided in three highly supported clades which are geographically restricted to Colombia, Brazil and Argentina populations. Moreover, a lower genetic structure was identified in basal clades of LAm A followed by expansions of highly supported clonal branches, which could be driven by fast host/environmental adaptation. These studies open new (old?) evolutionary questions regarding the
  <em>
   Histoplasma
  </em>
  ssp. speciation mainly in Latin American populations, as well to be further considered for population genomics studies.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410607
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Metabolism    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 A biocombinatorial engineering approach for production of novel synthetic natural products.
 <br/>
 <b>
  Maria Lund Nielsen
 </b>
 , Jakob B. Nielsen, Uffe H. Mortensen, Mikael R. Andersen, Thomas O. Larsen. Systems Biology, DTU, Kgs. Lyngby, Denmark
 <br/>
 <p>
  Fungal secondary metabolism is the source of a large number of structurally diverse natural products that hold a wide variety of biological activities. Many of these compounds are known to possess activities of medical relevance, such as anti-bacterial-, anti-cancer-, cholesterol-lowering- and immunosuppressive. The need for new bioactive compounds calls for an approach to further expand the diversity of fungal secondary metabolites. One such approach is the biocombinatorial synthesis of novel natural products through the engineering of proteins involved in secondary metabolism. In particular, engineering of polyketide synthases (PKSs), non-ribosomal peptide synthetases (NRPSs) and PKS-NRPS hybrid proteins has been attempted; however, so far this approach has obtained only limited success. In order to fully exploit the possibilities of combinatorial biosynthesis we first need to understand the rules governing product formation. Such understanding would enable the rational design of functional chimeric enzymes and permit the synthesis of derivatives of known natural products. The PKS-NRPS hybrids have received particular interest as targets for combinatorial biosynthesis due to the potential of creating novel compounds of mixed polyketide-non-ribosomal peptide origin. One critical step in designing functional PKS-NRPS fusions is undoubtedly the determination of the optimal site for linking the two parts of the chimera.
 </p>
 <p>
  In this study we focused on the PKS-NRPS from
  <em>
   Aspergillus clavatus
  </em>
  involved in the synthesis of the potential anti-cancer compound cytochalasin E. This enzyme appears to contain an intermodular linker of approximately 150 amino acids with very low sequence homology to other known PKS-NRPS hybrids. We investigated the role of this linker by constructing a number of linker-modified variants of the
  <em>
   A. clavatus
  </em>
  PKS-NRPS hybrid, including linker swaps, linker truncations and synthetic linkers. These experiments will help to unveil the importance of the linker, and facilitate determination of the optimal length of the intermodular linker, along with a suitable site for linking the two modules.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410256
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Cell Wall    Signal Transduction
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Sensing and responding to cell wall stress in Aspergillus niger requires at least three transcription factors - RlmA, MsnA and CrzA.
 <br/>
 <b>
  Markus RM Fiedler
 </b>
 <sup>
  1
 </sup>
 , Annett Lorenz
 <sup>
  1
 </sup>
 , Benjamin M Nitsche
 <sup>
  1
 </sup>
 , Mark Arentshorst
 <sup>
  2,4
 </sup>
 , Cees AMJJ van den Hondel
 <sup>
  3
 </sup>
 , Arthur FJ Ram
 <sup>
  2,4
 </sup>
 , Vera Meyer
 <sup>
  1,4
 </sup>
 . 1) Applied and Molecular Microbiology, Institute of Biotechnology, Berlin, Germany; 2) Institute of Biology Leiden, Molecular Microbiology and Biotechnology, Sylviusweg 72, 2333 BE Leiden, The Netherlands; 3) HiTeXacoat, Waterlelie 124, 2804 PZ Gouda, The Netherlands; 4) Kluyver Centre for Genomics of Industrial Fermentation, P.O. Box 5057, 2600 GA Delft, The Netherlands
 <br/>
 Cell wall integrity, vesicle transport and protein secretion are key factors contributing to the vitality and productivity of filamentous fungal cell factories such as Aspergillus niger. Transcriptomics signatures of A. niger and phenotypic analyses of selected null mutant strains were used to predict regulator proteins mediating the survival responses against compounds interfering with the cell wall integrity including caspofungin, aureobasidin A, fenpropimorph and FK506. This integrated approach allowed us to reconstruct a model for the cell wall salvage gene network of A. niger that ensures survival of the fungus upon cell surface stress. The model predicts that (i) caspofungin and aureobasidin A induce the cell wall integrity pathway as a main compensatory response via induction of the Rho-GTPases RhoB and RhoD, respectively, eventually activating the mitogen-activated protein kinase kinase MkkA and the transcription factor RlmA. (ii) RlmA is the main transcription factor required for the protection against calcofluor white but it cooperates with MsnA and CrzA to ensure survival of A. niger when challenged with caspofungin or aureobasidin A. (iii) Membrane stress provoked by aureobasidin A via disturbance of sphingolipid synthesis induces cell wall stress, whereas fenpropimorph-induced disturbance of ergosterol synthesis does not. The present work uncovered a sophisticated defence system of A. niger which employs at least three transcription factors - RlmA, MsnA and CrzA - to protect itself against cell wall stress. The transcriptomic data furthermore predicts a fourth transfactor, SrbA, which seems to be specifically important to survive fenpropimorph-induced cell membrane stress.
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410566
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Gene Expression    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Biological role and characterization of aegerolysins and proteins with MACPF domain in filamentous fungus
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  Marusa Novak
 </b>
 <sup>
  1
 </sup>
 , Urska Cepin
 <sup>
  2,3
 </sup>
 , Nada Krasevec
 <sup>
  4
 </sup>
 , Peter Macek
 <sup>
  1
 </sup>
 , Gregor Anderluh
 <sup>
  4
 </sup>
 , Kristina Sepcic
 <sup>
  1
 </sup>
 . 1) University of Ljubljana, Biotechnical Faculty, Department of Biology, Ljubljana, Slovenia; 2) National Institute of Biology, Ljubljana, Slovenia; 3) BioSistemika Ltd, Ljubljana, Slovenia; 4) L11, National Institute of Chemistry, Ljubljana, Slovenia
 <br/>
 <p>
  Aegerolysins and MACPF domain-containing proteins (Pfam06355 and 01823 protein families, respectively) are found in various kingdoms of life including fungi. In Basidiomycota, proteins of these two families seem to be involved in development of primordia and fruiting bodies, while in filamentous fungi they can also act as virulence factors. Various fungal members of both protein families have been shown to form pores in biological and artificial lipid membranes, either sole or in combination with one another. It appears that the roles of these proteins are pleiotropic and adapted to the fungal life-style.
 </p>
 <p>
  <em>
   Aspergillus niger
  </em>
  is a saprophytic, filamentous fungus found throughout the world. In its genome, we identified two nucleotide sequences encoding aegerolysins and two nucleotide sequences encoding proteins with MACPF domain. Our aim is to determine the biological role(s) and some characteristics of these proteins in
  <em>
   A. niger
  </em>
  using systematic gene expression studies, gene deletions and protein labeling. So far, we showed that the increase of the expression of all four target genes coincides with the beginning of conidiation in
  <em>
   A. niger
  </em>
  , and that the prevention of conidiation (either physical or genetic) alters the expression profiles and negatively affects their expression. Deletion of either of the aegerolysin genes did not affect the rate of conidiation or growth on different media. Moreover, the localization studies using fluorescently labeled proteins showed proteins to be localized in hyphae, conidiophores heads and spores. Our results suggest that aegerolysins and MACPF domain-containing proteins are produced during conidiation of
  <em>
   A. niger
  </em>
  , but are not actively involved in this process, indicating that their role(s) might be related to some other physiological processes in the fungus, e.g. defense mechanisms, rather than development.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410522
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolism    Aspergillus oryzae    RNA-seq
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 A software tool for visual analysis of secondary metabolism gene clusters.
 <br/>
 Akira Ohyama
 <sup>
  1,4
 </sup>
 , Myco Umemura
 <sup>
  2,4
 </sup>
 , Itaru Takeda
 <sup>
  3
 </sup>
 , Takashi Kubo
 <sup>
  4
 </sup>
 , Makoto Matsui
 <sup>
  4
 </sup>
 , Yuki Miyamura
 <sup>
  4
 </sup>
 , Kaoru Nemoto
 <sup>
  4
 </sup>
 , Tomoko Ishii
 <sup>
  4
 </sup>
 ,
 <b>
  Masayuki Machida
 </b>
 <sup>
  2,4
 </sup>
 . 1) in silico biology, inc., Yokohama, Japan; 2) National Institute of Advanced Industrial Science and Technology (AIST), Sapporo/Tsukuba, Japan; 3) Tokyo University of Agriculture and Technology, Tokyo, Japan; 4) Technology Research Association of Highly Efficient Gene Design, Tokyo, Japan
 <br/>
 <p>
  We developed software tools for the motif-independent prediction of secondary metabolism biosynthesis (SMB) gene clusters, MIDDAS-M
  <sup>
   1
  </sup>
  and MIPS-CG
  <sup>
   2
  </sup>
  . Recently, MIDDAS-M successfully detected the gene cluster for the biosynthesis of ustiloxin B, a circular peptide compound which possesses N-methylation and a norvaline moiety. This was the first finding of RiPS (Ribosomal Peptide Synthesis) pathway from filamentous fungi
  <sup>
   3
  </sup>
  . Once SMB gene cluster are predicted, their accuracy and function are to be experimentally evaluated by mobilizing knowledge of SMB genes, especially those other than so-called &#8220;Core Genes&#8221; such as PKS and NRPS especially for those of novel SMB pathways such as RiPS. We have developed in silico MolecularClonig Genome Design Suite (IMCDS) to effectively design the validation experiments. IMCDS allows visual analysis and comparison of predicted and known SMB gene clusters under multiple representation capability including gene expression from genome-wide to nucleotide sequence views. It includes automatic color visualization of genes based on their functional categories and designing primers for the preparation of a DNA fragment to disrupt a target gene.
 </p>
 <p>
  1. Umemura et al., PLoS One 8, e84028 (2013).
 </p>
 <p>
  2. Takeda et al., DNA Res (2014).
 </p>
 <p>
  3. Umemura et al., Fungal Genet Biol (2014).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410433
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Metabolism    Transport    Aspergillus niger
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Transport proteins for itaconic acid production in
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  Matthias G. Steiger
 </b>
 <sup>
  1,2,4
 </sup>
 , Peter J. Punt
 <sup>
  2,3
 </sup>
 , Arthur F. J. Ram
 <sup>
  2
 </sup>
 , Diethard Mattanovich
 <sup>
  1,4
 </sup>
 , Michael Sauer
 <sup>
  1,4
 </sup>
 . 1) Austrian Centre of Industrial Biotechnology (ACIB GmbH), Muthgasse 11, 1190 Vienna, Austria; 2) University Leiden, Institute of Biology, Sylviusweg 72, 2333 BE Leiden, The Netherlands; 3) TNO Microbiology and Systems biology, Zeist, AJ 3700, The Netherlands; 4) University of Natural Resources and Life Sciences, Department of Biotechnology, Muthgasse 18, 1190 Vienna, Austria.
 <br/>
 <p>
  <em>
   Aspergillus niger
  </em>
  is a well-established host organism for the production of carboxylic acids. Acids like citric, gluconic and oxalic acids can be produced and high titers are obtained. The formation of carboxylic acids involves the shuttling of intermediate metabolites between different intracellular compartments and utilizes different enzymatic capabilities of the respective compartment. The knowledge about the involved shuttling mechanisms and the localization of the necessary enzymes is still fragmentary.
 </p>
 <p>
  In order to analyze the influence of compartmentalization on organic acid production, we have chosen itaconic acid as a target substance. Itaconic acid, which was selected by the US Department of Energy as one of the 12 building block chemicals for the industrial biotechnology, is currently produced by
  <em>
   A. terreus
  </em>
  . Heterologous expression of
  <em>
   A. terreus cadA
  </em>
  gene enables the formation of itaconic acid in
  <em>
   A. niger
  </em>
  although only low titers are obtained. An increase of the productivity was obtained by targeting the pathway to the mitochondria. Furthermore, it was shown that the heterologous expression of two transport proteins which are found in close proximity to the
  <em>
   cadA
  </em>
  gene in
  <em>
   A. terreus
  </em>
  , have a positive impact on the itaconic acid formation.
 </p>
 <p>
  These two transport proteins are now characterized in more detail making use of an inducible promoter system. Their localization in the cell is determined tagging the proteins with GFP and fluorescence microscopy. Furthermore, the substrate specificity of the mitochondrial transporter is elucidated by a rearrangement of synthetic metabolic pathways. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410167
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Gene Expression    Proteomics    Diversity
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 A genomic survey of proteases in Aspergilli.
 <br/>
 <b>
  Miaomiao Zhou
 </b>
 <sup>
  1,3
 </sup>
 , Sebnem Ozturkoglu Budak
 <sup>
  1,2,3
 </sup>
 , Carlo Brouwer
 <sup>
  1
 </sup>
 , Ad Wiebenga
 <sup>
  1,3
 </sup>
 , Isabelle Benoit
 <sup>
  1,3
 </sup>
 , Marcos Di Falco
 <sup>
  4
 </sup>
 , Adrian Tsang
 <sup>
  4
 </sup>
 , Ronald de Vries
 <sup>
  1,3
 </sup>
 . 1) CBS-KNAW, Utrecht, Netherlands; 2) Faculty of Agriculture, Department of Dairy Technology, University of Ankara, Ankara, Turkey; 3) Fungal Molecular Physiology, Utrecht University, Utrecht, The Netherlands; 4) Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, QC H4B 1R6 Canada
 <br/>
 <p>
  Proteases can hydrolyze peptides in aqueous environments. This property has made proteases the most important industrial enzymes by taking up about 60% of the total enzyme market. Microorganisms are the main sources for industrial protease production due to their high yield and a wide range of biochemical properties. Several Aspergilli have the ability to produce a variety of proteases, but no comprehensive comparative study has been carried out on protease productivity in this genus so far.
 </p>
 <p>
  We have performed a combined analysis of comparative genomics, proteomics and enzymology tests on seven
  <em>
   Aspergillus
  </em>
  species grown on wheat bran and sugar beet pulp. Putative proteases were identified by homology search and Pfam domains. These genes were then clusters based on orthology and extracellular proteases were identified by protein subcellular localization prediction. Proteomics was used to identify the secreted enzymes in the cultures, while protease essays with and without inhibitors were performed to determine the overall protease activity per protease class. All this data was then integrated to compare the protease productivities in Aspergilli.
 </p>
 <p>
  Genomes of
  <em>
   Aspergillus
  </em>
  species contain a similar proportion of protease encoding genes. According to comparative genomics, proteomics and enzymatic experiments serine proteases make up the largest group in the protease spectrum across the species. In general wheat bran gives higher induction of proteases than sugar beet pulp. Interesting differences of protease activity, extracellular enzyme spectrum composition, protein occurrence and abundance were identified for species. By combining
  <em>
   in silico
  </em>
  and wet-lab experiments, we present the intriguing variety of protease productivity in Aspergilli.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410581
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Biomass    Other (Other Keyword: Aspergillus, feruloyl esterase)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Heterologous expression of feruloyl esterases of
 <em>
  Aspergillus clavatus
 </em>
 and
 <em>
  Aspergillus terreus
 </em>
 .
 <br/>
 <b>
  Miia Mäkelä
 </b>
 , Salla Koskela, Kristiina Hildén. Department of Food and Environmental Sciences, University of Helsinki, Helsinki, Finland
 <br/>
 <p>
  In the cell walls of gramineous plants, hemicelluloses are crosslinked to the aromatic lignin polymer via hydroxycinnamic acids (ferulic acid and
  <em>
   p
  </em>
  -coumaric acid). Feruloyl esterases (ferulic acid esterases, EC 3.1.1.73), classified in CAZy family CE1 (www.cazy.org), are enzymes that catalyse the cleavage of covalent ester bonds between carbohydrate and lignin moieties in plant cell walls. Due to the ability to specifically cleave ester linkages, feruloyl esterases are promising biocatalysts for a broad range of biotechnological applications. These include e.g. pharmaceutical, agricultural and food industries, as well as the production of biofuel.
 </p>
 <p>
  <em>
   Aspergillus
  </em>
  species are one of the best studied fungi, largely due to their applicability in biotechnology and relevance in human health.  In this study, putative feruloyl esterase encoding genes from
  <em>
   Aspergillus clavatus
  </em>
  and
  <em>
   Aspergillus terreus
  </em>
  were cloned and expressed heterologously in the methylotrophic yeast
  <em>
   Pichia pastoris
  </em>
  . Biochemical properties, including substrate specificity and thermotolerance of the recombinant
  <em>
   A. clavatus
  </em>
  and
  <em>
   A. terreus
  </em>
  feruloyl esterases will be presented. In addition, the ability of the recombinant feruloyl esterases to hydrolyze different plant biomass based substrates will be evaluated.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410386
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Secondary metabolite    Transcriptome
 </b>
 <br/>
 Presentation Preference:
 <b>
  Invited
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Discovery of fungal secondary metabolic pathways from large-scale genomic and transcriptome information.
 <br/>
 <b>
  M. Umemura
 </b>
 <sup>
  1
 </sup>
 , N. Nagano
 <sup>
  2
 </sup>
 , H. Koike
 <sup>
  1
 </sup>
 , K. Tamano
 <sup>
  1
 </sup>
 , K. Abe
 <sup>
  3
 </sup>
 , K. Shin-ya
 <sup>
  4
 </sup>
 , K. Asai
 <sup>
  2
 </sup>
 , M. Machida
 <sup>
  1
 </sup>
 . 1) Bioproduction Res. Inst., AIST, Sapporo, Japan; 2) Computational Biol. Res. Center, AIST, Tokyo, Japan; 3) Grad. Sch. Agricultural Sci., Tohoku Univ., Sendai, Japan; 4) Biomedical Res. Inst., AIST, Tokyo, Japan
 <br/>
 <p>
  Fungi produce numerous secondary metabolites, many of which are bioactive and valuable for medicinal uses. Major biosynthetic pathways responsible for the synthesis of their main skeletal structures are polyketide synthases (PKSs), non-ribosomal peptide synthetases (NRPSs) and terpene cyclases, which are detectable in genome sequences owing to their sequence motifs. Other unknown types of secondary metabolite biosynthetic (SMB) pathways can even still exist, because it is just a fraction of the metabolites whose biosynthetic pathways are elucidated in filamentous fungi. Here, to explore genome information for SMB pathways, a motif-independent method, MIDDAS-M, is presented. The MIDDAS-M algorithm is based on the characteristics that SMB genes are clustered and cooperatively expressed on a fungal genome. It can detect novel types of SMB pathways using only gene-annotated genome information and transcriptome data. Thanks to this method, the fungal secondary metabolite, ustiloxin B, turned out to be produced by a ribosomal peptide synthetic (RiPS) pathway, which is the first report for filamentous fungi excluding
  <em>
   Amanita
  </em>
  mushroom. This RiPS pathway is quite unique and different from other known bacterial ones because its precursor protein contains 16-fold repeated core peptides that construct the cyclic portion of the compound. Based on the characteristics, other fungal RiPS pathways were searched and two corresponding novel RiPS compounds have been recently identified. In addition to RiPS, unknown but functionally characteristic gene clusters are sharply detected by MIDDAS-M, which might broaden the field of fungal secondary metabolites further.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410022
 </b>
 <br/>
 Research Area:
 <b>
  Population and Evolutionary Genetics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Fruiting Body    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Identification of novel genes regulating sexual reproduction in
 <em>
  Aspergillus
 </em>
 species.
 <br/>
 <b>
  N Salih
 </b>
 , P. Dyer. University of Nottingham, Nottingham, United Kingdom, NG7 2RD
 <br/>
 <p>
  Having an understanding of the reproductive mode of an organism is of great importance for gaining insights into the potential for the evolution of a given species, whilst the sexual cycle also provides a valuable tool for strain improvement of species  used in industrial applications. About one third of
  <em>
   Aspergillus
  </em>
  species are reported to be capable of sexual reproduction, but the majority of species are only known to reproduce asexually. However, sexual cycles have recently been discovered in a number of
  <em>
   Aspergillus
  </em>
  species that were previously thought to be strictly asexual. This has provoked increased research interest both in the possible genetic basis of asexuality and the molecular genetic control of sexual development. Studies are therefore in progress.to identify and characterise novel genes involved in sexual reproduction of
  <em>
   Aspergillus
  </em>
  species, to provide insights into sexual development and identify candidate genes possibly linked to asexuality. In previous work a set of genes had been identified that was differentially expressed in
  <em>
   MAT1-1
  </em>
  and
  <em>
   MAT1-2
  </em>
  mating-type strains of
  <em>
   A. oryzae
  </em>
  , most of which were of unknown function. Homologous gene were identified and then deleted from the homothallic (sexually self fertile)
  <em>
   A. nidulans
  </em>
  by targeted gene replacement in a Delta
  <em>
   nkuA
  </em>
  strain. Transformant strains were tested for sexual fertility to investigate the effect of gene deletion on sexuality. A range of effects was observed, from no obvious impact, to moderate loss or gain of fertility, to complete loss of sexuality. A gene complementation test, restoring the gene of interest, was then performed to confirm that observed major changes in sexuality were due to the deletion of target gene.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410699
 </b>
 <br/>
 Research Area:
 <b>
  Other
 </b>
 <br/>
 Keyword(s):
 <b>
  Fusarium oxysporum    Diversity
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Diversity of
 <em>
  Fusarium oxysporum
 </em>
 f.sp.
 <em>
  cubence
 </em>
 isolated from local banana cultivars in Indonesia.
 <br/>
 <b>
  Nani Maryani
 </b>
 <sup>
  1,2
 </sup>
 , Fajarudin Ahmad
 <sup>
  1,3
 </sup>
 , Sarah Schmidt
 <sup>
  4
 </sup>
 , Muhammad Ilyas
 <sup>
  3
 </sup>
 , Yuyu Poerba
 <sup>
  3
 </sup>
 , Pedro Crous
 <sup>
  5
 </sup>
 , Gert Kema
 <sup>
  1
 </sup>
 . 1) Wageningen University and Research Center, Plant Research International, Netherlands; 2) University of Sultan Ageng Tirtayasa (UNTIRTA) Banten, Indonesia.; 3) Research Centre for Biology, Indonesian Institute of Sciences (LIPI), Cibinong, Indonesia; 4) University of Amsterdam, Netherlands; 5) CBS-KNAW Fungal Biodiversity Center, Utrecht, Netherlands
 <br/>
 <p>
  Fusarium wilt, known as Panama disease, is one of the major constrains in banana cultivation. The disease is caused by the soil-born fungus
  <em>
   Fusarium oxysporum
  </em>
  f.sp.
  <em>
   cubense
  </em>
  (
  <em>
   Foc
  </em>
  ) and has devastated banana plantations in almost every banana-growing country. A new strain of
  <em>
   Foc,
  </em>
  commonly known as Tropical Race 4 (TR4), was first detected and disseminated throughout SE Asia since the 1990s and was recently reported in Jordan, Mozambique, Oman, Pakistan and Lebanon. Indonesia is the center of origin for both wild and cultivated banana that likely have co-evolved with
  <em>
   Foc,
  </em>
  hence we hypothesize a wide
  <em>
   Foc
  </em>
  diversity throughout the country. We have recently established a comprehensive collection of 114
  <em>
   Foc
  </em>
  isolates from the tree main islands of Indonesia: Java, Kalimantan and Sumatra. We isolated
  <em>
   Foc
  </em>
  from the vascular tissue of banana plants showing wilting symptoms. As a preliminary result we morphologically and molecularly characterized &#8211; using specific diagnostics - the obtained collection. Around 65% of the isolates belong to TR4. Further extension of the collection and htp-genotyping will enable us to describe the
  <em>
   Foc
  </em>
  landscape in Indonesia. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410048
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Hyphal Growth    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Invited
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Maintenance of active directional growth by continual assembly and disassembly of polarity sites.
 <br/>
 <b>
  Norio Takeshita
 </b>
 . Dept. of Microbiology, Karlsruhe Institute of Technology, Karlsruhe, Germany
 <br/>
 <p>
  Polar cell extension depends on spatially defined insertion of new materials controlled by a group of cell-end marker proteins that are concentrated at the plasma membrane. In filamentous fungus
  <em>
   Aspergillus nidulans
  </em>
  , hyphal tips extend with speeds of 0.3-1.0 µm/min, requiring high frequency of exocytosis events (400-2000 vesicles/min). Here, the cell-end marker protein TeaR in
  <em>
   A. nidulans
  </em>
  is used to investigate the maintenance of polarized growth in the midst of rapid influx of vesicles by using a combination of fluorescence microscopy, a super-resolution microscopy technique (PALM) and computational modelling. We report that TeaR cluster is spatially dynamic and transiently stable. Accumulated TeaR triggers downstream processes of actin polymerization and active exocytosis, which results in localised cell extension and TeaR dispersion along the membrane. The reestablishment of polarity is driven by microtubules. These findings suggest a cyclic mechanism by which TeaR polarity is assembled and disassembled repeatedly to maintain polarity despite massive membrane flow.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410300
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Development    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Global examination of the molecular roles, localizations and interactomes of F-box proteins in fungal development.
 <br/>
 <b>
  Oezguer Bayram
 </b>
 <sup>
  1
 </sup>
 , Oezlem Sarikaya Bayram
 <sup>
  1
 </sup>
 , Sabine Reen
 <sup>
  2
 </sup>
 , Gustavo Goldman
 <sup>
  3
 </sup>
 , Gerhard Braus
 <sup>
  2
 </sup>
 . 1) Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland; 2) Department of Molecular Microbiology and Genetics, Georg-August University, Göttingen, Germany; 3) Faculdade de Ciências Farmacêuticas de Ribeirão Preto, Universidade de São Paulo, Brazil
 <br/>
 <p>
  Multiprotein complex, Skp-Cul-Fbox (SCF) E3 ubiquitin ligases, are the largest family of E3 ligases that are responsible for marking of target proteins with ubiquitin and subsequent proteasome-dependent degradation. SCF E3 ligases are involved in many cellular processes, including transcription, cell-cycle control by determining protein levels of target proteins. The F-box component of the SCF complex is essential for the substrate specificity of the SCF complex by recruiting target proteins for ubiuitination. In this study, we have systematically investigated the molecular functions of 73 F-box or F-box-like protein encoding genes of the eukaryotic model system
  <em>
   Aspergillus nidulans
  </em>
  . Deletion of 73
  <em>
   fbx
  </em>
  genes revealed that only 8-10 % of the
  <em>
   fbx
  </em>
  genes are required for proper fungal development and light response. Only
  <em>
   fbx25,
  </em>
  which encodes SconB necessary for sulphur metabolism
  <em>
   ,
  </em>
  is essential for fungal growth and survival. 50 % of the F-box proteins (30-35) are associated with the SCF complexes through the adaptor SkpA protein during fungal development. Several F-box proteins show development and stress specific interactions with the SkpA protein. 30 % of the F-box proteins are exclusively localized to the nuclear fraction whereas the rest show other localization patterns including, cytoplasmic, hyphal tip and plasma membrane. High scoring F-box proteins (Fbx1 to Fbx48) interact with more than 1500 proteins including SkpA and CullinA. These data suggest that F-box proteins interact with at least 15% of the total proteome and control developmental responses to environmental stimuli and stresses.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410040
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Conidiation    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Functional domains of the developmental regulator FlbB mediate the tip-to-nucleus communication in
 <em>
  Aspergillus nidulans
 </em>
 vegetative hyphae.
 <br/>
 Erika Herrero-García
 <sup>
  2
 </sup>
 , Elixabet Perez de Naclares-Arregui
 <sup>
  1
 </sup>
 , Marc S. Cortese
 <sup>
  1
 </sup>
 , Ane Markina-Iñarrairaegui
 <sup>
  1
 </sup>
 ,
 <b>
  Oier Etxebeste
 </b>
 <sup>
  1
 </sup>
 , Eduardo A. Espeso
 <sup>
  2
 </sup>
 , Unai O. Ugalde
 <sup>
  1
 </sup>
 . 1) Biochemistry II Lab., Dept. of Applied Chemistry, The University of The Basque Country, 20018, San Sebastian, Gipuzkoa, Spain; 2) Department of Cellular and Molecular Biology, Centro de Investigaciones Biológicas (CSIC), Ramiro de Maeztu 9, 28040 Madrid, Spain.
 <br/>
 <p>
  Polar cells have developed multiple mechanisms to convey environmental signals from the polarity site to the nucleus and induce the appropriate cellular response. These mechanisms include transcription factors located at the polarity site, such as FlbB, which signals asexual development in vegetative hyphae of the filamentous fungus
  <em>
   Aspergillus nidulans
  </em>
  . FlbB is detected at the tip and apical (but not distal) nuclei, and understanding the relationship between these pools is crucial for the elucidation of the mechanisms that induce conidiation. Photo-convertible tagging with Dendra2 demonstrated a directionality of FlbB movement from the tip to nuclei, in a process that required an N-terminally located nuclear localization signal. Tip localization of a constitutively expressed GFP::FlbB chimera was abolished in the null mutant of its apical interactor FlbE, while the nuclear pool was increased. The aconidial phenotype of this strain demonstrated that tip processing of FlbB is a prerequisite for the induction of conidiation in nuclei. The bZIP domain of FlbB is essential and sufficient to enable the interaction with FlbE. However, the retention of FlbB at the tip also requires the C-terminal domain, since the substitution of the cysteine 382 by an alanine disrupted the apical localization. Overall, these findings demonstrate that fungal-specific adaptors and the establishment of a specific three dimensional conformation are key requirements for the apical localization of FlbB and demonstrate that nuclei are asymmetrically fed with a transcriptionally active pool originating at the tip.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410209
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Proteomics    Aspergillus niger    Biomass
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Comparisons of the responses of fungi to lignocellulosic substrates using single and mixed cultures.
 <br/>
 <b>
  Paul Daly
 </b>
 <sup>
  1
 </sup>
 , Matt Kokolski
 <sup>
  1
 </sup>
 , Juliana Velasco de Castro Oliveira
 <sup>
  2
 </sup>
 , Gustavo Borin
 <sup>
  2
 </sup>
 , Martin Blythe
 <sup>
  1
 </sup>
 , Gustavo Goldman
 <sup>
  2,3
 </sup>
 , David Archer
 <sup>
  1
 </sup>
 . 1) University of Nottingham, Nottingham, United Kingdom; 2) CTBE, Campinas, São Paulo, Brazil; 3) Universidade de São Paulo, São Paulo, Brazil
 <br/>
 <p>
  Fungi are the main source of enzymes used for saccharification of lignocellulose. The efficiency of saccharification can be improved by better understanding the responses of fungi to lignocelluloses. We investigated the responses of
  <em>
   Trichoderma reesei
  </em>
  ,
  <em>
   Aspergillus niger
  </em>
  and
  <em>
   Penicillium chrysogenum
  </em>
  to lignocelluloses partly with the aim of establishing conditions for a transcriptomic study of mixed cultures using Dual-RNAseq.
 </p>
 <p>
  For single culture experiments, supernatants were sampled for proteomics from a time-course of shake-flask cultures with bagasse or wheat straw. For mixed culture experiments, mycelia from single species glucose-grown pre-cultures were combined for straw cultures and qPCR of the ITS region from each fungus was used to quantify species-specific fungal RNA.
 </p>
 <p>
  In the comparison of the proteomics responses of the fungal single cultures to straw or bagasse, there were more proteins significantly different in abundance between the
  <em>
   T. reesei
  </em>
  straw and bagasse cultures than between the
  <em>
   A. niger
  </em>
  straw and bagasse cultures. This could partly be due to a later secretory response of
  <em>
   T. reesei
  </em>
  to straw compared to bagasse whilst
  <em>
   A. niger
  </em>
  secreted proteins at similar times on both substrates. In the mixed cultures, over a period of 24 h, the relative amounts of RNA from each fungus in two species mixed cultures of either
  <em>
   A. niger
  </em>
  and
  <em>
   P. chrysogenum
  </em>
  or
  <em>
   T. reesei
  </em>
  and
  <em>
   P. chrysogenum
  </em>
  didn&#8217;t change substantially. In contrast, the relative amount of
  <em>
   A. niger
  </em>
  RNA declined in the mixed culture with
  <em>
   T. reesei.
  </em>
 </p>
 <p>
  Differences in gene content or regulation between
  <em>
   A. niger
  </em>
  and
  <em>
   T. reesei
  </em>
  could explain the delay in secretion of enzymes by
  <em>
   T. reesei
  </em>
  on straw compared to bagasse single cultures. Mixed cultures of
  <em>
   P. chrysogenum
  </em>
  with either of the other two fungi are likely to be more suitable to study gene expression in lignocellulose mixed cultures with limited antagonism than mixed cultures of
  <em>
   A. niger
  </em>
  and
  <em>
   T. reesei
  </em>
  .
  <strong>
  </strong>
  <u>
   (Financial support: FAPESP, Brazil and BBSRC, UK.)
  </u>
  .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410494
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Characterization of septum association of a Woronin body-tethering protein Leashin in
 <em>
  Aspergillus oryzae
 </em>
 .
 <br/>
 <b>
  Pei Han
 </b>
 , Jun-ichi Maruyama, Katsuhiko Kitamoto. Department of Biotechnology, The University of Tokyo, Tokyo, Japan
 <br/>
 <p>
  Woronin body, a Pezizomycotina-specific organelle, is tethered to the septum in normal growth condition, and it plugs the septal pore in response to cellular wounding, preventing the excessive loss of cytoplasm. Recently, we identified a Woronin body tethering protein AoLAH in
  <em>
   Aspergillus oryzae
  </em>
  <sup>
   1)
  </sup>
  . AoLAH is a single polypeptide with 5,727 amino acids; it is composed of conserved N- and C-terminal regions and a long non-conserved middle region. We found that AoLAH N-terminal and C-terminal regions function for Woronin body association and tethering to the septum, respectively. AoLAH middle region confers positional flexibility to the tethered Woronin bodies. However, how the AoLAH C-terminal region associates specifically with the septum is not known. The aim of this study is to investigate the mechanism for septal association of AoLAH.
 </p>
 <p>
  AoLAH C-terminal region consists of 1,018 amino acids. According to secondary structure analysis, the former 505 amino acids of this region is mostly structurally disordered while in the latter 513 amino acids several secondary structures such as &#945;-helix and &#946;-sheet were predicted. This structure prediction is similar to those of the C-terminal region of LAH proteins from other
  <em>
   Aspergillus
  </em>
  species. AoLAH C-terminal region fused with EGFP was closely associated with the septum as well as hyphal tip, and it was also observed as moving tubular structures formed in a microtubule-dependent manner. This gives the possibility that AoLAH C-terminal region could be functionally related with microtubules. After deleting the former 505 amino acids with predicted disordered feature, no motile tubular structures were observed, indicating that this part is needed for tubular structure formation and movement along hyphae. However, the localization associated at the septum and hyphal tip was still detected. Thus, the latter 513 amino acids with &#945;-helix and &#946;-sheet structures are sufficient to function for association to the septum and hyphal tip. This suggests the AoLAH C-terminal region may target to the septum and hypahl tip in a similar mechanism.       1) Han
  <em>
   et al
  </em>
  . (2014)
  <em>
   Eukaryot. Cell
  </em>
  13: 866-77.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410270
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Transcription factor    Whole Genome Sequencing
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 <em>
  Aspergillus niger
 </em>
 regulatory mutant strains for improved protein production: strain selection and mutant gene identification.
 <br/>
 <b>
  Peter Punt
 </b>
 <sup>
  1
 </sup>
 , Jing Niu
 <sup>
  2
 </sup>
 , Deepa Nair
 <sup>
  1,2
 </sup>
 , Ellen Lagendijk
 <sup>
  2
 </sup>
 , Mark Arendhorst
 <sup>
  2
 </sup>
 , Arthur Ram
 <sup>
  2
 </sup>
 . 1) Microbiology, TNO, Zeist, Netherlands; 2) Leiden University, Institute of Biology Leiden, Molecular Microbiology and Biotechnology, Sylviusweg 72, 2333 BE Leiden, The Netherlands
 <br/>
 <p>
  Optimized protein production in filamentous fungi requires the availability of fungal strains with low levels of secreted protease activity. Already for several decades research has been carried out to obtain these type of mutants, leading to the isolation of mutants with very favourable characteristics. Complementation studies have allowed identification of several of these mutants, one being a mutation in a transcriptional regulatory gene, prtT (e.g. Punt et al;., 2008).
 </p>
 <p>
  Based on this mutant strain further improved strains have been selected using positive selection approaches. Controlled fermentation experiments with these strains revealed different protease profiles, whereas full genome sequencing was carried out in an attempt to identify the genetic basis of the mutant phenotypes.
 </p>
 <p>
  Another method to identify regulatory protease deficient mutants, was based on the use of collections of regulatory gene knock-out strains in N. crassa and A. niger. Based on positive selection approaches and classical milkhalo screening novel mutant strains with modified protease production profiles were obtained.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410311
 </b>
 <br/>
 Research Area:
 <b>
  Other
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Industrial strain construction: Improving the toolbox.
 <br/>
 <b>
  Peter van de Vondervoort
 </b>
 , Rene Verwaal, Yvonne Arendsen, Noël van Peij. Genetics, DSM Biotechnology Center, Delft, Netherlands
 <br/>
 <p>
  For the industrial production of enzymes at DSM we use various micro-organisms such as
  <em>
   Aspergillus niger
  </em>
  ,
  <em>
   Kluyveromyces lactis
  </em>
  and
  <em>
   Bacillus subtilis
  </em>
  . These hosts have a long history of safe use. For each new enzyme, a production host needs to be developed to produce the protein of interest. The field of molecular biology is developing rapidly. More tools become available to design strains in a rationalized way, build them in a faster way and/or increase the throughput of testing strains. A number of tools developed will be described such as advanced transformation and cloning methods and a Cre-Lox based method for marker removal. We will present an overview of recently developed tools and how they can be used in improved, rationalized and faster strain construction with a higher success rate.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410643
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Transcription factor    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Regulation of copper homeostasis in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Philipp Wiemann
 </b>
 <sup>
  1
 </sup>
 , Joshua A. Baccile
 <sup>
  2
 </sup>
 , Frank C. Schroeder
 <sup>
  2
 </sup>
 , Nancy P. Keller
 <sup>
  1
 </sup>
 . 1) Medical Microbiology &amp; Immunology, University of Wisconsin-Madison, Madison, WI; 2) Boyce Thompson Institute and Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY
 <br/>
 <p>
  Regulation of metal acquisition and detoxification plays a critical role in survival and virulence of the opportunistic pathogen
  <em>
   Aspergillus fumigatus
  </em>
  . Macrophages phagocytizing
  <em>
   A. fumigatus
  </em>
  spores are thought to employ two major strategies for killing the pathogen, 1. Acidification and export of reduced Fe
  <sup>
   2+
  </sup>
  and 2. Production of superoxide by the NADPH oxidase (NOX) complex and influx of Cu
  <sup>
   +
  </sup>
  ions. Regulation of iron homeostasis is best understood in
  <em>
   A. fumigatus
  </em>
  whereas very little is known about copper-dependent regulation. We recently demonstrated that the non-ribosomal peptide, hexadehydroastechrome (HAS; a tryptophan-derived iron (III)-complex) is involved in
  <em>
   A
  </em>
  .
  <em>
   fumigatus
  </em>
  virulence and iron homeostasis
  <sup>
   1,2
  </sup>
  . Recently we found that HAS also influences genes involved in copper acquisition and detoxification. Here we present our latest results on copper binding of HAS and regulation of copper-dependent genes mediated by homologs of the
  <em>
   Saccharomyces cerevisiae
  </em>
  transcription factors Mac1/Cuf1 and Ace1/Cup2 in
  <em>
   A. fumigatus
  </em>
  .
 </p>
 <p>
  <sup>
   1
  </sup>
  Yin
  <em>
   et a
  </em>
  l., (2013)
  <em>
   J Am Chem Soc
  </em>
  <strong>
   135
  </strong>
  :2064-2067.
 </p>
 <p>
  <sup>
   2
  </sup>
  Wiemann
  <em>
   et al
  </em>
  ., (2014)
  <em>
   Front Microbiol
  </em>
  <strong>
   5
  </strong>
  :530.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410016
 </b>
 <br/>
 Research Area:
 <b>
  Pathogenic and Mutalistic Interactions
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Aspergillosis    Metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Invited
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Fungal bioenergetics drives human fungal pathogenicity: Focus on
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 Dawoon Chung, Arsa Thammahong, Sourabh Dhingra, Sarah Beattie,
 <b>
  Robert Cramer
 </b>
 . Microbiology and Immunology, Geisel School of Medicine at Dartmouth, Hanover, NH.
 <br/>
 <p>
  Over the course of evolutionary history, mammals have evolved elegant mechanisms to resist severe infections by fungi. However, over the last four decades, the incidence of often lethal invasive fungal infections has dramatically risen. While individuals that are susceptible to an invasive fungal infection are most often immune compromised, the spectrum of fungi that causes disease remains limited. Many genes and proteins have been found to be critical for pathogenicity in these select fungi, but it remains to be definitively determined why only certain fungi are capable of causing invasive disease in specific immune compromised individuals. Utilizing the most common filamentous fungal agent of human disease,
  <em>
   Aspergillus fumigatus
  </em>
  , and closely related so called non-pathogens, we are exploring the hypothesis that fungal bioenergetics is the primary driving force behind human fungal pathogenicity. Our hypothesis predicts that measurments of fungal bioenergetics between and within species may allow one to predict the virulence of a given fungal isolate in a given patient. Key genes and biochemical pathways are being identified that drive these virulence associated bioenergetics. Targeting virulence bioenergetics through drug mediated and non-drug mediated mechanisms, in a patient context dependent manner, is a promising novel therapeutic approach to tackle these recalcitrant and devastating infections. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410085
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Gene Expression    Other (Other Keyword: Hydrophobin)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Possible roles of hydrophobic surface interacting proteins in the breakdown of lignocellulose by
 <em>
  Aspergillus niger
 </em>
 .
 <br/>
 <b>
  R. Raulo
 </b>
 , J. van Munster, P. Daly, R. Fetherston, M. Kokolski, D. Archer. Life Sciences, Unviersity of Nottingham, Nottingham, United Kingdom
 <br/>
 <p>
  Enzymes from filamentous fungi have a key role in the degradation of many of the biopolymers found in nature, including cellulose and hemicelluloses. For this reason, these enzymes are of great interest in the industrial conversion of lignocellulose into biofuels. RNA-sequencing analysis has been carried out to investigate the transcriptional changes that occur when
  <em>
   Aspergillus niger
  </em>
  is transferred from the simple carbon source glucose onto the complex lignocellulosic biomass wheat straw. This has highlighted the up-regulation in transcript level of genes encoding glycosyl hydrolase (GH) enzymes as well as hydrophobic surface interacting proteins (HSIPs) that may be involved in the interface between lignocellulosic biomass and
  <em>
   A. niger
  </em>
  . Possible roles for the HSIPs include: mediating GH enzyme-substrate interactions, fungal-substrate interactions and the induction of the fungal response to lignocellulose.
 </p>
 <p>
  To investigate the role of HSIPs in the response of
  <em>
   A. niger
  </em>
  to wheat straw, single gene deletion strains for
  <em>
   hfbD
  </em>
  ,
  <em>
   hyp1
  </em>
  and
  <em>
   hsbA
  </em>
  as well as the double deletion strain (
  <em>
   hfbD
  </em>
  <em>
   hyp1
  </em>
  ) have been constructed. The expression of selected genes encoding GH enzymes was then followed in these strains using qRT-PCR. The results showed that the transcript levels from the GH genes studied were lowered in the HSIPs deletion strains when compared to the wild-type strain, when the cultures were transferred from glucose medium to wheat straw. Also, a role for pH in regulating the expression of GH-encoding genes became apparent because variations in pH in cultures were observed between the different HSIPs deletion strains. As the expression of some GH-encoding genes is known to be pH-dependent, pH could be an important factor for efficient saccharification of wheat straw in
  <em>
   A. niger
  </em>
  . The precise nature of such a role is under further investigation and may provide new areas of improvement for industrial processes for production of second generation biofuels.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410498
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Gene Expression    Gene Deletion
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 <em>
  Hyperproduction of biomass-degrading
 </em>
 enzymes by double deletion of CreA and CreB involved in carbon catabolite repression in
 <em>
  Aspergillus oryzae
 </em>
 .
 <br/>
 <b>
  Sakurako Ichinose
 </b>
 , Mizuki Tanaka, Takahiro Shintani, Katsuya Gomi. Tohoku university, Sendai, Miyagi, Japan
 <br/>
 <p>
  <em>
   Aspergillus oryzae
  </em>
  produces large amounts of amylolytic enzymes in the presence of maltooligosaccharides. In the presence of glucose, however, their production is repressed by carbon catabolite repression system. In
  <em>
   Aspergillus nidulans
  </em>
  , it has been proposed that glucose repression is regulated by the transcription factor CreA via ubiquitination and deubiquitination, and CreA stability and activity are modulated by the ubiquitin processing protease CreB in the presence of glucose. We generated the single and double knockout mutants of
  <em>
   creA
  </em>
  and
  <em>
   creB
  </em>
  genes in
  <em>
   A. oryzae
  </em>
  , and demonstrated that &#945;-amylase production levels of
  <em>
   &#916;creA&#916;creB
  </em>
  were much higher than those of the
  <em>
   &#916;creA
  </em>
  and
  <em>
   &#916;
  </em>
  <em>
   creB
  </em>
  (1). In this study, to examine the effects of
  <em>
   creA
  </em>
  and
  <em>
   creB
  </em>
  deletion on production levels of various hydrolytic enzymes other than &#945;-amylase, cellulase, xylanase, and &#946;-glucosidase activities of each knockout mutant were compared with those of the wild-type strain.
 </p>
 <p>
  In xylose medium, the wild-type and
  <em>
   &#916;creB
  </em>
  strains showed a considerably poor growth compared with
  <em>
   creA
  </em>
  knockout mutants and thus dry mycelia weights of
  <em>
   creA
  </em>
  mutants were approximately 3-fold higher than the wild-type. This suggested that CreA repressed the utilization of xylose, probably through repressing the transcription of XlnR. Although no apparent differences in cellulase activities were observed between all mutant strains and the wild-type, xylanase activities of
  <em>
   creA
  </em>
  knockout mutants were much higher than those of the wild-type. In particular,
  <em>
   &#916;creA&#916;creB
  </em>
  had approximately 100-fold higher xylanase activity than the wild-type. In addition, &#946;-glucosidase activity of
  <em>
   &#916;creA&#916;creB
  </em>
  was also much higher than those of
  <em>
   &#916;creA
  </em>
  and
  <em>
   &#916;creB.
  </em>
  These results indicated that double knockout of
  <em>
   creA
  </em>
  and
  <em>
   creB
  </em>
  is also substantially effective in achieving high-level production of lignocellulolytic enzymes.
 </p>
 <p>
  (1) Ichinose et al.,
  <em>
   Appl. Microbiol. Biotechnol.
  </em>
  <strong>
   98
  </strong>
  , 335&#8211;343 (2014).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410369
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Signal Transduction    Sexual Development    Aspergillus nidulans
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 The conserved and divergent roles of the MAP kinase gene,
 <em>
  mpkB
 </em>
 , in
 <em>
  Aspergillus flavus
 </em>
 .
 <br/>
 <b>
  Sang-Cheol Jun
 </b>
 <sup>
  1
 </sup>
 , Kwang-Yeop Jahng
 <sup>
  1
 </sup>
 , Dong-Min Han
 <sup>
  2
 </sup>
 , Kap-Hoon Han
 <sup>
  3
 </sup>
 , Jong-Hwa Kim
 <sup>
  3
 </sup>
 . 1) Life Science, Chonbuk National University, Jeonju, Jeonbuk, South Korea; 2) Division of Life Science, Wonkwang University, Iksan, 570-749, South Korea; 3) Department of Pharmaceutical Engineering, Woosuk University, Wanju, 565-701, South Korea
 <br/>
 <p>
  Eukaryotes&#8217; developmental processes are controlled by the multiple regulatory systems. Among them, MAP kinase pathways play important roles in regulation of growth, development, and stress responses. To characterize the function of MAP kinase in an important pathogenic and toxigenic fungus
  <em>
   Aspergillus flavus
  </em>
  , the
  <em>
   AflmpkB
  </em>
  gene (AFL2G_02589), an orthologue of the yeast
  <em>
   FUS3
  </em>
  gene and the
  <em>
   mpkB
  </em>
  gene of
  <em>
   Aspergillus nidulans
  </em>
  , was deleted and analyzed. In
  <em>
   A. nidulans
  </em>
  , previous studies revealed that MpkB positively regulates the sexual and asexual differentiation as well as secondary metabolite production. In this study, deletion of
  <em>
   AflmpkB
  </em>
  resulted in no mycelial growth change, while the conidial production was reduced about 60% comparing to the wild-type. Also, the mutant produced immature and abnormal conidiophores such as vesicular dome-immaturity in the conidiophore head, decreased number of the phialides and very short stalks, although expression of the
  <em>
   brlA
  </em>
  gene, a key regulator of conidiation, was up-regulated in the mutant. Moreover, &#916;
  <em>
   AflmpkB
  </em>
  couldn&#8217;t produce any sclerotia, suggesting that the
  <em>
   AflmpkB
  </em>
  gene plays an important role not only in conidiophore generation but also in sclerotia development. However, unlike
  <em>
   A. nidulans
  </em>
  where the sterigmatocystin biosynthesis was strongly decreased in
  <em>
   mpkB
  </em>
  mutant,
  <em>
   AflmpkB
  </em>
  mutants produced normal level of aflatoxin B
  <sub>
   1
  </sub>
  . Taking together,
  <em>
   A. flavus
  </em>
  <em>
   mpkB
  </em>
  gene plays a positive regulatory role in the production of the conidiation and the sclerotia formation but not in the production of the secondary metabolites including aflatoxin B
  <sub>
   1
  </sub>
  . .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410114
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Cell Cycle
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 A novel role for an SR/RRM family mRNA shuttling binding protein in cell cycle regulation in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  S. L. Anglin
 </b>
 <sup>
  1
 </sup>
 , S. James
 <sup>
  2
 </sup>
 . 1) Dept. Biology, Millsaps College, Jackson, MS.; 2) Dept. Biology, Gettysburg College, Gettysburg, PA.
 <br/>
 <p>
  SR/RRM proteins are a class of small nuclear ribonucleoproteins known to have functions in spliceosome assembly and catalysis as well as in mRNA transcription and export.  Heretofore, this family of proteins has not been implicated in the regulation of cell division.  We recently reported that the
  <em>
   Aspergillus nidulans
  </em>
  <em>
   snxA
  </em>
  gene encodes an ortholog of budding yeast Hrb1/Gbp2 and of human hnRNP-M, members of the SR/RRM protein family.  hnRNP-M is a ubiquitous and highly expressed nuclear protein in human tissues, and is known to control alternative splicing of developmentally regulated genes.  SNXA localizes to the nucleus but is excluded from the nucleolus in
  <em>
   A. nidulans
  </em>
  , paralleling the localization of hnRNP-M.  The
  <em>
   snxA1
  </em>
  mutation was originally identified as an extragenic suppressor of mutations in the G2/M regulatory gene
  <em>
   nimX
   <sup>
    cdc2
   </sup>
  </em>
  , and we have shown that it suppresses mutations in multiple components of the CDC2/CYCLINB regulatory pathway, including
  <em>
   nimT23
   <sup>
    cdc25
   </sup>
  </em>
  and
  <em>
   nimE6
   <sup>
    cyclinB
   </sup>
  </em>
  but not
  <em>
   nimA5
  </em>
  or
  <em>
   nimA1
  </em>
  .  We further found that both
  <em>
   snxA1
  </em>
  and a second allele,
  <em>
   snxA2
  </em>
  , are hypomorphic and that both mutations result in significantly decreased
  <em>
   snxA
  </em>
  transcription and SNXA protein levels.  Additionally, mutation or deletion of
  <em>
   snxA
  </em>
  alters NIME
  <sup>
   CYCLINB
  </sup>
  localization patterns.  Our data suggest that SNXA may normally function to restrain the G2/M transition by affecting the CDC2/CYCLINB regulatory pathway, suggesting a novel function for the SR/RRM family that links RNA metabolism and transport to regulated cell division.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410098
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Secondary metabolite    Gene Expression
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Exploring the potential of
 <em>
  Aspergillus niger
 </em>
 as secondary metabolite producer.
 <br/>
 <b>
  Simon Boecker
 </b>
 <sup>
  1
 </sup>
 , Sophia Zobel
 <sup>
  2
 </sup>
 , David Schirmer
 <sup>
  2
 </sup>
 , Roderich D Suessmuth
 <sup>
  2
 </sup>
 , Vera Meyer
 <sup>
  1
 </sup>
 . 1) Dept. of Applied and Molecular Microbiology, Berlin Institute of Technology, Berlin, Germany; 2) Dept. of Biological Chemistry, Berlin Institute of Technology, Berlin, Germany
 <br/>
 <p>
  Recently, we could show that
  <em>
   Aspergillus niger
  </em>
  is an excellent expression host for the production of fungal secondary metabolites (SMs).
  <sup>
   1
  </sup>
  For the proof-of-concept study, we heterologously expressed the 350 kDa non-ribosomal enniatin synthetase (ESYN) from
  <em>
   Fusarium oxysporum
  </em>
  in
  <em>
   A. niger.
  </em>
  ESYN catalyzes the formation of cyclic depsipeptides of the enniatin family, which exhibit antimicrobial, antiviral and anticancer activities. The encoding gene
  <em>
   esyn1
  </em>
  was put under control of the tunable Tet-On expression system. By using optimum cultivation and feeding conditions, yields up to 4.5 g L
  <sup>
   -1
  </sup>
  were achieved in
  <em>
   A. niger
  </em>
  fed batch bioreactor cultivation. This titer by far outpaces yields obtained in other microbial expression hosts, such as
  <em>
   B. subtilis,
  </em>
  which produces around 1 mg L
  <sup>
   -1
  </sup>
  of enniatin.
  <sup>
   2
  </sup>
 </p>
 <p>
  In addition, another cyclohexadepsipeptide (beauvericin) with anticancer and insecticidal properties can be produced with comparable high titers in
  <em>
   A. niger
  </em>
  . Furthermore,
  <em>
   A. niger
  </em>
  is a suitable expression host for new-to-nature artificial chimeric peptide synthetases. Corresponding data will be shown.
 </p>
 <p>
  [1]
  <strong>
   Richter L, Wanka F, Boecker S, Storm D, Kurt T, Vural Ö, Süßmuth R, Meyer V
  </strong>
  . 2014. Engineering of
  <em>
   Aspergillus niger
  </em>
  for the production of secondary metabolites. Fungal Biol. Biotechnol.
  <strong>
   1
  </strong>
  :4.
 </p>
 <p>
  [2]
  <strong>
   Zobel S, Kumpfmüller J, Süssmuth RD, Schweder T
  </strong>
  . 2014.
  <em>
   Bacillus subtilis
  </em>
  as heterologous host for the secretory production of the non-ribosomal cyclodepsipeptide enniatin. Appl. Microbiol. Biotechnol. (Epub ahead of print).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410632
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Proteomics    Virulence
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Regulation of SrbA in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Sourabh Dhingra
 </b>
 <sup>
  1
 </sup>
 , Dawoon Chung
 <sup>
  1
 </sup>
 , Özgür Bayram
 <sup>
  2
 </sup>
 , Jay C. Buckey
 <sup>
  3
 </sup>
 , Robert A. Cramer
 <sup>
  1
 </sup>
 . 1) Department of Microbiology and Immunology, Geisel School of Medicine at Dartmouth, Hanover, NH.; 2) Department of Biology, Maynooth University, National University of Ireland, Kildare, Ireland; 3) Department of Medicine, Geisel School of Medicine at Dartmouth, Lebanon, NH.
 <br/>
 <p>
  <em>
   Aspergillus fumigatus
  </em>
  (Afum) is the primary causal agent of invasive aspergillosis. In the infection site microenvironment, Afum encounters and adapts to hypoxia, a necessary virulence trait. The sterol regulatory element binding proteins (SREBPs), SrbA and SrbB, are necessary for hypoxia adaptation and virulence, however, mechanisms of SrbA/SrbB regulation remain to be elucidated. Importantly, Afum like other Eurotiomycetes appears to lack the sterol sensing protein SCAP critical for SREBP regulation in other eukaryotes. A golgi E3 ubiquitin ligase complex encoded by the
  <em>
   dsc
  </em>
  genes is critical for proteolytic cleavage and regulation of SrbA. Yet, the mechanisms by which this cleavage event is regulated remain to be determined. Proteins that directly interact with SrbA are candidates for potential novel regulatory mechanisms.To identify SrbA interacting proteins, we performed GFP-trap pull-down experiments with GFP attached to either the N terminus or C terminus of SrbA and constitutively active SrbA. Our results identified different classes of proteins bound to the N terminus and C terminus of SrbA, suggesting potential layers of regulatory mechanisms. To identify upstream genetic regulatory elements of SrbA activation, we are also initiating protein pull downs with DscA::S-tag as the bait protein.As the SrbA genetic network responds to hypoxia and is critical for virulence, we are also testing the effects of alleviating hypoxia at the infection site by using hyperbaric oxygen (HBO). Our preliminary results show HBO therapy (HBOT) can reduce the rate of fungal germination and metabolism
  <em>
   in vitro,
  </em>
  in part through inhibition of the SrbA genetic network. In a murine model of IPA HBOT reduced fungal burden and increased survival. Taken together, our data suggest that manipulation of infection site oxygen availability can alter the physiology of the invading fungus in part through manipulation of a major fungal oxygen sensing pathway. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410703
 </b>
 <br/>
 Research Area:
 <b>
  Gene Regulation
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Secondary metabolism    Toxin
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Regulating Nonribosomal Peptide Synthesis: Post-biosynthetic bis-Thiomethylation of Gliotoxin Attenuates Gliotoxin Biosynthesis in
 <em>
  Aspergillus fumigatus
 </em>
 .
 <br/>
 <b>
  Stephen Dolan
 </b>
 , Rebecca Owens, Grainne O'Keeffe, Gary Jones, Sean Doyle. Department of Biology, Maynooth University, Kildare, Ireland
 <br/>
 <p>
  Gliotoxin (GT) is a redox-active nonribosomal peptide produced by
  <em>
   Aspergillus fumigatus
  </em>
  . Like many other disulfide-containing epipolythiodioxopiperazines, a bis-thiomethylated form is also produced. In the case of GT, bisdethiobis(methylthio)gliotoxin (BmGT) is formed for unknown reasons by a cryptic enzyme. Here, we identify the S-adenosylmethionine dependent gliotoxin bis-thiomethyltransferase (GtmA), which converts dithiol GT to BmGT.
 </p>
 <p>
  Disruption of this previously unclassified non-
  <em>
   gli
  </em>
  cluster encoded methyltransferase completely abrogated organismal ability to biosynthesize and secrete BmGT, while GT production and secretion were increased (p=0.0056). Surprisingly, exposure of
  <em>
   A. fumigatus &#916;gtmA
  </em>
  to exogenous GT did not reveal the acquisition of a sensitive phenotype compared to the wild-type strain. Thus, GtmA-mediated GT bismethylation is not essential for self-protection against GT. The activity of GtmA, which is induced by exogenous GT, is only detectable in protein lysates of
  <em>
   A. fumigatus
  </em>
  deficient in the gliotoxin oxidoreductase,
  <em>
   gliT
  </em>
  . Recombinant GtmA was shown to bismethylate dithiol GT using S-adenosyl methionine as methyl donor, via a novel LC-MS enabled activity assay. GtmA is the first characterised methyltransferase capable of substrate bis-thiomethylation in any organism. Label-free quantitative (LFQ) proteomics of
  <em>
   A. fumigatus
  </em>
  wild-type,
  <em>
   &#916;gtmA
  </em>
  , and
  <em>
   gtmAC
  </em>
  strains cultured in Czapek-Dox media revealed an elevated abundance of
  <em>
   gli
  </em>
  cluster encoded enzymes in the
  <em>
   &#916;gtmA
  </em>
  mutant strain. Also, LFQ proteomics revealed that exogenously added GT and BmGT induce differential remodelling of the
  <em>
   A. fumigatus
  </em>
  proteome. Phylogenetic analysis of this enzyme revealed that there are 124 GtmA homologs within the Ascomycota phylum.
 </p>
 <p>
  We now propose that the purpose of GtmA mediated BmGT formation primarily serves to attenuate GT biosynthesis. This appears to be the first example of postbiosynthetic regulation of nonribosomal peptide synthesis in any organism.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410475
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus oryzae    Other (Other Keyword: ionic interaction, hydrophobin RolA, cutinase CutL)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 D73 of
 <em>
  Aspergillus oryzae
 </em>
 cutinase CutL1 is cooperatively involved in the ionic interaction between fungal hydrophobin RolA and CutL1 with other acidic amino acid residues of CutL1.
 <br/>
 Y. Kim
 <sup>
  1
 </sup>
 , Y. Terauchi
 <sup>
  2
 </sup>
 ,
 <b>
  T. Tanaka
 </b>
 <sup>
  1
 </sup>
 , Y. Tsushima
 <sup>
  1
 </sup>
 , K. Uehara
 <sup>
  1
 </sup>
 , T. Takahashi
 <sup>
  3
 </sup>
 , K. Abe
 <sup>
  1,2,3
 </sup>
 . 1) Grad. Sci. Agric. Sci., Tohoku university, Sendai, Miyagi, Japan; 2) Dept. Agric., Tohoku university, Sendai, Miyagi, Japan; 3) NICHe., Tohoku university, Sendai, Miyagi, Japan
 <br/>
 <p>
  Hydrophobin RolA and polyesterase CutL1 are co-expressed when the fungus
  <em>
   Aspergillus oryzae
  </em>
  is grown in a liquid medium containing the biodegradable polyester polybutylene succinate-
  <em>
   co
  </em>
  -adipate (PBSA) as the sole carbon source. RolA which adheres to PBSA interacts with CutL1 and promotes the PBSA degradation by concentrating CutL1 on the PBSA surface. In our previous studies, we revealed that positively charged amino acid residues (H32, K34) of RolA and negatively charged amino acid residues (E31, D142, D171) of CutL1 are cooperatively play an important role in the ionic interaction between RolA and CutL1. However, the amount of a CutL1 variant (E31S/D142S/D171S) recruited by a RolA variant (H32S/K34S) in the presence of NaCl (250 mM) was decreased significantly compared to that without NaCl. This result suggested some remaining charged resides in CutL1-E31S/D142S/D171S are participated in the ionic interaction with RolA.
 </p>
 <p>
  In the present study, in order to elucidate negatively charged residues involved in RolA-CutL1 interaction other than the three residues (E31, D142, and D171) of CutL1, we selected several candidates that putatively participate in the interaction based on the alignment analysis among CutL1 homologues and analysis using 3D-structure of CutL1. Among the candidates in CutL1, we chose D73 and constructed CutL1 variants of which D73 was substituted with serine. We performed kinetic analysis of interaction between CutL1-D73S and CutL1-E31S/D73S/D142S/D171S with wild-type RolA by using Quartz Crystal Microbalance and pull-down assay with RolA-coated Teflon. The D73S substitution of CutL1 showed a decreased affinity to RolA, suggesting that D73 also cooperatively participates in the ionic interaction with RolA by the multivalent effect with other negatively charged residues of CutL1 (E31, D142, D171).
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410625
 </b>
 <b>
  <font color="#ff0000">
   Withdrawn
  </font>
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Secondary metabolism
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 Redox regulation of secondary metabolism in
 <em>
  Aspergillus nidulans
 </em>
 .
 <br/>
 <b>
  Tamas Emri
 </b>
 <sup>
  1
 </sup>
 , Vera Szarvas
 <sup>
  1
 </sup>
 , Erzsébet Orosz
 <sup>
  1
 </sup>
 , Károly Antal
 <sup>
  2
 </sup>
 , HeeSoo Park
 <sup>
  3
 </sup>
 , Kap-Hoon Han
 <sup>
  4
 </sup>
 , Jae-Hyuk Yu
 <sup>
  3
 </sup>
 , István Pócsi
 <sup>
  1
 </sup>
 . 1) University of Debrecen, Debrecen, Hungary; 2) Eszterházy Károly College, Eger, Hungary; 3) University of Wisconsin, Madison, WI; 4) Woosuk University, Wanju, Republic of Korea
 <br/>
 <p>
  AtfA is a b-Zip type transcription factor which plays an important role in the regulation of oxidative and osmotic stress responses in
  <em>
   Aspergillus nidulans
  </em>
  . Genome-wide expression changes caused by various oxidative and NaCl stresses were studied in a wild type and a
  <em>
   DatfA
  </em>
  mutant strain. Stress conditions either induced or repressed expression of some secondary metabolite gene clusters in the wild type strain.
  <em>
   E.g.
  </em>
  induction of the AN7884 cluster was observed during menadione sodium bisulfite (MSB) stress, while repression of the emericellamide cluster was detected in MSB,
  <em>
   t
  </em>
  -butylhydroperoxide and diamide stresses. Deletion of
  <em>
   atfA
  </em>
  induced transcription of certain clusters (
  <em>
   e.g.
  </em>
  pkf cluster) under unstressed conditions and enhanced the stress responsiveness of further clusters including
  <em>
   xptA
  </em>
  containing cluster, mdp cluster and AN8504 cluster. However, the majority of secondary metabolite gene clusters were not stress-responsive at all. Stress dependent regulation seems to be a frequent but far not a general feature of the regulation of secondary metabolism in
  <em>
   A. nidulans
  </em>
  . Deletion of an oxidative stress response regulator gene in combination with mild oxidative stress can be applicable to overproduce the products of certain gene clusters, which may lead to identification of new secondary metabolites.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410604
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus niger    Evolution    Genome Sequencing
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Comparative genomics and gene cluster identification in 28 species of
 <em>
  Aspergillus
 </em>
 section
 <em>
  Nigri
 </em>
 .
 <br/>
 <b>
  Tammi Vesth
 </b>
 <sup>
  1
 </sup>
 , Jane Nybo
 <sup>
  1
 </sup>
 , Sebastian Theobald
 <sup>
  1
 </sup>
 , Ellen K. Lyhne
 <sup>
  1
 </sup>
 , Martin E. Kogle
 <sup>
  1
 </sup>
 , Igor Grigoriev
 <sup>
  3
 </sup>
 , Uffe H. Mortensen
 <sup>
  1
 </sup>
 , Scott E. Baker
 <sup>
  2
 </sup>
 , Mikael R. Andersen
 <sup>
  1
 </sup>
 . 1) Department of Systems Biology, Technical University of Denmark, Kgs. Lyngby, Denmark; 2) Joint BioEnergy Institute, Emeryville, CA and Environmental Molecular Sciences Laboratory, Pacific Northwest National Laboratory, Richland, WA; 3) Joint Genome Institute, Walnut Creek, CA, USA
 <br/>
 <p>
  The filamentous fungus
  <em>
   Aspergillus niger
  </em>
  and its close relatives in
  <em>
   Aspergillus
  </em>
  section
  <em>
   Nigri
  </em>
  are of broad interest to the scientific community including applied, medical and basic research. The fungi are prolific producers of native and heterologous proteins, organic acids (in particular citrate), and secondary metabolites (including bioactives and toxins such as ochratoxin A). Because of these abilities they represent a substantial economic interests in bioenergy applications. While 8 individual species from this group has been whole-genome sequenced, the genetic basis for these diverse phenotypes remains largely unidentified.
  <br/>
  In this study, we have de novo sequenced the genomes of 20 additional species of the section
  <em>
   Nigri
  </em>
  , thus allowing the genome comparison of all members of this important section of fungal species. Here we present the results of this large-scale genomic analysis where we have examined the core genome of these 28 species and identified variations in the genetic makeup of individual species and groups of species. In particular, we have found genes unique to
  <em>
   Aspergillus
  </em>
  section
  <em>
   Nigri
  </em>
  , as well as genes which are only found in subgroups of the section. Our analysis here correlates these genes to the phenotypes of the fungi.
  <br/>
  Furthermore, we have predicted secondary metabolite gene clusters in all 28 species. We present here an overview of these gene clusters and how they are shared and vary between species. We also correlate the presence of gene clusters to presence of known fungal metabolites.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410283
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Other (Other Keyword: Septation)
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Lab Head
 </b>
 <br/>
 <br/>
 The
 <em>
  sepG
 </em>
 gene in
 <em>
  Aspergillus nidulans
 </em>
 encodes an IQGAP homologue.
 <br/>
 <b>
  Terry Hill
 </b>
 <sup>
  1
 </sup>
 , Loretta Jackson-Hayes
 <sup>
  2
 </sup>
 , Kristen Wendt
 <sup>
  1
 </sup>
 . 1) Dept Biol, Rhodes College, Memphis, TN.; 2) Dept. Chem, Rhodes College, Memphis, TN.
 <br/>
 <p>
  The study of septation in filamentous fungi received a significant boost twenty years ago through the identification and characterization of several temperature-sensitive septation (
  <em>
   sep
  </em>
  ) mutants in
  <em>
   Aspergillus nidulans
  </em>
  by Steven Harris and colleagues (Harris et al., 1994, Genetics 136: 517-532).   One continuing loose end from that study has been the identity of the mutation designated
  <em>
   sepG1
  </em>
  .  Through meiotic mapping of
  <em>
   sepG1
  </em>
  &#8217;s position on Chromosome II along with sequencing of candidate genes in the indicated chromosomal region, we have identified
  <em>
   sepG
  </em>
  as
  <em>
   Aspergillus nidulans
  </em>
  gene AN9463.  The predicted gene product shows 35% identity and 55% similarity to the
  <em>
   Schizosaccharomyces pombe
  </em>
  Rng2 protein, described as an IQGAP homologue involved in septation.  The mutation in
  <em>
   sepG1
  </em>
  is a G-to-A transition at position 5082 of the 5333-nucleotide open reading frame, predicted to cause a glycine-to-arginine substitution at residue 1637 of the 1737-amino acid product.  The
  <em>
   sepG1
  </em>
  phenotype is complemented by the cloned wild type allele.  The
  <em>
   N
  </em>
  -terminal GFP-tagged product of wild type AN9463 (GFP::SepG) colocalizes with actin and myosin during cell division, beginning with the &#8220;actin/myosin tangle&#8221; phase that precedes mature contractile ring formation, and colocalization continues through ring contraction and dissipation.  GFP::SepG localization at septation sites is blocked by the
  <em>
   sepH1
  </em>
  and
  <em>
   sepA1
  </em>
  mutations.  The
  <em>
   sepD5
  </em>
  mutation does not prevent localization of GFP::SepG in peripheral rings, but it does block ring constriction.  Reduced expression of wild type AN9463 under the regulatable
  <em>
   AlcA
  </em>
  promoter blocks septum formation and the localization of a number of GFP-tagged septal ring-associated proteins such as AspB and the
  <em>
   A. nidulans
  </em>
  homologue of
  <em>
   S. cerevisiae
  </em>
  Cdc14.  However, in a strain expressing both C-terminal tagged MyoB::GFP and the
  <em>
   in vivo
  </em>
  actin-labeling probe Lifeact::mRFP, down-regulation of AN9463 did not prevent actin or myosin from colocalizing as a ring at putative septation sites. .
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410786
 </b>
 <br/>
 Research Area:
 <b>
  Comparative and Functional Genomics
 </b>
 <br/>
 Keyword(s):
 <b>
  Microarray    Saccharomyces cerevisiae    Oxidative Stress
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 Autophagy relieves the stress of reactive oxygen species (ROS) triggered by caloric restriction (CR).
 <br/>
 <b>
  Thomas Hahn
 </b>
 <sup>
  1,2
 </sup>
 , Fusheng Tang
 <sup>
  1
 </sup>
 , Helen Benes
 <sup>
  2
 </sup>
 , Richard Segall
 <sup>
  3
 </sup>
 . 1) Information Science, University of Arkansas at Little Rock (UALR), Little Rock, AR.; 2) University of Arkansas Medical Science (UAMS), Little Rock, AR; 3) Arkansas State University, Jonesboro, AR
 <br/>
 <p>
  We previously observed that CR extends lifespan of wild type yeast cells but shortens the lifespan of mutants missing either Atg15 or Erg6. CR also causes ROS to accumulate in the mutants, suggesting that CR may trigger some responses detrimental to lifespan.
 </p>
 <p>
  To identify such responses, we compared the expression profile of wild type (WT) and mutants under normal and CR-media. In atg15-delta, we found that mitochondrial respiratory chain, mitochondrial organization, and mitochondrial protein localization are differentially regulated by CR than they are in WT. Since a longevity mechanism of CR is the up-regulation of respiration, the altered responses of mitochondria in atg15-delta may explain the accumulation of ROS in the cytoplasm of atg15-delta in CR.
 </p>
 <p>
  In erg6-delta, more pathways are differentially regulated by CR than in atg15-delta. Cellular amino acid metabolic process, biosynthetic process, and sterol metabolic process are the top 3 pathways that are differentially modulated by CR compared to WT. Inspection of individual pathways showed that CR up-regulates the generation of NADH and NADPH in WT cells. However, many of these genes cannot be up-regulated in erg6-delta. In WT cells, the elevation of NADH and NADPH may counteract the oxidative stress caused by CR.
 </p>
 <p>
  Erg6-delta cannot up-regulate the production of NADH/NADPH because of the accumulation of ROS in CR.  Atg15 is required for lifespan extension longevity mutants mimicking CR. Thus, we propose that autophagy relieves the ROS-stress induced by CR.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410698
 </b>
 <br/>
 Research Area:
 <b>
  Pathogenic and Mutalistic Interactions
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Virulence
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Postdoc
 </b>
 <br/>
 <br/>
 Conidia of the human pathogenic fungus
 <em>
  Aspergillus fumigatus
 </em>
 interfere
 <em>
 </em>
 with the maturation of macrophage phagolysosomes.
 <br/>
 Hella Schmidt, Andreas Thywi&#946;en, Axel Brakhage,
 <b>
  Thorsten Heinekamp
 </b>
 . Molecular and Applied Microbiology, Hans Knoell Institute, Jena, Germany
 <br/>
 <p>
  <em>
   Aspergillus fumigatus
  </em>
  is a major cause for fungal infection in immunocompromised hosts. Conidia of this pathogen are ubiquitous in nature and enter the human host via the airway, where they infect the lung tissue and intrude to the lower respiratory system.
 </p>
 <p>
  Invading conidia in the lung tissue are phagocytosed by alveolar macrophages and degraded in the phagolysosome. However,
  <em>
   A. fumigatus
  </em>
  prevents
  <em>
  </em>
  its killing by arresting the maturation of the phagolysosome. The pigment dihydroxynaphthalene (DHN)-melanin is crucial in this process.
 </p>
 <p>
  Here, we aim to decipher in more detail the immune evasion strategy of
  <em>
   A. fumigatus
  </em>
  conidia from phagolysosomes. For that purpose, melanized wild-type conidia and non-melanized
  <em>
   pksP
  </em>
  mutant conidia were co-incubated with macrophages and the ratio of phagocytosis events and acidification of phagolysosomes was monitored. Furthermore, phagolysosomes containing conidia of both strains were purified from cell extracts. The proteome of the organelle was assessed by means of western blot analyses and liquid chromatograpy/mass spectrometry.
 </p>
 <p>
  Conidia-containing phagolysosomes were purified by labeling the conidia with magnetic beads. Their identity was confirmed by the detection of specific phagolysosomal marker proteins. Proteome analysis revealed several candidates important in the maturation process.
 </p>
 <p>
  We propose a mechanism by which conidial DHN-melanin interferes with the acquisition of a specific protein composition of phagolysosomes, hampering thereby the efficient intracellular clearance of invading
  <em>
   A. fumigatus
  </em>
  conidia.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410109
 </b>
 <br/>
 Research Area:
 <b>
  Cell Biology and Development
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus fumigatus    Morphogenesis    Protein Localization
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 The
 <em>
  Aspergillus fumigatus
 </em>
 farnesyltransferase &#946;-subunit, Ram1, regulates Ras protein localization, conidial viability and antifungal susceptibility.
 <br/>
 <b>
  Tiffany S. Norton
 </b>
 , Rachel V. Lovingood, Qusai Al Abdallah, Jarrod R. Fortwendel. University of South Alabama, Mobile, AL.
 <br/>
 <p>
  Post-translational prenylation mechanisms, including farnesylation and geranylgeranylation, mediate both subcellular localization and protein-protein interaction in eukaryotes. The farnesyltransferase (FT) enzyme complex is composed of two subunits: the &#945;-subunit, an essential protein shared with the geranylgeranyltransferase complex; and a &#946;-subunit, termed Ram1. FT activity is an important mediator of Ras pathway signaling via control of Ras protein localization. Our previous data show that
  <em>
   A. fumigatus
  </em>
  RasA localizes primarily to the plasma membrane where it functions in processes controlling morphogenesis and virulence. However, the importance of FT activity to Ras protein function, filamentous fungal growth, and
  <em>
   A. fumigatus
  </em>
  virulence is currently unknown. To explore this, we generated an
  <em>
   A. fumigatus
  </em>
  deletion mutant lacking the FT &#946;-subunit (&#916;
  <em>
   ram1
  </em>
  ). Conidial germination rate was reduced in the &#916;
  <em>
   ram1
  </em>
  mutant, with a concomitant reduction in conidial viability of 45%. Although no polarity defects of hyphae were apparent, the &#916;
  <em>
   ram1
  </em>
  mutant displayed reduced radial growth rate, an average increase in hyphal width of 26%, and altered nuclear positioning in growing hyphae. Furthermore, loss of
  <em>
   ram1
  </em>
  resulted in resistance to triazole antifungal drugs such as voriconazole. Complementation of the &#916;
  <em>
   ram1
  </em>
  mutant with the
  <em>
   ram1
  </em>
  gene (&#916;
  <em>
   ram1+ram1
  </em>
  ) restored the wild-type phenotype for each of these processes. To define molecular mechanisms for Ram1-mediated processes, we generated strains expressing GFP-RasA in the &#916;
  <em>
   ram1
  </em>
  genetic background. The absence of
  <em>
   ram1
  </em>
  resulted in mislocalization of RasA from the plasma membrane. Interestingly, mutation of RasA to enhance selectivity for geranylgeranylation as an alternative membrane targeting mechanism in the absence of Ram1 restored RasA plasma membrane localization but not radial growth. These data suggest that Ras-independent mechanisms are at least partially responsible for phenotypes exhibited by the &#916;
  <em>
   ram1
  </em>
  mutant. Together, these data point to a crucial role for the Ram1 farnesyltransferase in mediating
  <em>
   A. fumigatus
  </em>
  growth and antifungal susceptibility.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body><body>
 Abstract Number:
 <b>
  145410601
 </b>
 <br/>
 Research Area:
 <b>
  Biochemistry and Metabolism
 </b>
 <br/>
 Keyword(s):
 <b>
  Aspergillus nidulans    Gene Expression    Secondary metabolite
 </b>
 <br/>
 Presentation Preference:
 <b>
  Poster
 </b>
 <br/>
 Current Status:
 <b>
  Graduate Student
 </b>
 <br/>
 <br/>
 <em>
  Aspergillus nidulans
 </em>
 as cell factory for production of mycophenolic acid.
 <br/>
 <b>
  Zofia D. Jarczynska
 </b>
 <sup>
  1
 </sup>
 , Jakob B. Nielsen
 <sup>
  1
 </sup>
 , Freja Aasted
 <sup>
  1
 </sup>
 , Dorte M. K. Holm
 <sup>
  2
 </sup>
 , Kiran R. Patil
 <sup>
  3
 </sup>
 , Kristian F. Nielsen
 <sup>
  1
 </sup>
 , Uffe H. Mortensen
 <sup>
  1
 </sup>
 . 1) Systems Biology, DTU, Kgs. Lyngby, Denmark; 2) Novozymes A/S, Denmark; 3) European Molecular Biology Laboratory, Heidelberg, Germany
 <br/>
 <p>
  Filamentous fungi are well-known producers of a wide range of valuable secondary metabolites (SMs), which can be advantageously exploited e.g. in pharmaceutical industry. One of the most prominent examples is mycophenolic acid (MPA), an immunosuppressant molecule that inhibits inosine-5&#8217;-monophosphate dehydrogenase (IMPDH). IMPDH catalyzes the rate limiting step in the guanine nucleotide synthesis in B- and T-lymphocytes. Recent studies have successfully identified the gene cluster, coding for the MPA synthesis, in
  <em>
   Penicillium brevicompactum
  </em>
  . Moreover, it has been demonstrated that two first steps in MPA production are catalysed, respectively, by polyketide synthase (PKS), MpaC, producing 5-methylorsellinic acid (5-MOA), and MpaDE, which strikingly is a natural fusion enzyme catalysing the production of 5,7-dihydroxy-4-methylphtalide (DHMP). Additionally,
  <em>
   mpaF
  </em>
  has been characterized as IMPDH-encoding gene which confers the resistance to MPA. In order to characterize the remaining part of the MPA biosynthetic pathway, we have heterologously expressed the
  <em>
   mpa
  </em>
  cluster genes in a stepwise manner in
  <em>
   Aspergillus nidulans
  </em>
  . We have demonstrated that MpaA possesses prenyl transferase activity and catalyzes the conversion from DHMP to 6-farnesyl-5,7-dihydroxy-4-methylphtalide (FDHMP). To our surprise, this strain was also able to produce demethyl-MPA, which is the next intermediate in MPA biosynthesis. Interestingly, we have found two homologs of
  <em>
   mpaH
  </em>
  in
  <em>
   A. nidulans
  </em>
  , which is hypothesized to encode the conversion of FDHMP to demethyl-MPA, and we speculate that one or both of these genes deliver hydrolase activity similar to the one encoded by MpaH. Lastly, we have confirmed that MpaH and MpaG catalyze the last two enzymatic steps in the biosynthesis of MPA, resulting in the production of demethyl-MPA and MPA, respectively. In conclusion, we have successfully characterized the full biosynthetic pathway of the top-selling drug, MPA. Moreover, we have demonstrated that
  <em>
   A. nidulans
  </em>
  is a suitable cell factory for heterologous production of MPA.
 </p>
 <br/>
 <hr noshade="" size="1"/>
 <br/>
</body></html>